
<!-- saved from url=(0082)https://unimed2.nlm.nih.gov/MRD/Documentation/2024AA/APPENDIX/attribute_names.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><link rel="stylesheet" href="./attribute_names_files/appendix_style.css">
</head><body><a name="top"></a>
<button id="myBtn"><a href="https://unimed2.nlm.nih.gov/MRD/Documentation/2024AA/APPENDIX/attribute_names.html#top" style="color: white">Back to Top</a></button><table>
<thead>
<tr>
<th>ATN (Attribute Name)</th><th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td>AAL_TERM</td><td>AAL term</td>
</tr>
<tr>
<td>ACCEPTABILITYID</td><td>Acceptability Id</td>
</tr>
<tr>
<td>ACCEPTED_THERAPEUTIC_USE_FOR</td><td>Accepted therapeutic use for</td>
</tr>
<tr>
<td>ACTIVE</td><td>Active</td>
</tr>
<tr>
<td>ADDED_MEANING</td><td>Additional descriptive information</td>
</tr>
<tr>
<td>ADDITIONAL_GUIDELINE</td><td>Additional explanatory text that is applicable to a concept (code/heading/subheading).</td>
</tr>
<tr>
<td>ADDON_CODE</td><td>A "T" in this field indicates that it is an "Add-on" code, i.e. it is commonly carried out in addition to the primary procedure performed</td>
</tr>
<tr>
<td>AGR</td><td>Alliance of Genome Resources</td>
</tr>
<tr>
<td>AMBIGUITY_FLAG</td><td>Source atom ambiguity flag</td>
</tr>
<tr>
<td>AMT</td><td>AOT uses MeSH term</td>
</tr>
<tr>
<td>ANADA</td><td>Abbreviated New Animal Drug application number for the generic drug for MTHSPL</td>
</tr>
<tr>
<td>ANATOMICAL_COORDINATE</td><td>Anatomical coordinate</td>
</tr>
<tr>
<td>ANDA</td><td>Abbreviated New (Generic) Drug application number for the MTHSPL drug</td>
</tr>
<tr>
<td>ANSWER_CODE_SYSTEM</td><td>Answer code system</td>
</tr>
<tr>
<td>ANSWER_CODE</td><td>Answer code</td>
</tr>
<tr>
<td>ANSWER_LIST_ID</td><td>Answer list ID</td>
</tr>
<tr>
<td>ANSWER_LIST_NAME</td><td>Answer list name</td>
</tr>
<tr>
<td>ANSWER_LIST_OID</td><td>Answer list OID</td>
</tr>
<tr>
<td>AN</td><td>MeSH Annotation - an informative MeSH note written primarily for indexers or catalogers that may also be useful in explaining the use of a MeSH term to online searchers.</td>
</tr>
<tr>
<td>AQL</td><td>MeSH Allowable Qualifier - list of allowable qualifier abbreviations for MeSH main headings (e.g. AA, CL, CS, DF, DU, IM, I,P ME, PK)</td>
</tr>
<tr>
<td>ASK_AT_ORDER_ENTRY</td><td>Ask at Order Entry (AOE) observations for a clinical observation or laboratory test</td>
</tr>
<tr>
<td>ASSOC_OBSERVATIONS</td><td>Associated observations</td>
</tr>
<tr>
<td>ATC_LEVEL</td><td>ATC LEVEL</td>
</tr>
<tr>
<td>ATTRIBUTEDESCRIPTION</td><td>Attribute Description</td>
</tr>
<tr>
<td>ATTRIBUTEORDER</td><td>Attribute Order</td>
</tr>
<tr>
<td>ATTRIBUTETYPE</td><td>Attribute Type</td>
</tr>
<tr>
<td>AUTHORITY_NAME</td><td>Authority name</td>
</tr>
<tr>
<td>AUTHORITY</td><td>Authority</td>
</tr>
<tr>
<td>BIOCARTA_ID</td><td>BioCarta online maps of molecular pathways, adapted for NCI use ID</td>
</tr>
<tr>
<td>BLA</td><td>Therapeutic Biologic Applications number for the MTHSPL drug</td>
</tr>
<tr>
<td>BLOCK</td><td>Block code</td>
</tr>
<tr>
<td>BRAININFO_URL</td><td>URL of the central directory page in BrainInfo</td>
</tr>
<tr>
<td>CASE_SIGNIFICANCE_ID</td><td>Case significance ID</td>
</tr>
<tr>
<td>CAS_REGISTRY</td><td>CAS Registry</td>
</tr>
<tr>
<td>CATEGORY</td><td>Category</td>
</tr>
<tr>
<td>CCDS_ID</td><td>CCDS ID</td>
</tr>
<tr>
<td>CCF</td><td>Canonical Clinical Problem Statement System (CCPSS) frequency - the number of times a CCPSS term appears in a patient record. </td>
</tr>
<tr>
<td>CCI</td><td>ICD-9-CM code(s) clusters in a Clinical Classifications Software (CCS) category - individual ICD-9-CM codes (or ranges of such codes) classified into CCS categories. </td>
</tr>
<tr>
<td>CELL_APPENDAGE_TYPE</td><td>Cell appendage type</td>
</tr>
<tr>
<td>CFR</td><td>Code of Federal Regulation Number (e.g. 862.3220, 892.1610)</td>
</tr>
<tr>
<td>CHANGE_CLASS</td><td>Identifies the type of change made to this version of the Code (i.e., changes from the prior version published in "CDT-2011-2012"), with the following meanings: 30 = No change to existing code, nomenclature or descriptor; 31 = New procedure code and nomenclature, and descriptor, if present; 35 = Revision in the nomenclature or descriptor (or both), or to indicate a revision to an entire subcategory; 39 = Editorial (e.g., syntax; spelling); 99 = Entire CDT Code entry is no longer valid (i.e., deleted) in this version.</td>
</tr>
<tr>
<td>CHANGE_REASON_PUBLIC</td><td>Detailed explanation about special changes to the term over time.</td>
</tr>
<tr>
<td>CHAPTER</td><td>Chapter code</td>
</tr>
<tr>
<td>CHARACTERISTIC_TYPE_ID</td><td>Characteristic type ID</td>
</tr>
<tr>
<td>CHEBI_ID</td><td>CHEBI ID</td>
</tr>
<tr>
<td>CHEMICAL_FORMULA</td><td>Chemical Formula</td>
</tr>
<tr>
<td>CHROMOSOME_PAIRS_PER_NUCLEUS</td><td>Chromosome pairs per nucleus</td>
</tr>
<tr>
<td>CITATION</td><td>Citation</td>
</tr>
<tr>
<td>CLINVAR_VARIATION_ID</td><td>CLINVAR VARIATION ID</td>
</tr>
<tr>
<td>CMA_LABEL</td><td>Center for Morphological Analysis</td>
</tr>
<tr>
<td>CNU</td><td>ISO 3166-1 standard country code in numeric (three-digit) format</td>
</tr>
<tr>
<td>COATING</td><td>FDA Structured Product Label imprint attribute for coating</td>
</tr>
<tr>
<td>CODE_ALSO</td><td>Instructs that 2 codes may be required to fully describe a condition but the sequencing of the two codes is discretionary, depending on the severity of the conditions and the reason for the encounter.</td>
</tr>
<tr>
<td>CODE_FIRST</td><td>Certain conditions have both an underlying etiology and multiple body system manifestations due to the underlying etiology. ICD-10-CM has a coding convention that requires the underlying condition be sequenced first followed by the manifestation. Wherever such a combination exists there is a "code first" note at the manifestation code. These instructional notes indicate the proper sequencing order of the codes, etiology followed by manifestation. In most cases the manifestation codes will have in the code title, "in diseases classified elsewhere." Codes with this title are a component of the etiology/manifestation convention. The code title indicates that it is a manifestation code. "In diseases classified elsewhere" codes are never permitted to be used as first listed or principle diagnosis codes. They must be used in conjunction with an underlying condition code and they must be listed following the underlying condition.</td>
</tr>
<tr>
<td>CODING_TIP</td><td>Coding Tip</td>
</tr>
<tr>
<td>COLORTEXT</td><td>FDA Structured Product Label imprint attribute for color text</td>
</tr>
<tr>
<td>COLOR</td><td>FDA Structured Product Label imprint attribute for color</td>
</tr>
<tr>
<td>COMBO_SCORE_NO_TOP_WORDS</td><td>Combo Score that has no top words</td>
</tr>
<tr>
<td>COMBO_SCORE</td><td>Combo Score</td>
</tr>
<tr>
<td>COMMENT</td><td>Comment</td>
</tr>
<tr>
<td>COMMON_ORDER_RANK</td><td>Ranking of approximately 300 common orders performed by laboratories in USA</td>
</tr>
<tr>
<td>COMMON_TEST_RANK</td><td>Numeric ranking of approximately 2,000 common tests performed by hospitals</td>
</tr>
<tr>
<td>CONCENTRATION_STRENGTH_DENOMINATOR_VALUE</td><td>Concentration strength denominator value (defining)</td>
</tr>
<tr>
<td>CONCENTRATION_STRENGTH_NUMERATOR_VALUE</td><td>Concentration strength numerator value (defining)</td>
</tr>
<tr>
<td>CONCEPT_TYPE</td><td>Concept type</td>
</tr>
<tr>
<td>CONDITIONAL_NADA</td><td>Conditional Nada</td>
</tr>
<tr>
<td>CONSIDER</td><td>Codes to consider before definitive diagnosis</td>
</tr>
<tr>
<td>CONSUMER_NAME</td><td>An experimental (beta) consumer friendly name</td>
</tr>
<tr>
<td>CONTEXT_SCORE</td><td>Context Score</td>
</tr>
<tr>
<td>CONTRIBUTING_AUTHOR</td><td>contributing author</td>
</tr>
<tr>
<td>COPYRIGHT</td><td>Copyright</td>
</tr>
<tr>
<td>CORRELATIONID</td><td>Correlation Id</td>
</tr>
<tr>
<td>COUNT_OF_ACTIVE_INGREDIENT</td><td>Count of active ingredient (defining)</td>
</tr>
<tr>
<td>COUNT_OF_BASE_AND_MODIFICATION_PAIR</td><td>Count of base and modification pair</td>
</tr>
<tr>
<td>COUNT_OF_BASE_OF_ACTIVE_INGREDIENT</td><td>Count of base of active ingredient (defining)</td>
</tr>
<tr>
<td>CPF</td><td>CPT Full Description - complete text of the CPT full description, in cases where the CPT term in the "STR" field of MRCON has been trimmed from its original form.</td>
</tr>
<tr>
<td>CPTLINK_CONCEPT_ID</td><td>CPT Link concept identifier</td>
</tr>
<tr>
<td>CPT_LEVEL</td><td>For headings, a value ranging from H1 (a top-level heading such as Surgery) to H6 (a 6th-level subheading) and HS (a heading for a small family of related codes). For CPT codes, either PC for Parent Code or CC for Child Code.</td>
</tr>
<tr>
<td>CTV3ID</td><td>The Read Code for a SNOMED CT concept taken from the United Kingdom?s Clinical Terms Version 3 terminology.</td>
</tr>
<tr>
<td>CUI_SCORE</td><td>CUI Score</td>
</tr>
<tr>
<td>CV_ALGORITHM</td><td>Content view algorithm</td>
</tr>
<tr>
<td>CV_CATEGORY</td><td>Content view category</td>
</tr>
<tr>
<td>CV_CLASS</td><td>Content view class</td>
</tr>
<tr>
<td>CV_CODE</td><td>Content view code</td>
</tr>
<tr>
<td>CV_CONTRIBUTOR_DATE</td><td>Date corresponding to the contributor version of this concept view</td>
</tr>
<tr>
<td>CV_CONTRIBUTOR_URL</td><td>URL corresponding to the contributor version of this concept view</td>
</tr>
<tr>
<td>CV_CONTRIBUTOR_VERSION</td><td>Version of this content view submitted by the contributor</td>
</tr>
<tr>
<td>CV_CONTRIBUTOR</td><td>Content view contributor</td>
</tr>
<tr>
<td>CV_DESCRIPTION</td><td>Content view description</td>
</tr>
<tr>
<td>CV_INCLUDE_OBSOLETE</td><td>Content view includes obsolete data: YN</td>
</tr>
<tr>
<td>CV_IS_GENERATED</td><td>Content view generated: Y/N</td>
</tr>
<tr>
<td>CV_MAINTAINER_DATE</td><td>Date corresponding to the maintainer version of this concept view</td>
</tr>
<tr>
<td>CV_MAINTAINER_URL</td><td>URL corresponding to the maintainer version of this concept view</td>
</tr>
<tr>
<td>CV_MAINTAINER_VERSION</td><td>Version of this content view submitted by the maintainer</td>
</tr>
<tr>
<td>CV_MAINTAINER</td><td>Content view maintainer</td>
</tr>
<tr>
<td>CV_MEMBER</td><td>Describes the membership for an atom belonging to a particular content view. The value for this attribute is a tilde-delimited microsyntax. The first field is the content view bitflag (e.g. 2048), followed by field name-value pairs describing the member.</td>
</tr>
<tr>
<td>CV_PREVIOUS_META</td><td>Previous UMLS Metathesaurus version used to generate content view. A null value means the content view is generated based on current UMLS Metathesaurus version.</td>
</tr>
<tr>
<td>CX</td><td>MeSH Consider Also Note - other word roots or prefixes that should be consulted for concepts related to this MeSH concept, e.g., the value for "Heart" is "consider also terms at cardi- and myocardi-".</td>
</tr>
<tr>
<td>CYTOGENETIC_LOCATION</td><td>Cytogenetic location of the gene</td>
</tr>
<tr>
<td>DATE_CREATED</td><td>Date created</td>
</tr>
<tr>
<td>DATE_FIRST_PUBLISHED</td><td>Date first published</td>
</tr>
<tr>
<td>DATE_LAST_MODIFIED</td><td>Date last modified</td>
</tr>
<tr>
<td>DATE_MODIFIED</td><td>Date Modified</td>
</tr>
<tr>
<td>DATE_NAME_CHANGED</td><td>Date name changed</td>
</tr>
<tr>
<td>DATE_SYMBOL_CHANGED</td><td>Date symbol changed</td>
</tr>
<tr>
<td>DAYS_POST_FERTILIZATION</td><td>Days post fertilization</td>
</tr>
<tr>
<td>DAYS_RESTRICTION</td><td>Days restriction</td>
</tr>
<tr>
<td>DA</td><td>Metathesaurus Date of entry - YYYYMMDD, e.g., 19920830 - date of entry of the concept into the Metathesaurus. </td>
</tr>
<tr>
<td>DB_XR_ID</td><td>Database cross-reference ID</td>
</tr>
<tr>
<td>DCSA</td><td>Controlled Substance Act designation code (e.g. 0,2,3n)</td>
</tr>
<tr>
<td>DC</td><td>MeSH Descriptor class - type of MeSH term the concept name represents.</td>
</tr>
<tr>
<td>DDFA</td><td>Drug Doseform Abbreviation (e.g. SOLN)</td>
</tr>
<tr>
<td>DDF</td><td>Drug Doseform (e.g. chewable tablet)</td>
</tr>
<tr>
<td>DEFAULT_LANGUAGECODE</td><td>Default language code value</td>
</tr>
<tr>
<td>DEFAULT_MODULE_ID</td><td>Default module ID</td>
</tr>
<tr>
<td>DEFINITION_STATUS_ID</td><td>Definition status ID</td>
</tr>
<tr>
<td>DESCRIPTIONFORMAT</td><td>Description Format</td>
</tr>
<tr>
<td>DESCRIPTIONLENGTH</td><td>Description Length</td>
</tr>
<tr>
<td>DESCRIPTION_FORMAT</td><td>Description format</td>
</tr>
<tr>
<td>DESCRIPTION_LENGTH</td><td>Description length</td>
</tr>
<tr>
<td>DESCRIPTION</td><td>Description</td>
</tr>
<tr>
<td>DESCRIPTOR_ELEMENT</td><td>Elements of a multi-paragraph descriptor, such as, E&amp;M, lab panels, and molecular pathology</td>
</tr>
<tr>
<td>DESCRIPTOR_EXTENSION</td><td>Extension of full description of the concept</td>
</tr>
<tr>
<td>DESIGN_NOTE</td><td>Design note.</td>
</tr>
<tr>
<td>DESI_DESC</td><td>Source drug efficacy study implementation indicator</td>
</tr>
<tr>
<td>DEVTYPE</td><td>Device Type</td>
</tr>
<tr>
<td>DHJC</td><td>Multum HCPCS J-code Multum clinical drugs linked to HCPCS J-codes where applicable (e.g. J7507)</td>
</tr>
<tr>
<td>DID</td><td>Descriptor Identifier</td>
</tr>
<tr>
<td>DIETARY_SUPPLEMENT</td><td>Dietary Supplement</td>
</tr>
<tr>
<td>DIMENSION</td><td>Dimension code</td>
</tr>
<tr>
<td>DISJOINT_FROM</td><td>Disjoint from</td>
</tr>
<tr>
<td>DISPARAGED</td><td>Disparaged</td>
</tr>
<tr>
<td>DIV</td><td>NCBI Division/Phyla (e.g. DIV[NCBI]Viruses)</td>
</tr>
<tr>
<td>DM_SPL_ID</td><td>DailyMed internal identifier for MTHSPL atom</td>
</tr>
<tr>
<td>DPC</td><td>Multum Pregnancy Hazard Classification Code assigned to Multum clinical drugs. (e.g. X, D) </td>
</tr>
<tr>
<td>DQ</td><td>MeSH Date Qualifier Established YYYYMMDD - date the qualifier became available for indexing MEDLARS citations.</td>
</tr>
<tr>
<td>DRTA</td><td>Drug Route of Administration Abbreviation (e.g. INJ)</td>
</tr>
<tr>
<td>DRT</td><td>Drug Route of Administration (e.g. Injection (systemic) )</td>
</tr>
<tr>
<td>DRUG_CLASS_TYPE</td><td>VA Drug class type - Values can be "Major","Minor","Sub Class", or "0","1","2" respectively</td>
</tr>
<tr>
<td>DST</td><td>Drug Strength (e.g. 0.01%, 0.02 MG, 0.02 MG/ML)</td>
</tr>
<tr>
<td>DX</td><td>MeSH Date major descriptor established YYYYMMDD - first day of the Index Medicus publication month in which the descriptor (in any form) was available for searching as a major descriptor.</td>
</tr>
<tr>
<td>EC</td><td>Specifies the MeSH main heading to be used instead of the Descriptor/Qualifier combination specified in the EC_OF attribute.</td>
</tr>
<tr>
<td>EFFECTIVE_TIME</td><td>Effective time</td>
</tr>
<tr>
<td>ENA</td><td>International Nucleotide Sequence Database Collaboration accession number(s)</td>
</tr>
<tr>
<td>ENSEMBLGENE_ID</td><td>Ensembl gene ID</td>
</tr>
<tr>
<td>ENTREZGENE_ID</td><td>EntrezGene ID</td>
</tr>
<tr>
<td>EPONYM</td><td>Eponym</td>
</tr>
<tr>
<td>EQUIVALENCE</td><td>Equivalence</td>
</tr>
<tr>
<td>ESSENTIAL_AMINO_ACID</td><td>Essential Amino Acid</td>
</tr>
<tr>
<td>ESSENTIAL_FATTY_ACID</td><td>Essential Fatty Acid</td>
</tr>
<tr>
<td>EUPROSTATE16</td><td>Euprostate 16</td>
</tr>
<tr>
<td>EUPROSTATE27</td><td>Euprostate 27</td>
</tr>
<tr>
<td>EXAMPLE_UCUM_UNITS</td><td>The Unified Code for Units of Measure (UCUM)</td>
</tr>
<tr>
<td>EXAMPLE_UNITS</td><td>Example units</td>
</tr>
<tr>
<td>EXCLUDES1</td><td>A pure excludes. It means "NOT CODED HERE!" Indicates that the code excluded should never be used at the same time as the code above the Excludes1 note. Used for when two conditions cannot occur together, such as a congenital form versus an acquired form of the same condition.</td>
</tr>
<tr>
<td>EXCLUDES2</td><td>Represents "Not included here". Indicates that the condition excluded is not part of the condition it is excluded from but a patient may have both conditions at the same time. When an Excludes2 note appears under a code it is acceptable to use both the code and the excluded code together.</td>
</tr>
<tr>
<td>EXCLUDE_DI_CHECK</td><td>Exclude drug interaction check</td>
</tr>
<tr>
<td>EXEMPT_DEVICE</td><td>Exempt Device</td>
</tr>
<tr>
<td>EXEMPT_MOD</td><td>Contains 51 or 63 for codes where Modifier 51 or 63 is Exempt</td>
</tr>
<tr>
<td>EXTENSIBLE_LIST</td><td>Extensible List</td>
</tr>
<tr>
<td>EXTERNAL_COPYRIGHT_LINK</td><td>External copyright link</td>
</tr>
<tr>
<td>EZ</td><td>Enzyme Commission Number - International Union of Biochemists Enzyme Commission number for an enzyme concept.</td>
</tr>
<tr>
<td>FDA_APPROVAL_PENDING</td><td>T for codes marked with the FDA approval pending symbol</td>
</tr>
<tr>
<td>FDA_TABLE</td><td>FDA Table</td>
</tr>
<tr>
<td>FDA_UNII_CODE</td><td>FDA UNII Code</td>
</tr>
<tr>
<td>FIRST_IN_SUBSET</td><td>Version of subset first containing this concept</td>
</tr>
<tr>
<td>FOURTHLEVEL</td><td>Fourth Level code</td>
</tr>
<tr>
<td>FREESURFER_TERM</td><td>Freesurfer term</td>
</tr>
<tr>
<td>FREQUENCY</td><td>Frequency</td>
</tr>
<tr>
<td>FROMRSAB</td><td>Root source abbreviation for the "from" identifiers of a map set</td>
</tr>
<tr>
<td>FROMVSAB</td><td>Versioned source abbreviation for the "from" identifiers of a map set</td>
</tr>
<tr>
<td>FR</td><td>MeSH Frequency</td>
</tr>
<tr>
<td>FX</td><td>MeSH MH Mapping - maps a MeSH MH to a 'See Related' MH.</td>
</tr>
<tr>
<td>GENBANK_ACCESSION_NUMBER</td><td>GenBank Accession Number</td>
</tr>
<tr>
<td>GENCC</td><td>HGNC ID used within the GenCC database</td>
</tr>
<tr>
<td>GENELOCUS</td><td>Gene Locus</td>
</tr>
<tr>
<td>GENESYMBOL</td><td>Gene Symbol</td>
</tr>
<tr>
<td>GENE_ENCODES_PRODUCT</td><td>Gene Encodes Product</td>
</tr>
<tr>
<td>GENE_FAM_DESC</td><td>Gene family description</td>
</tr>
<tr>
<td>GENE_FAM_ID</td><td>ID used to designate a gene family or group</td>
</tr>
<tr>
<td>GESTATION_AGE_WEEKS</td><td>Gestation Age Weeeks</td>
</tr>
<tr>
<td>GLOBAL_PERIOD_CODE</td><td>Global period code</td>
</tr>
<tr>
<td>GO_COMMENT</td><td>GO Comment field data</td>
</tr>
<tr>
<td>GO_NAMESPACE</td><td>Go Namespace field data</td>
</tr>
<tr>
<td>GO_SUBSET</td><td>Go Subset field data</td>
</tr>
<tr>
<td>GUIDELINE</td><td>Guideline</td>
</tr>
<tr>
<td>GXR</td><td>GO Cross Reference to external databases (e.g. MetaCyc:TRNA-CHARGING-PWY)</td>
</tr>
<tr>
<td>HAC</td><td>HCPCS action code - code denoting the change made to a procedure or modifier code within the HCPCS system.</td>
</tr>
<tr>
<td>HAD</td><td>HCPCS Action Effective Date - effective date of action to a procedure or modifier code.</td>
</tr>
<tr>
<td>HAQ</td><td>HCPCS Anesthesia Base Unit Quantity - base unit represents the level of intensity for anesthesia procedure services that reflects all activities except time.</td>
</tr>
<tr>
<td>HAS_BOUNDARY</td><td>Has boundary</td>
</tr>
<tr>
<td>HAS_DIMENSION</td><td>Has dimension</td>
</tr>
<tr>
<td>HAS_DIRECT_CELL_LAYER</td><td>Has direct cell layer</td>
</tr>
<tr>
<td>HAS_DIRECT_PLOIDY</td><td>Has direct ploidy</td>
</tr>
<tr>
<td>HAS_INHERENT_3D_SHAPE</td><td>has inherent 3d shape</td>
</tr>
<tr>
<td>HAS_MASS</td><td>Has mass</td>
</tr>
<tr>
<td>HBT</td><td>HCPCS Berenson-Eggers Type of Service Code - BETOS for the procedure code based on generally agreed upon clinically meaningful groupings of procedures and services.</td>
</tr>
<tr>
<td>HCC</td><td>HCPCS Coverage Code - code denoting Medicare coverage status. There are two subelements separated by "=".</td>
</tr>
<tr>
<td>HCD</td><td>HCPCS Code Added Date - year the HCPCS code was added to the HCFA Common Procedure Coding System.</td>
</tr>
<tr>
<td>HCO</td><td>HL7 Comment</td>
</tr>
<tr>
<td>HGNC_ID</td><td>HGNC ID</td>
</tr>
<tr>
<td>HIR</td><td>HCPCS Coverage Issues Manual Reference Section Number - number identifying the Reference Section of the Coverage Issues Manual.</td>
</tr>
<tr>
<td>HIT</td><td>HL7 Interpretation: German Interpretation according to the German HL/ user group</td>
</tr>
<tr>
<td>HL7AI</td><td>HL7 approved indicator</td>
</tr>
<tr>
<td>HL7AP</td><td>HL7 associated concept property</td>
</tr>
<tr>
<td>HL7AR</td><td>HL7 package artifact</td>
</tr>
<tr>
<td>HL7AT</td><td>HL7 applies to</td>
</tr>
<tr>
<td>HL7BR</td><td>HL7 binding realm name</td>
</tr>
<tr>
<td>HL7CC</td><td>HL7 complete codes indicator</td>
</tr>
<tr>
<td>HL7CD</td><td>HL7 conductible</td>
</tr>
<tr>
<td>HL7CI</td><td>HL7 package combined Id</td>
</tr>
<tr>
<td>HL7CP</td><td>HL7 copyright owner</td>
</tr>
<tr>
<td>HL7CSC</td><td>HL7 conceptual space for class code</td>
</tr>
<tr>
<td>HL7CST</td><td>HL7 coding strength</td>
</tr>
<tr>
<td>HL7CS</td><td>HL7 code status</td>
</tr>
<tr>
<td>HL7CY</td><td>HL7 copyright years</td>
</tr>
<tr>
<td>HL7DC</td><td>HL7 is document characteristic</td>
</tr>
<tr>
<td>HL7DI</td><td>HL7 deprecation info</td>
</tr>
<tr>
<td>HL7DK</td><td>HL7 definition kind</td>
</tr>
<tr>
<td>HL7ED</td><td>HL7 effective date</td>
</tr>
<tr>
<td>HL7HA</td><td>HL7 how applies</td>
</tr>
<tr>
<td>HL7HI</td><td>HL7 history item</td>
</tr>
<tr>
<td>HL7ID</td><td>HL7 internal Id</td>
</tr>
<tr>
<td>HL7II</td><td>HL7 is immutable</td>
</tr>
<tr>
<td>HL7IM</td><td>HL7 is mandatory indicator</td>
</tr>
<tr>
<td>HL7IN</td><td>HL7 inverse name</td>
</tr>
<tr>
<td>HL7IR</td><td>HL7 inverse relationship</td>
</tr>
<tr>
<td>HL7IS</td><td>HL7 is selectable</td>
</tr>
<tr>
<td>HL7MI</td><td>HL7 maintained indicator</td>
</tr>
<tr>
<td>HL7MN</td><td>HL7 vocabulary model name</td>
</tr>
<tr>
<td>HL7NA</td><td>HL7 is navigable</td>
</tr>
<tr>
<td>HL7OA</td><td>HL7 owning affiliate</td>
</tr>
<tr>
<td>HL7OD</td><td>HL7 OID</td>
</tr>
<tr>
<td>HL7PK</td><td>HL7 package kind</td>
</tr>
<tr>
<td>HL7PL</td><td>HL7 preferred for language</td>
</tr>
<tr>
<td>HL7PR</td><td>HL7 package root</td>
</tr>
<tr>
<td>HL7RA</td><td>HL7 rendering application</td>
</tr>
<tr>
<td>HL7RD</td><td>HL7 release date</td>
</tr>
<tr>
<td>HL7RF</td><td>HL7 reflexivity</td>
</tr>
<tr>
<td>HL7RK</td><td>HL7 relationship kind</td>
</tr>
<tr>
<td>HL7RN</td><td>HL7 package realm namespace</td>
</tr>
<tr>
<td>HL7RT</td><td>HL7 rendering time</td>
</tr>
<tr>
<td>HL7SCS</td><td>HL7 supported code system</td>
</tr>
<tr>
<td>HL7SL</td><td>HL7 supported language</td>
</tr>
<tr>
<td>HL7SV</td><td>HL7 schema version</td>
</tr>
<tr>
<td>HL7SY</td><td>HL7 symmetry</td>
</tr>
<tr>
<td>HL7TI</td><td>HL7 title</td>
</tr>
<tr>
<td>HL7TR</td><td>HL7 transitive</td>
</tr>
<tr>
<td>HL7TY</td><td>HL7 property type</td>
</tr>
<tr>
<td>HL7UN</td><td>HL7 usage notes</td>
</tr>
<tr>
<td>HL7VC</td><td>HL7 value set content</td>
</tr>
<tr>
<td>HL7VD</td><td>HL7 version date</td>
</tr>
<tr>
<td>HL7VE</td><td>HL7 package version</td>
</tr>
<tr>
<td>HL7XN</td><td>HL7 xml namespace</td>
</tr>
<tr>
<td>HL7_ATTACHMENT_REQUEST</td><td>HL7 attachment request</td>
</tr>
<tr>
<td>HL7_ATTACHMENT_STRUCTURE</td><td>HL7 attachment structure</td>
</tr>
<tr>
<td>HL7_FIELD_SUBFIELD_ID</td><td>A value which indicates the content should be delivered in the named field/subfield of the HL7 message. When NULL, the data should be sent in an OBX segment with this LOINC code stored in OBX-3 and with the value in the OBX-5</td>
</tr>
<tr>
<td>HLC</td><td>HCPCS Lab Certification Code - code used to classify laboratory procedures according to the specialty certification categories listed by CMS(formerly HCFA).</td>
</tr>
<tr>
<td>HMP</td><td>HCPCS Multiple Pricing Indicator Code - code used to identify instances where a procedure could be priced.</td>
</tr>
<tr>
<td>HMR</td><td>HCPCS Medicare Carriers Manual reference section number - number identifying a section of the Medicare Carriers Manual.</td>
</tr>
<tr>
<td>HM</td><td>MeSH Heading Mapped To - heading mapped to attribute in C-MeSH containing repeating (MH or MH/SH) elements (e.g. HM = PYRROLIDINONES, HM = *TARTRATES, HM = ESTRONE/* analogs &amp; derivatives)</td>
</tr>
<tr>
<td>HN</td><td>History Note - for MeSH history notes, the year when the current form of the MeSH term was established as a major and/or minor descriptor.</td>
</tr>
<tr>
<td>HOMOLOGOUS_GENE</td><td>Homologous Gene</td>
</tr>
<tr>
<td>HPD</td><td>HCPCS ASC payment group effective date - date the procedure is assigned to the ASC payment group.</td>
</tr>
<tr>
<td>HPF</td><td>HCPCS Full Description - complete text of the HCPCS full description, in cases where the HCPCS term in the "STR" field of MRCON has been trimmed from its original form due to length limits.</td>
</tr>
<tr>
<td>HPG</td><td>HCPCS ASC payment group code which represents the dollar amount of the facility charge payable by Medicare for the procedure.</td>
</tr>
<tr>
<td>HPI</td><td>HCPCS Pricing Indicator Code - used to identify the appropriate methodology for developing unique pricing amounts under Part B.</td>
</tr>
<tr>
<td>HPN</td><td>HCPCS processing note number identifying the processing note contained in Appendix A of the HCPCS Manual.</td>
</tr>
<tr>
<td>HPO_COMMENT</td><td>HPO Comment</td>
</tr>
<tr>
<td>HSE</td><td>HL7 Section: Chapter</td>
</tr>
<tr>
<td>HSNU</td><td>HL7 Sort Number: Number for sorting the values according to the official document</td>
</tr>
<tr>
<td>HSN</td><td>HCPCS Statute Number identifying statute reference for coverage or noncoverage of procedure or service.</td>
</tr>
<tr>
<td>HTA</td><td>HL7 Anchor: HTML-Reference to the Standard</td>
</tr>
<tr>
<td>HTD</td><td>HCPCS Termination Date - last date for which a procedure or code may be used by Medicare Providers.</td>
</tr>
<tr>
<td>HTG</td><td>HL7 Generate: Yes/No</td>
</tr>
<tr>
<td>HTID</td><td>HL7 Table ID</td>
</tr>
<tr>
<td>HTS</td><td>HCPCS Type of Service Code - carrier assigned HCFA Type of Service which describes the particular kind(s) of service represented by the procedure code.</td>
</tr>
<tr>
<td>HTT</td><td>HL7 Table Type: Number specifying the type of this table (user defined, HL7 defined, ..)</td>
</tr>
<tr>
<td>HUS</td><td>HL7 Usage: How is this data element used (Original, Added, Deleted)</td>
</tr>
<tr>
<td>HXR</td><td>HCPCS Cross reference code - an explicit reference crosswalking a deleted code or a code that is not valid for Medicare to a valid current code (or range of codes).</td>
</tr>
<tr>
<td>IAA</td><td>ICD10AM Abbreviated Descriptor</td>
</tr>
<tr>
<td>IAC</td><td>ICD10AM Australian Code - Contains a flag "1" for codes that have been exclusively developed in Australia.</td>
</tr>
<tr>
<td>IAD</td><td>ICD10AM Effective from - indicates the date that the code is effective from.</td>
</tr>
<tr>
<td>IAH</td><td>ICD10AM Age Edit, higher limit - minimum age is expressed as a three digit field.</td>
</tr>
<tr>
<td>IAL</td><td>ICD10AM Age Edit, lower limit - minimum age is expressed as a three digit field.</td>
</tr>
<tr>
<td>IAN</td><td>Annotation or Usage Note - a value of "+" or "DAGGER," denotes a code describing the etiology or underlying cause of the disease; an "*" or "ASTERISK," denotes a code describing the manifestation of a disease. A value of "DEFAULT" indicates that dual coding is not indicated.</td>
</tr>
<tr>
<td>IAR</td><td>ICD10AM Relationship Note - code embedded in the term, may have a * or +.</td>
</tr>
<tr>
<td>IAS</td><td>ICD10AM Sex Edit flag to indicate whether the code is valid for a particular sex. Sex flags are 1 (male) or 2 (female).</td>
</tr>
<tr>
<td>IAT</td><td>ICD10AM Sex Edit Type - all sex flagged codes are followed by a "sex edit type" flag. The sex edit type flags are 1 (fatal) or 2 (warning). </td>
</tr>
<tr>
<td>IAY</td><td>ICD10AM Age Edit Type - all age flagged codes are followed by an age edit type flag. The age edit type flags are 1 (fatal) or 2 (warning). </td>
</tr>
<tr>
<td>ICA</td><td>ICD Additional Codes Note - an ICD-9-CM instruction to signal the coder that an additional code should be used if the information is available to provide a more complete picture of that diagnoses or procedure.</td>
</tr>
<tr>
<td>ICC</td><td>ICD Code Also Note - an ICD-9-CM instruction to signal the coder to code additional information.</td>
</tr>
<tr>
<td>ICD-O-3_CODE</td><td>ICD-O-3 Code</td>
</tr>
<tr>
<td>ICE</td><td>ICD Entry Term (e.g. Diarrhea:{dysenteric; epidemic}; Infectious diarrheal disease)</td>
</tr>
<tr>
<td>ICF</td><td>ICD Fifth-digit Code Note - instruction to coders indicating use of a fifth-digit code.</td>
</tr>
<tr>
<td>ICNP_ID</td><td>ICNP ID</td>
</tr>
<tr>
<td>ICN</td><td>ICD Note - instruction providing additional coding information.</td>
</tr>
<tr>
<td>ICPCCODE</td><td>ICPC Code</td>
</tr>
<tr>
<td>IDNR</td><td>Original ID number for source file</td>
</tr>
<tr>
<td>IEEE_CF_CODE10</td><td>IEEE CF Code10</td>
</tr>
<tr>
<td>IEEE_REFID</td><td>IEEE REF ID</td>
</tr>
<tr>
<td>II</td><td>MeSH Indexing Information - for MeSH chemical terms (Term Type=NM), MeSH headings that may be relevant to articles that are also assigned the NM term.</td>
</tr>
<tr>
<td>IMAGING_DOCUMENT_VALUE_SET</td><td>A value of "TRUE" indicates that the LOINC NUM is a member of the Imaging Documents Value Set</td>
</tr>
<tr>
<td>IMPRINT_CODE</td><td>Imprint Code</td>
</tr>
<tr>
<td>INACTIVATION_INDICATOR</td><td>Inactivation indicator</td>
</tr>
<tr>
<td>INC</td><td>ICD10AM Neoplasm code (e.g. C34.- )</td>
</tr>
<tr>
<td>IND_CODE</td><td>IND Code</td>
</tr>
<tr>
<td>INFOODS</td><td>Infoods</td>
</tr>
<tr>
<td>IPX</td><td>ICD10 code related to an ICPC code - a + indicates that the ICD10 code is broader than the ICPC code; a - indicates that the ICD10 code is narrower than the ICPC code. </td>
</tr>
<tr>
<td>ISLEAF</td><td>Leaf flag: 0 for hierarchical nodes, 1 for leaf nodes </td>
</tr>
<tr>
<td>IS_DRUG_CLASS</td><td>Is Drug Class</td>
</tr>
<tr>
<td>IS_RETIRED_FROM_SUBSET</td><td>In future, some concepts will be marked retired if they are retired by IHTSDO or no longer considered to be useful e.g. when there are more appropriate SNOMED CT concepts</td>
</tr>
<tr>
<td>JHU_DTI-81</td><td>JHU DTI-81</td>
</tr>
<tr>
<td>JHU_WHITE_MATTER_TRACTOGRAPHY_ATLAS</td><td>JHU white matter tractography atlas</td>
</tr>
<tr>
<td>JXR</td><td>OMIM code that is related to Jablonski term.</td>
</tr>
<tr>
<td>KEGG_ID</td><td>Kyoto Encyclopedia of Genes and Genomes (KEGG), KEGG Pathway Database, adapted for NCI use ID</td>
</tr>
<tr>
<td>LABELER</td><td>FDA Structured Product Label Manufacturer/Distributor/Supplier name</td>
</tr>
<tr>
<td>LABEL_TYPE</td><td>DailyMed label type</td>
</tr>
<tr>
<td>LANGUAGECODE</td><td>SNOMED CT string identifying a language and, if appropriate, a dialect in which this description is valid.</td>
</tr>
<tr>
<td>LANGUAGE</td><td>Language</td>
</tr>
<tr>
<td>LAST_IN_SUBSET</td><td>The version of Subset last containing this concept.</td>
</tr>
<tr>
<td>LATERALITY</td><td>Laterality</td>
</tr>
<tr>
<td>LCL</td><td>LOINC Class - arbitrary classification of terms in LOINC designed to assist LOINC development and to group related observations together (e.g. ABXBACT = Antibiotic susceptibility)</td>
</tr>
<tr>
<td>LCN</td><td>LOINC Classtype - 1 = Laboratory Class; 2 = Clinical Class; 3 = Claims Attachment; 4 = Surveys</td>
</tr>
<tr>
<td>LCS</td><td>LOINC status - valid values are "ACTIVE," "TRIAL," "DISCOURAGED," "DEPRECATED." For "DEPRECATED" or "DISCOURAGED" concepts, the term that should now be used may appear in the LMP element.</td>
</tr>
<tr>
<td>LCT</td><td>LOINC Change Type Code - type of change made to a LOINC term.</td>
</tr>
<tr>
<td>LEA</td><td>LOINC Example Answers - for some tests and measurements, LOINC has supplied examples of valid answers. These values differ from those in the ANSWERLIST field because that details possible values for nominal scale terms.</td>
</tr>
<tr>
<td>LEGALLY_MARKETED_UNAPPROVED_NEW_ANIMAL_DRUGS_FOR_MINOR_SPECIES</td><td>Legally Marketed Unapproved New Animal Drugs for Minor Species</td>
</tr>
<tr>
<td>LEVEL</td><td>ICF: hierarchical level number. USPMG: value of column from original spreadsheet, indicating whether an atom is a Class, Category or Example Drug.</td>
</tr>
<tr>
<td>LFO</td><td>LOINC Formula - regression equation details for many OB.US calculated terms.</td>
</tr>
<tr>
<td>LMP</td><td>LOINC Map to Code of the term that has superseded a term with a LCS value of DEPRECATED.</td>
</tr>
<tr>
<td>LNCIPEDIA</td><td>Compendium of human long non-coding RNAs</td>
</tr>
<tr>
<td>LOCUS_GROUP</td><td>Locus group</td>
</tr>
<tr>
<td>LOCUS_SPECIFIC_DB_XR</td><td>Locus specific database cross-reference</td>
</tr>
<tr>
<td>LOCUS_TYPE</td><td>Locus Type</td>
</tr>
<tr>
<td>LOE</td><td>Level of effort</td>
</tr>
<tr>
<td>LOINC_COMPONENT</td><td>LOINC component</td>
</tr>
<tr>
<td>LOINC_METHOD_TYP</td><td>LOINC method type</td>
</tr>
<tr>
<td>LOINC_PROPERTY</td><td>LOINC property</td>
</tr>
<tr>
<td>LOINC_SCALE_TYP</td><td>LOINC scale type</td>
</tr>
<tr>
<td>LOINC_SCORE</td><td>A numeric score associated with a LOINC answer ('LA') concept.</td>
</tr>
<tr>
<td>LOINC_SYSTEM</td><td>LOINC system</td>
</tr>
<tr>
<td>LOINC_TIME_ASPECT</td><td>LOINC time aspect</td>
</tr>
<tr>
<td>LOR</td><td>LOINC ORDER_OBS field. Defines term as order only, observation only, or both. Values are: BOTH OBSERVATION ORDER. A fourth category, Subset, is used for terms that are subsets of a panel but do not represent a package that is known to be orderable.</td>
</tr>
<tr>
<td>LQS</td><td>Survey Question Source</td>
</tr>
<tr>
<td>LQT</td><td>Survey Question Text</td>
</tr>
<tr>
<td>LRN2</td><td>Related names 2</td>
</tr>
<tr>
<td>LT</td><td>Indicates if a chemicals or medical device is a tradename (present in older versions of the Metathesaurus and was discontinued, then brought back starting in 2002AD)</td>
</tr>
<tr>
<td>LUR</td><td>Units required when used as OBX segment - a Y/N field that indicates that units are required when this LOINC is included as an OBX segment in a HIPAA attachment</td>
</tr>
<tr>
<td>MACRONUTRIENT</td><td>Macronutrient</td>
</tr>
<tr>
<td>MANE_SELECT</td><td>NCBI and Ensembl transcript IDs/acessions for one high-quality representative transcript per protein-coding gene.</td>
</tr>
<tr>
<td>MANUFACTURER_STATUS</td><td>Manufacturer Status</td>
</tr>
<tr>
<td>MAPADVICE</td><td>Map Advice</td>
</tr>
<tr>
<td>MAPCATEGORYID</td><td>Map Category Id</td>
</tr>
<tr>
<td>MAPGROUP</td><td>Map Group</td>
</tr>
<tr>
<td>MAPPED_UCSC_ID</td><td>UCSC ID (mapped data)</td>
</tr>
<tr>
<td>MAPPRIORITY</td><td>Map Priority</td>
</tr>
<tr>
<td>MAPRULE</td><td>Map Rule</td>
</tr>
<tr>
<td>MAPSETGRAMMAR</td><td>Grammar used to describe boolean expressions used in a map set</td>
</tr>
<tr>
<td>MAPSETNAME</td><td>Official name of a map set</td>
</tr>
<tr>
<td>MAPSETRSAB</td><td>Root source abbreviation for a map set</td>
</tr>
<tr>
<td>MAPSETSID</td><td>Source asserted identifier for a map set</td>
</tr>
<tr>
<td>MAPSETTYPE</td><td>Indicates the nature of a map set. Its value is map set specific. It can be used to indicate the inclusion of one to one, one to many and choices of maps.</td>
</tr>
<tr>
<td>MAPSETVERSION</td><td>Version of the map set</td>
</tr>
<tr>
<td>MAPSETVSAB</td><td>Versioned source abbreviation for a map set</td>
</tr>
<tr>
<td>MAPTARGET</td><td>Map Target</td>
</tr>
<tr>
<td>MARKETING_CATEGORY</td><td>Marketing category</td>
</tr>
<tr>
<td>MARKETING_EFFECTIVE_TIME_HIGH</td><td>The expiration date of the last lot distributed provided by the Structured Product Labeling (SPL) author. Products that are actively being marketed will not have a marketing end date</td>
</tr>
<tr>
<td>MARKETING_EFFECTIVE_TIME_LOW</td><td>The date the Structured Product Labeling (SPL) author indicates when it started marketing the packaged product</td>
</tr>
<tr>
<td>MARKETING_STATUS</td><td>The Marketing Status of the MTHSPL drug</td>
</tr>
<tr>
<td>MDA</td><td>MeSH date of entry YYYYMMDD - date the term was added to the MeSH file, which is prior to the date the term became available for indexing and searching MEDLARS citations. Terms that have been part of MeSH for many years may have no value in this element.</td>
</tr>
<tr>
<td>MEASURE_DEVELOPER</td><td>Name of the measure developer associated with the Category II code</td>
</tr>
<tr>
<td>MEA</td><td>In NOC, the "measurement scale" used for a particular outcome.</td>
</tr>
<tr>
<td>MENU_PARENT</td><td>Link information to Term Menu parent. This may not always be the same as the TermParent. Element will not exist for top level menus, e.g. cancer.</td>
</tr>
<tr>
<td>MENU_TYPE</td><td>Values are Clinical Trials--Cancer Type, Clinical Trials--Drug, or Cancer Information.</td>
</tr>
<tr>
<td>MGD_ID</td><td>MGD ID</td>
</tr>
<tr>
<td>MGI_ACCESSION_ID</td><td>MGI Accession ID</td>
</tr>
<tr>
<td>MICRONUTRIENT</td><td>Micronutrient</td>
</tr>
<tr>
<td>MIMTYPEMEANING</td><td>OMIM MimType Meaning</td>
</tr>
<tr>
<td>MIMTYPEVALUE</td><td>OMIM MimType Value</td>
</tr>
<tr>
<td>MIMTYPE</td><td>OMIM Entry Type</td>
</tr>
<tr>
<td>MIRBASE_ID</td><td>miRBase ID</td>
</tr>
<tr>
<td>MISO</td><td>MedDRA Serial Code International SOC Sort Order Digit (01-26)</td>
</tr>
<tr>
<td>MMR</td><td>MeSH revision date YYYYMMDD - date of the last major revision to the term's MeSH record.</td>
</tr>
<tr>
<td>MMX_RXO</td><td>Micromedex Product Category Code</td>
</tr>
<tr>
<td>MN</td><td>MeSH hierarchical number for the concept in the MeSH tree structures. This number also appears in the HCD subelement of the REL and CXT elements.</td>
</tr>
<tr>
<td>MODIFIER_ID</td><td>Modifier ID</td>
</tr>
<tr>
<td>MODIFIER</td><td>Modifier</td>
</tr>
<tr>
<td>MODULE_ID</td><td>Module ID</td>
</tr>
<tr>
<td>MOLECULAR_WEIGHT_OF_ANALYTE</td><td>Molecular weight of analyte</td>
</tr>
<tr>
<td>MONOGRAPH_OFFICIAL_DATE</td><td>Monograph Official Date</td>
</tr>
<tr>
<td>MONOGRAPH_STATUS</td><td>Monograph Status</td>
</tr>
<tr>
<td>MOVED_FROM</td><td>Moved from</td>
</tr>
<tr>
<td>MPS</td><td>MedDRA primary SOC (PTs may have multiple treepositions, but each has a primary soc)</td>
</tr>
<tr>
<td>MP_GROUP_URL</td><td>MEDLINEPLUS group URL</td>
</tr>
<tr>
<td>MP_OTHER_LANGUAGE_URL</td><td>MEDLINEPLUS health topic other-language URL</td>
</tr>
<tr>
<td>MP_PRIMARY_INSTITUTE_URL</td><td>MEDLINEPLUS health topic primary-institute URL</td>
</tr>
<tr>
<td>MR</td><td>Major revision date YYYYMMDD - date the Metathesaurus entry for the concept underwent any revision in content. </td>
</tr>
<tr>
<td>MSC</td><td>Minimal Standard (Terminology) Class</td>
</tr>
<tr>
<td>MSP</td><td>SPN Medical Specialty Panel (responsible for reviewing the product). </td>
</tr>
<tr>
<td>MTH_MAPFROMCOMPLEXITY</td><td>Indicates the complexity of "from" expressions used in a map set</td>
</tr>
<tr>
<td>MTH_MAPFROMEXHAUSTIVE</td><td>Indicates whether or not the "from" source of a map set is completely mapped</td>
</tr>
<tr>
<td>MTH_MAPSETCOMPLEXITY</td><td>Indicates the overall complexity of a map set</td>
</tr>
<tr>
<td>MTH_MAPTOCOMPLEXITY</td><td>Indicates the complexity of "to" expressions in a map set</td>
</tr>
<tr>
<td>MTH_MAPTOEXHAUSTIVE</td><td>Indicates whether or not the "to" source is completely mapped</td>
</tr>
<tr>
<td>NADA</td><td>New Animal Drug Application number for MTHSPL drug</td>
</tr>
<tr>
<td>NCBI_TAXON_ID</td><td>NCBI_Taxon_ID</td>
</tr>
<tr>
<td>NCI_DRUG_DICTIONARY_ID</td><td>NCI Drug Dictionary</td>
</tr>
<tr>
<td>NCI_ID</td><td>NCI thesaurus concept id</td>
</tr>
<tr>
<td>NDA_AUTHORIZED_GENERIC</td><td>New Drug Application number for authorized generic MTHSPL drug</td>
</tr>
<tr>
<td>NDA</td><td>New Drug Application number for MTHSPL drug</td>
</tr>
<tr>
<td>NDC</td><td>National Drug Code corresponding to a clinical drug (e.g. 000023082503)</td>
</tr>
<tr>
<td>NDF_TRANSMIT_TO_CMOP</td><td>NDF Transmit to Consolidated Mail Outpatient Pharmacy (CMOP)</td>
</tr>
<tr>
<td>NEOPLASTIC_STATUS</td><td>Neoplastic Status</td>
</tr>
<tr>
<td>NEUROLEX_ID</td><td>Neurolex ID</td>
</tr>
<tr>
<td>NFI</td><td>National formulary indicator - "YES" or "NO" indicating whether a drug is in the VA's National Formulary</td>
</tr>
<tr>
<td>NF_INACTIVATE</td><td>National Formulary Inactivation Date - the date a drug was removed from the VA's National Formulary</td>
</tr>
<tr>
<td>NF_NAME</td><td>National Formulary Name</td>
</tr>
<tr>
<td>NHRIC</td><td>National Health Related Items Code</td>
</tr>
<tr>
<td>NON_VACCINE</td><td>Non Vaccine</td>
</tr>
<tr>
<td>NOTE</td><td>Note</td>
</tr>
<tr>
<td>NSC_CODE</td><td>NSC Code</td>
</tr>
<tr>
<td>NSC_NUMBER</td><td>NSC Number</td>
</tr>
<tr>
<td>NSR</td><td>Neuronames Species Restriction - indication that a Neuronames concept applies to humans, macaques, primates, rats, etc. Most Neuronames concepts have no species restriction.</td>
</tr>
<tr>
<td>NUTRIENT</td><td>Nutrient</td>
</tr>
<tr>
<td>OCCURRENCE</td><td>Number of institutions having this concept on their problem list (from 1 to 7), not populated for concepts retired from Subset</td>
</tr>
<tr>
<td>OL</td><td>MeSH Online Note - information helpful to online searchers of MEDLINE, especially when the history of a term or cross-reference has implications for online searching. This is a potential source of useful information for rules for search interface programs.</td>
</tr>
<tr>
<td>OMIM_ID</td><td>OMIM ID</td>
</tr>
<tr>
<td>OMIM_NUMBER</td><td>OMIM Number</td>
</tr>
<tr>
<td>ORDER_NO</td><td>Order number</td>
</tr>
<tr>
<td>ORIG_CODE</td><td>Original code associated with this string</td>
</tr>
<tr>
<td>ORIG_SOURCE</td><td>Original source associated with this string</td>
</tr>
<tr>
<td>ORIG_STY</td><td>Original semantic type</td>
</tr>
<tr>
<td>OR_GROUP</td><td>Indicates that relationships with the same CUI2 and RG are disjunctive expressions</td>
</tr>
<tr>
<td>OTC_MONOGRAPH_DRUG</td><td>OTC Monograph Drug</td>
</tr>
<tr>
<td>OTC_MONOGRAPH_FINAL</td><td>FDA Structured Product Label OTC monograph status</td>
</tr>
<tr>
<td>OTC_MONOGRAPH_NOT_FINAL</td><td>FDA Structured Product Label OTC monograph status</td>
</tr>
<tr>
<td>OUTDATED_MEANING</td><td>Outdated Meaning</td>
</tr>
<tr>
<td>PANEL_TYPE</td><td>Panel type</td>
</tr>
<tr>
<td>PARENT_CLASS</td><td>VA Internal entry number of the parent class</td>
</tr>
<tr>
<td>PARTITION</td><td>Partition</td>
</tr>
<tr>
<td>PART_SEQ_ORDER</td><td>Part sequence order</td>
</tr>
<tr>
<td>PART_TYPE</td><td>Part type</td>
</tr>
<tr>
<td>PA</td><td>Pharmacologic Action of MeSH main headings (MH) for drugs and supplementary concept names (NM). The information in this element is also represented by an "isa" relationship between the MH or NM concept and the MeSH concept name for the class of drugs with a particular pharmacologic action.</td>
</tr>
<tr>
<td>PCL</td><td>Pharmacy Practice Activity Classification (PPAC) Category - all terms are assigned to one of five categories, which connote their hierarchy.</td>
</tr>
<tr>
<td>PDC</td><td>SPN Product Device Class (level of CDRH regulation: class 1, 2, or 3). </td>
</tr>
<tr>
<td>PDQ_CLOSED_TRIAL_SEARCH_ID</td><td>PDQ Closed Trial Search ID</td>
</tr>
<tr>
<td>PDQ_OPEN_TRIAL_SEARCH_ID</td><td>PDQ Open Trial Search ID</td>
</tr>
<tr>
<td>PHYSICAL_STATE</td><td>Physical state</td>
</tr>
<tr>
<td>PID_ID</td><td>National Cancer Institute Nature Pathway Interaction Database ID</td>
</tr>
<tr>
<td>PID</td><td>Legacy PDQ ID</td>
</tr>
<tr>
<td>PIRADSV2</td><td>Piradsv 2</td>
</tr>
<tr>
<td>PI</td><td>MeSH heading or heading/subheading combination(s) followed by a date range in parentheses (YYYY).</td>
</tr>
<tr>
<td>PLACE_OF_SERVICE_CODE</td><td>Place of service code</td>
</tr>
<tr>
<td>PLR</td><td>Pharmacy Practice Activity Classification (PPAC) Last Revision Date (Format: M/DD/YY time)</td>
</tr>
<tr>
<td>PMID</td><td>Pubmed ID</td>
</tr>
<tr>
<td>PM</td><td>Public MeSH note - combines key information from the HN and PI elements in a format that is printed in the MeSH publications.</td>
</tr>
<tr>
<td>POLARITY</td><td>Polarity</td>
</tr>
<tr>
<td>PRC</td><td>Product Third Party Review Code from SPN. </td>
</tr>
<tr>
<td>PREF_FLAG_ID</td><td>Preferred flag</td>
</tr>
<tr>
<td>PREMARKET_APPLICATION</td><td>Premarket Application</td>
</tr>
<tr>
<td>PREMARKET_NOTIFICATION</td><td>FDA Structured Product Label premarket notification</td>
</tr>
<tr>
<td>PRESENTATION_STRENGTH_DENOMINATOR_VALUE</td><td>Presentation strength denominator value (defining)</td>
</tr>
<tr>
<td>PRESENTATION_STRENGTH_NUMERATOR_VALUE</td><td>Presentation strength numerator value (defining)</td>
</tr>
<tr>
<td>PROPRIETARY_NAME</td><td>Proprietary name</td>
</tr>
<tr>
<td>PTR</td><td>SPN Product Tier (level of CDRH triage: 1, 2, 3, or E{xempt}). </td>
</tr>
<tr>
<td>PUBLISH_VALUE_SET</td><td>publish value set</td>
</tr>
<tr>
<td>PUBMEDID_PRIMARY_REFERENCE</td><td>PubMedID Primary Reference</td>
</tr>
<tr>
<td>PXC</td><td>PDQ Protocol Exclusion Criteria - terms with type "exclusion criteria," which may be indexed on protocol records to identify conditions that exclude a patient from eligibility. </td>
</tr>
<tr>
<td>PYR</td><td>PsychInfo year designation</td>
</tr>
<tr>
<td>QUALIFIER</td><td>has qualifer</td>
</tr>
<tr>
<td>RADLEX_ID</td><td>RadLex ID</td>
</tr>
<tr>
<td>RADLEX_PLAYBOOK_LONG_NAME</td><td>RadLex Playbook long name</td>
</tr>
<tr>
<td>RADLEX_PREF_NAME</td><td>RadLex preferred name</td>
</tr>
<tr>
<td>RANK</td><td>NCBI Rank (e.g. RANK[NCBI]species)</td>
</tr>
<tr>
<td>REFSEQ_ID</td><td>RefSeq ID</td>
</tr>
<tr>
<td>REFSET_PATTERN</td><td>Refset pattern</td>
</tr>
<tr>
<td>REF</td><td>References or citations related to a given atom or concept.</td>
</tr>
<tr>
<td>REMARKS</td><td>Specific coding instructions</td>
</tr>
<tr>
<td>REPLACED_BY_SNOMED_CID</td><td>Current version SNOMEDCT concept id this concept is replaced by</td>
</tr>
<tr>
<td>REPORTABLE</td><td>Indicates whether a code is reportable</td>
</tr>
<tr>
<td>RGD_ID</td><td>Rat genome database gene ID</td>
</tr>
<tr>
<td>RID</td><td>Read Codes Term_id - identifier assigned to a Read term, used in referring to the term in the Read file structure, and may be used in clinical information systems.</td>
</tr>
<tr>
<td>RN</td><td>Registry Number - series of numbers and hyphens (any leading zeros in an RN are dropped) or a series of numbers and periods, preceded by EC</td>
</tr>
<tr>
<td>RPID</td><td>RadLex Playbook ID</td>
</tr>
<tr>
<td>RR</td><td>The Chemical Abstracts Registry numbers for salts, optical isomers, or isotope-labeled versions of the concept followed by the relationship of this RR to the RN (in parentheses.) Applies to chemicals only. These numbers can be used as links to information in a number of chemical and toxicological databases.</td>
</tr>
<tr>
<td>RVU_FACILITY_PRACTICE_EXPENSE_DISCONTINUED</td><td>RVU facility practice expense discontinued</td>
</tr>
<tr>
<td>RVU_FACILITY_PRACTICE_EXPENSE_PROFESSIONAL</td><td>RVU facility practice expense professional</td>
</tr>
<tr>
<td>RVU_FACILITY_PRACTICE_EXPENSE_TECHNICAL</td><td>RVU facility practice expense technical</td>
</tr>
<tr>
<td>RVU_FACILITY_PRACTICE_EXPENSE</td><td>RVU facility practice expense</td>
</tr>
<tr>
<td>RVU_MEDICARE_GLOBAL_PERIOD_DISCONTINUED</td><td>RVU Medicare global period discontinued</td>
</tr>
<tr>
<td>RVU_MEDICARE_GLOBAL_PERIOD_PROFESSIONAL</td><td>RVU Medicare global period professional</td>
</tr>
<tr>
<td>RVU_MEDICARE_GLOBAL_PERIOD_TECHNICAL</td><td>RVU Medicare global period technical</td>
</tr>
<tr>
<td>RVU_MEDICARE_GLOBAL_PERIOD</td><td>RVU Medicare global period</td>
</tr>
<tr>
<td>RVU_NONFACILITY_PRACTICE_EXPENSE_DISCONTINUED</td><td>RVU nonfacility practice expense discontinued</td>
</tr>
<tr>
<td>RVU_NONFACILITY_PRACTICE_EXPENSE_PROFESSIONAL</td><td>RVU nonfacility practice expense professional</td>
</tr>
<tr>
<td>RVU_NONFACILITY_PRACTICE_EXPENSE_TECHNICAL</td><td>RVU nonfacility practice expense technical</td>
</tr>
<tr>
<td>RVU_NONFACILITY_PRACTICE_EXPENSE</td><td>RVU nonfacility practice expense</td>
</tr>
<tr>
<td>RVU_PLI_DISCONTINUED</td><td>RVU PLI discontinued</td>
</tr>
<tr>
<td>RVU_PLI_PROFESSIONAL</td><td>RVU PLI professional</td>
</tr>
<tr>
<td>RVU_PLI_TECHNICAL</td><td>RVU PLI technical</td>
</tr>
<tr>
<td>RVU_PLI</td><td>RVU PLI</td>
</tr>
<tr>
<td>RVU_TOTAL_FACILITY_DISCONTINUED</td><td>RVU total facility discontinued</td>
</tr>
<tr>
<td>RVU_TOTAL_FACILITY_PROFESSIONAL</td><td>RVU total facility professional</td>
</tr>
<tr>
<td>RVU_TOTAL_FACILITY_TECHNICAL</td><td>RVU total facility technical</td>
</tr>
<tr>
<td>RVU_TOTAL_FACILITY</td><td>RVU total facility</td>
</tr>
<tr>
<td>RVU_TOTAL_NONFACILITY_DISCONTINUED</td><td>RVU total nonfacility discontinued</td>
</tr>
<tr>
<td>RVU_TOTAL_NONFACILITY_PROFESSIONAL</td><td>RVU total nonfacility professional</td>
</tr>
<tr>
<td>RVU_TOTAL_NONFACILITY_TECHNICAL</td><td>RVU total nonfacility technical</td>
</tr>
<tr>
<td>RVU_TOTAL_NONFACILITY</td><td>RVU total nonfacility</td>
</tr>
<tr>
<td>RVU_WORK_DISCONTINUED</td><td>RVU work discontinued</td>
</tr>
<tr>
<td>RVU_WORK_PROFESSIONAL</td><td>RVU work professional</td>
</tr>
<tr>
<td>RVU_WORK_TECHNICAL</td><td>RVU work technical</td>
</tr>
<tr>
<td>RVU_WORK</td><td>RVU work</td>
</tr>
<tr>
<td>RXAUI</td><td>RxNorm atom identifier for the atom it is connected to</td>
</tr>
<tr>
<td>RXCUI</td><td>RxNorm concept identifier for the atom this is connected to</td>
</tr>
<tr>
<td>RXN_ACTIVATED</td><td>Date the RxNorm atom was reactivated</td>
</tr>
<tr>
<td>RXN_AI</td><td>RXCUI of BOSS Active Ingredient preceded by RXCUI of SCDC responsible for value</td>
</tr>
<tr>
<td>RXN_AM</td><td>RXCUI of BOSS Active Moiety preceded by RXCUI of SCDC responsible for value</td>
</tr>
<tr>
<td>RXN_AVAILABLE_STRENGTH</td><td>Available drug strengths listed in the order of ingredients from the drug</td>
</tr>
<tr>
<td>RXN_BN_CARDINALITY</td><td>Cardinality of RxNorm Brand Name Atom</td>
</tr>
<tr>
<td>RXN_BOSS_FROM</td><td>RXN Boss From</td>
</tr>
<tr>
<td>RXN_BOSS_STRENGTH_DENOM_UNIT</td><td>RXN Boss Strength Denom Unit</td>
</tr>
<tr>
<td>RXN_BOSS_STRENGTH_DENOM_VALUE</td><td>RXN Boss Strength Denom Value</td>
</tr>
<tr>
<td>RXN_BOSS_STRENGTH_NUM_UNIT</td><td>RXN Boss Strength Num Unit</td>
</tr>
<tr>
<td>RXN_BOSS_STRENGTH_NUM_VALUE</td><td>RXN Boss Strength Num Value</td>
</tr>
<tr>
<td>RXN_HUMAN_DRUG</td><td>Drug available for use in Humans</td>
</tr>
<tr>
<td>RXN_IN_EXPRESSED_FLAG</td><td>Strength Expressed As Precise Flag</td>
</tr>
<tr>
<td>RXN_OBSOLETED</td><td>Date the RxNorm atom became obsolete</td>
</tr>
<tr>
<td>RXN_QUALITATIVE_DISTINCTION</td><td>RXN Qualitative Distinction</td>
</tr>
<tr>
<td>RXN_QUANTITY</td><td>Normal Form quantity factor</td>
</tr>
<tr>
<td>RXN_STRENGTH</td><td>Strength plus unit of SCDC</td>
</tr>
<tr>
<td>RXN_VET_DRUG</td><td>Drug available for use in animals</td>
</tr>
<tr>
<td>RXR</td><td>Replacement: cross reference from obsolete code to current code</td>
</tr>
<tr>
<td>RXTERM_FORM</td><td>The RxTerm dose form name for this drug</td>
</tr>
<tr>
<td>SB</td><td>SNOMED International subset indicator - valid values: * =can code using two T codes or G code for laterality B Bethesda system (Morphology); IC= ICDO (Oncology) related; N=Nursing; N* =Nursing, provisional; U=Ultrastructure (Morphology); V= Veterinary; V* =Veterinary AND can code using two T codes or G code for laterality (Topography) </td>
</tr>
<tr>
<td>SCORE</td><td>FDA Structured Product Label imprint attribute for score</td>
</tr>
<tr>
<td>SC</td><td>MeSH Supplemental record class.</td>
</tr>
<tr>
<td>SECONDLEVEL</td><td>Second Level code</td>
</tr>
<tr>
<td>SEQ_NO</td><td>Sequence number</td>
</tr>
<tr>
<td>SHAPETEXT</td><td>FDA Structured Product Label imprint attribute for shape text</td>
</tr>
<tr>
<td>SHAPE</td><td>FDA Structured Product Label imprint attribute for shape</td>
</tr>
<tr>
<td>SHF</td><td>SNOMED Hospital Formulary Code - the American Hospital Formulary Code for a chemical contained in SNOMED International (e.g. 84:24:12) </td>
</tr>
<tr>
<td>SIC</td><td>SNOMED ICD9CM Reference - the ICD9CM code or codes listed as relevant to the meaning of the concept in SNOMED International.</td>
</tr>
<tr>
<td>SID</td><td>Secondary GO ID (e.g. GO:0020034)</td>
</tr>
<tr>
<td>SIZE</td><td>FDA Structured Product Label imprint attribute for size</td>
</tr>
<tr>
<td>SLOT_SYNONYM</td><td>slot synonym</td>
</tr>
<tr>
<td>SMQ_ALGO</td><td>Standardised MedDRA Query (SMQ) Algorithm. Boolean expression of algorithm for the SMQ. "N" if the SMQ does not utilize an algorithm.</td>
</tr>
<tr>
<td>SMQ_LEVEL</td><td>Standardised MedDRA Query (SMQ) Level. Value between 1 and 5 identifying the level of the SMQ within the hierarchy of SMQs; 1 is the most general, 5 is the most narrow.</td>
</tr>
<tr>
<td>SMQ_SOURCE</td><td>Standardised MedDRA Query (SMQ) Source. Source for the development of the SMQ, e.g. medical references.</td>
</tr>
<tr>
<td>SMQ_STATUS</td><td>Standardised MedDRA Query (SMQ) Status. "A" = active; "I" = inactive.</td>
</tr>
<tr>
<td>SMQ_TERM_ADDVERSION</td><td>Standardised MedDRA Query (SMQ) Addition Version. Version of MedDRA in which term was added to the SMQ.</td>
</tr>
<tr>
<td>SMQ_TERM_CAT</td><td>Standardised MedDRA Query (SMQ) Term Category. A single alphabetical letter indicating the category of the term for application of the SMQ algorithm. If the SMQ does not use algorithms, then all Term_category values are assigned "A." For a child SMQ, this field is assigned "S."</td>
</tr>
<tr>
<td>SMQ_TERM_LEVEL</td><td>Standardised MedDRA Query (SMQ) Term Level. The MedDRA hierarchy level of a term (4=PT, 5=LLT) or 0 (zero) for a child SMQ.</td>
</tr>
<tr>
<td>SMQ_TERM_LMVERSION</td><td>Standardised MedDRA Query (SMQ) Last Modified Version. Version of MedDRA in which term was last modified in the SMQ.</td>
</tr>
<tr>
<td>SMQ_TERM_SCOPE</td><td>Standardised MedDRA Query (SMQ) Term Scope. Defines the MedDRA term as a member of the broad scope (1), narrow scope (2) of the SMQ search, or a child SMQ (0) (zero).</td>
</tr>
<tr>
<td>SMQ_TERM_STATUS</td><td>Standardised MedDRA Query (SMQ) Term Status. Identifies a term as active ("A") or inactive ("I") within this SMQ.</td>
</tr>
<tr>
<td>SMQ_TERM_WEIGHT</td><td>Standardised MedDRA Query (SMQ) Term Weight. Used for some SMQ algorithms; "0" is used as default.</td>
</tr>
<tr>
<td>SMX</td><td>SNOMED Multiaxial coding - an alphanumeric string that includes hyphens, parentheses, and sometimes ellipses. </td>
</tr>
<tr>
<td>SNGL_OR_MULT_SRC_PRD</td><td>Single or multi-source product</td>
</tr>
<tr>
<td>SORT</td><td>Sort field</td>
</tr>
<tr>
<td>SOS</td><td>Scope Statement</td>
</tr>
<tr>
<td>SOURCEEFFECTIVETIME</td><td>Source Effective Time</td>
</tr>
<tr>
<td>SOURCE_EFFECTIVE_TIME</td><td>Source effective time</td>
</tr>
<tr>
<td>SOURCE_UI</td><td>Unique identifier from the source</td>
</tr>
<tr>
<td>SPECIALTY</td><td>Specialty</td>
</tr>
<tr>
<td>SPECIES</td><td>Species</td>
</tr>
<tr>
<td>SPL_SET_ID</td><td>FDA Structured Product Label SET_ID code</td>
</tr>
<tr>
<td>SRC</td><td>MeSH Literature source of chemical name - a citation to an article in a journal indexed for MEDLINE in which the chemical has been identified. (Note: Not to be confused with source abbreviation of SRC)</td>
</tr>
<tr>
<td>STATE_OF_DETERMINATION</td><td>State of determination</td>
</tr>
<tr>
<td>STATUS_REASON</td><td>Classification of the reason for concept status in LOINC</td>
</tr>
<tr>
<td>STATUS_TEXT</td><td>Explanation of concept status in narrative text in LOINC</td>
</tr>
<tr>
<td>STATUS</td><td>Indicates status of "active" (current) or "inactive" (obsolete/deprecated)</td>
</tr>
<tr>
<td>ST</td><td>Concept Attributes Status - valid values: R Reviewed, U Unreviewed </td>
</tr>
<tr>
<td>SUBCATEGORY</td><td>Subcategory</td>
</tr>
<tr>
<td>SUBSEQUENT_TEXT_PROMPT</td><td>Subsequent text prompt</td>
</tr>
<tr>
<td>SUBSET_MEMBER</td><td>Subset member</td>
</tr>
<tr>
<td>SWISS_PROT</td><td>Swiss Prot</td>
</tr>
<tr>
<td>SWP</td><td>Swiss Protein Number</td>
</tr>
<tr>
<td>SYMBOL</td><td>FDA Structured Product Label imprint attribute for symbol</td>
</tr>
<tr>
<td>SYN_QUALIFIER</td><td>Synonym qualifier</td>
</tr>
<tr>
<td>TALAIRACH_TERM</td><td>Talairach Term</td>
</tr>
<tr>
<td>TARGETCOMPONENT</td><td>Target Component</td>
</tr>
<tr>
<td>TARGETEFFECTIVETIME</td><td>Target Effective Time</td>
</tr>
<tr>
<td>TARGET_EFFECTIVE_TIME</td><td>Target effective time</td>
</tr>
<tr>
<td>TA_ID</td><td>TAID</td>
</tr>
<tr>
<td>TELEMEDICINE</td><td>Telemedicine</td>
</tr>
<tr>
<td>TERMCODE</td><td>Term Code</td>
</tr>
<tr>
<td>TERMID</td><td>Term identifier</td>
</tr>
<tr>
<td>TERMSTATUS</td><td>Term status</td>
</tr>
<tr>
<td>TERMUI</td><td>Term unique identifier</td>
</tr>
<tr>
<td>TERM_BROWSER_VALUE_SET_DESCRIPTION</td><td>Term browser value set description</td>
</tr>
<tr>
<td>TERM_CODE</td><td>Term Code</td>
</tr>
<tr>
<td>TERM_STATUS</td><td>Term Status</td>
</tr>
<tr>
<td>THIRDLEVEL</td><td>Third Level code</td>
</tr>
<tr>
<td>TH</td><td>MeSH Thesaurus ID - identifies thesauri other than MeSH in which the MeSH heading or cross-reference is included.</td>
</tr>
<tr>
<td>TOLERABLE_LEVEL</td><td>Tolerable Level</td>
</tr>
<tr>
<td>TORSAB</td><td>Root source abbreviation for the "to" identifiers of a map set</td>
</tr>
<tr>
<td>TOVSAB</td><td>Versioned source abbreviation for the "to" identifiers of a map set</td>
</tr>
<tr>
<td>TRANSLATION</td><td>Translated name of the neuroanatomical structure</td>
</tr>
<tr>
<td>TYPE_ID</td><td>Type ID</td>
</tr>
<tr>
<td>TYPE</td><td>Multum Medical Supply Category (e.g. natural supplements)</td>
</tr>
<tr>
<td>UMLSRELA</td><td>UMLS relationship attribute</td>
</tr>
<tr>
<td>UMLSREL</td><td>The UMLS Metathesaurus REL relationship (SY, CHD, RN, RB, RO) assigned to SNOMED CT relationship identifiers.</td>
</tr>
<tr>
<td>UNAPPROVED_DRUG_OTHER</td><td>Marketing category unapproved_drug_other for MTHSPL atom</td>
</tr>
<tr>
<td>UNAPPROVED_MEDICAL_GAS</td><td>Unapproved Medical Gas</td>
</tr>
<tr>
<td>UNIT</td><td>Unit</td>
</tr>
<tr>
<td>UNIVERSAL_LAB_ORDERS_VALUE_SET</td><td>A value of "TRUE" indicates that the LOINC NUM is a member of the LOINC Universal Lab Orders Value Set</td>
</tr>
<tr>
<td>UPC</td><td>Universal Product Code</td>
</tr>
<tr>
<td>USAGE_NOTES</td><td>Usage notes</td>
</tr>
<tr>
<td>USAGE</td><td>The average usage percentage among all institutions (i.e. sum of individual usage percentages divided by 7), not populated for concepts retired from Subset</td>
</tr>
<tr>
<td>USDA_ID</td><td>USDA ID</td>
</tr>
<tr>
<td>USE_ADDITIONAL</td><td>Certain conditions have both an underlying etiology and multiple body system manifestations due to the underlying etiology. ICD-10-CM has a coding convention that requires the underlying condition be sequenced first followed by the manifestation. Wherever such a combination exists there is a "use additional code" note at the etiology code. These instructional notes indicate the proper sequencing order of the codes, etiology followed by manifestation. In most cases the manifestation codes will have in the code title, "in diseases classified elsewhere." Codes with this title are a component of the etiology/ manifestation convention. The code title indicates that it is a manifestation code. "In diseases classified elsewhere" codes are never permitted to be used as first listed or principle diagnosis codes. They must be used in conjunction with an underlying condition code and they must be listed following the underlying condition.</td>
</tr>
<tr>
<td>USE_FOR</td><td>Use for</td>
</tr>
<tr>
<td>US_RECOMMENDED_INTAKE</td><td>US Recommended Intake</td>
</tr>
<tr>
<td>UWT</td><td>A semantic type provided from terms from the University of Washington Digital Anatomist </td>
</tr>
<tr>
<td>VACCINE_STATUS</td><td>Vaccine status. CVX codes for inactive vaccines allow transmission of historical immunization records.</td>
</tr>
<tr>
<td>VAC</td><td>VA Class - the code of an NDF/HT drug class name (e.g. AM110)</td>
</tr>
<tr>
<td>VALUEID</td><td>Value Id</td>
</tr>
<tr>
<td>VALUE_SET_PAIR</td><td>Value Set Pair</td>
</tr>
<tr>
<td>VA_CLASS_NAME</td><td>VA class name</td>
</tr>
<tr>
<td>VA_DISPENSE_UNIT</td><td>VA dispense unit</td>
</tr>
<tr>
<td>VA_GENERIC_NAME</td><td>VA generic name</td>
</tr>
<tr>
<td>VEGA_ID</td><td>VEGA ID</td>
</tr>
<tr>
<td>VERSION_FIRST_RELEASED</td><td>Version first released</td>
</tr>
<tr>
<td>VERSION_LAST_CHANGED</td><td>Version last changed</td>
</tr>
<tr>
<td>VIEW</td><td>View</td>
</tr>
<tr>
<td>VMO</td><td>VA CMOP (central mail - order pharmacy) ID</td>
</tr>
<tr>
<td>FULL-MULTIPLE</td><td>Full contexts, multiple tree positions</td>
</tr>
<tr>
<td>FULL</td><td>Full contexts</td>
</tr>
<tr>
<td>CODE</td><td>Unique Identifier or code for string in source</td>
</tr>
<tr>
<td>CUI</td><td>Concept unique identifier</td>
</tr>
<tr>
<td>SCUI</td><td>Source asserted concept unique identifier</td>
</tr>
<tr>
<td>SDUI</td><td>Source asserted descriptor identifier</td>
</tr>
<tr>
<td>ARA</td><td>Arabic</td>
</tr>
<tr>
<td>BAQ</td><td>Basque</td>
</tr>
<tr>
<td>CHI</td><td>Chinese</td>
</tr>
<tr>
<td>CZE</td><td>Czech</td>
</tr>
<tr>
<td>DAN</td><td>Danish</td>
</tr>
<tr>
<td>DUT</td><td>Dutch</td>
</tr>
<tr>
<td>ENG</td><td>English</td>
</tr>
<tr>
<td>EST</td><td>Estonian</td>
</tr>
<tr>
<td>FIN</td><td>Finnish</td>
</tr>
<tr>
<td>FRE</td><td>French</td>
</tr>
<tr>
<td>GER</td><td>German</td>
</tr>
<tr>
<td>GRE</td><td>Greek</td>
</tr>
<tr>
<td>HEB</td><td>Hebrew</td>
</tr>
<tr>
<td>HUN</td><td>Hungarian</td>
</tr>
<tr>
<td>ITA</td><td>Italian</td>
</tr>
<tr>
<td>JPN</td><td>Japanese</td>
</tr>
<tr>
<td>KOR</td><td>Korean</td>
</tr>
<tr>
<td>LAV</td><td>Latvian</td>
</tr>
<tr>
<td>NOR</td><td>Norwegian</td>
</tr>
<tr>
<td>POL</td><td>Polish</td>
</tr>
<tr>
<td>POR</td><td>Portuguese</td>
</tr>
<tr>
<td>RUS</td><td>Russian</td>
</tr>
<tr>
<td>SCR</td><td>Croatian</td>
</tr>
<tr>
<td>SPA</td><td>Spanish</td>
</tr>
<tr>
<td>SWE</td><td>Swedish</td>
</tr>
<tr>
<td>TUR</td><td>Turkish</td>
</tr>
<tr>
<td>UKR</td><td>Ukrainian</td>
</tr>
<tr>
<td>ACTIVE</td><td>Active</td>
</tr>
<tr>
<td></td><td>Empty attribute name</td>
</tr>
<tr>
<td>AQ</td><td>Allowed qualifier</td>
</tr>
<tr>
<td>CHD</td><td>has child relationship in a Metathesaurus source vocabulary </td>
</tr>
<tr>
<td>DEL</td><td>Deleted concept</td>
</tr>
<tr>
<td>PAR</td><td>has parent relationship in a Metathesaurus source vocabulary </td>
</tr>
<tr>
<td>QB</td><td>can be qualified by.</td>
</tr>
<tr>
<td>RB</td><td>has a broader relationship</td>
</tr>
<tr>
<td>RL</td><td>the relationship is similar or "alike". the two concepts are similar or "alike". In the current edition of the Metathesaurus, most relationships with this attribute are mappings provided by a source, named in SAB and SL; hence concepts linked by this relationship may be synonymous, i.e. self-referential: CUI1 = CUI2. In previous releases, some MeSH Supplementary Concept relationships were represented in this way.</td>
</tr>
<tr>
<td>RN</td><td>has a narrower relationship</td>
</tr>
<tr>
<td>RO</td><td>has relationship other than synonymous, narrower, or broader</td>
</tr>
<tr>
<td>RQ</td><td>related and possibly synonymous.</td>
</tr>
<tr>
<td>RU</td><td>Related, unspecified</td>
</tr>
<tr>
<td>SY</td><td>source asserted synonymy.</td>
</tr>
<tr>
<td>XR</td><td>Not related, no mapping</td>
</tr>
<tr>
<td></td><td>Empty relationship</td>
</tr>
<tr>
<td>abnormal_cell_affected_by_chemical_or_drug</td><td>abnormal cell affected by chemical or drug</td>
</tr>
<tr>
<td>abnormality_associated_with_allele</td><td>abnormality associated with allele</td>
</tr>
<tr>
<td>absorbability_of</td><td>Absorbability of</td>
</tr>
<tr>
<td>access_device_used_by</td><td>Access device used by</td>
</tr>
<tr>
<td>access_instrument_of</td><td>Access instrument of</td>
</tr>
<tr>
<td>access_of</td><td>Access of</td>
</tr>
<tr>
<td>acted_on_by_process</td><td>Acted on by process</td>
</tr>
<tr>
<td>active_ingredient_of</td><td>Active ingredient of</td>
</tr>
<tr>
<td>active_metabolites_of</td><td>Active metabolites of</td>
</tr>
<tr>
<td>active_moiety_of</td><td>Active Moiety of</td>
</tr>
<tr>
<td>activity_of_allele</td><td>activity of allele</td>
</tr>
<tr>
<td>actual_outcome_of</td><td>Actual outcome of</td>
</tr>
<tr>
<td>add_on_code_for</td><td>Add on code for (Report with)</td>
</tr>
<tr>
<td>adheres_to</td><td>Adheres to</td>
</tr>
<tr>
<td>adjacent_to</td><td>Adjacent to</td>
</tr>
<tr>
<td>adjectival_form_of</td><td>Adjectival form of</td>
</tr>
<tr>
<td>adjustment_of</td><td>Adjustment of</td>
</tr>
<tr>
<td>afferent_to</td><td>Afferent to</td>
</tr>
<tr>
<td>after</td><td>After</td>
</tr>
<tr>
<td>alias_of</td><td>Alias_of</td>
</tr>
<tr>
<td>allele_absent_from_wild-type_chromosomal_location</td><td>allele absent from wild-type chromosomal location</td>
</tr>
<tr>
<td>allele_has_abnormality</td><td>allele has abnormality</td>
</tr>
<tr>
<td>allele_has_activity</td><td>allele has activity</td>
</tr>
<tr>
<td>allele_in_chromosomal_location</td><td>allele in chromosomal location</td>
</tr>
<tr>
<td>allele_plays_altered_role_in_process</td><td>allele plays altered role in process</td>
</tr>
<tr>
<td>allele_plays_role_in_metabolism_of_chemical_or_drug</td><td>allele plays role in metabolism of chemical or drug</td>
</tr>
<tr>
<td>allelic_variant_of</td><td>Allelic Variant of</td>
</tr>
<tr>
<td>alternative_of</td><td>Alternative of</td>
</tr>
<tr>
<td>analyzed_by</td><td>Analyzed by</td>
</tr>
<tr>
<td>analyzes</td><td>Analyzes</td>
</tr>
<tr>
<td>anatomic_structure_has_location</td><td>anatomic structure has location</td>
</tr>
<tr>
<td>anatomic_structure_is_physical_part_of</td><td>anatomic structure is physical part of</td>
</tr>
<tr>
<td>anatomical_entity_observed_in</td><td>anatomical entity observed in</td>
</tr>
<tr>
<td>anatomy_originated_from_biological_process</td><td>anatomy originated from biological process</td>
</tr>
<tr>
<td>answer_to_is_sterile</td><td>Answer to is sterile</td>
</tr>
<tr>
<td>answer_to</td><td>answer to</td>
</tr>
<tr>
<td>anterior_to</td><td>anterior to</td>
</tr>
<tr>
<td>anteroinferior_to</td><td>anteroinferior to</td>
</tr>
<tr>
<td>anterolateral_to</td><td>anterolateral to</td>
</tr>
<tr>
<td>anteromedial_to</td><td>anteromedial to</td>
</tr>
<tr>
<td>anterosuperior_to</td><td>anterosuperior to</td>
</tr>
<tr>
<td>approach_of_excluded</td><td>Approach of excluded</td>
</tr>
<tr>
<td>approach_of_possibly_included</td><td>Approach of possibly included</td>
</tr>
<tr>
<td>approach_of</td><td>Approach of</td>
</tr>
<tr>
<td>archetype_of</td><td>Archetype of</td>
</tr>
<tr>
<td>arterial_supply_of</td><td>Arterial supply of</td>
</tr>
<tr>
<td>articulates_with</td><td>Articulates with</td>
</tr>
<tr>
<td>associated_condition_of</td><td>Associated condition of</td>
</tr>
<tr>
<td>associated_disease</td><td>Associated disease</td>
</tr>
<tr>
<td>associated_etiologic_finding_of</td><td>Associated etiologic finding of</td>
</tr>
<tr>
<td>associated_finding_of_excluded</td><td>Associated finding of excluded</td>
</tr>
<tr>
<td>associated_finding_of_possibly_included</td><td>Associated finding of possibly included</td>
</tr>
<tr>
<td>associated_finding_of</td><td>Associated finding of</td>
</tr>
<tr>
<td>associated_function_of</td><td>Associated function of</td>
</tr>
<tr>
<td>associated_genetic_condition</td><td>Associated genetic condition</td>
</tr>
<tr>
<td>associated_morphology_of</td><td>Associated morphology of</td>
</tr>
<tr>
<td>associated_observation_of</td><td>Associated observation of</td>
</tr>
<tr>
<td>associated_procedure_of_excluded</td><td>Associated procedure of excluded</td>
</tr>
<tr>
<td>associated_procedure_of_possibly_included</td><td>Associated procedure of possibly included</td>
</tr>
<tr>
<td>associated_procedure_of</td><td>Associated procedure of</td>
</tr>
<tr>
<td>associated_with_malfunction_of_gene_product</td><td>associated with malfunction of gene product</td>
</tr>
<tr>
<td>associated_with</td><td>Associated with</td>
</tr>
<tr>
<td>attaches_to</td><td>Attaches to</td>
</tr>
<tr>
<td>away_from</td><td>Away from</td>
</tr>
<tr>
<td>basic_dose_form_of</td><td>Basic dose form of</td>
</tr>
<tr>
<td>basis_of_strength_substance_of</td><td>Basis of strength substance of</td>
</tr>
<tr>
<td>before</td><td>Before</td>
</tr>
<tr>
<td>biological_process_has_associated_location</td><td>biological process has associated location</td>
</tr>
<tr>
<td>biological_process_has_initiator_chemical_or_drug</td><td>biological process has initiator chemical or drug</td>
</tr>
<tr>
<td>biological_process_has_initiator_process</td><td>biological process has initiator process</td>
</tr>
<tr>
<td>biological_process_has_result_anatomy</td><td>biological process has result anatomy</td>
</tr>
<tr>
<td>biological_process_has_result_biological_process</td><td>biological process has result biological process</td>
</tr>
<tr>
<td>biological_process_has_result_chemical_or_drug</td><td>biological process has result chemical or drug</td>
</tr>
<tr>
<td>biological_process_involves_chemical_or_drug</td><td>biological process involves chemical or drug</td>
</tr>
<tr>
<td>biological_process_involves_gene_product</td><td>biological process involves gene product</td>
</tr>
<tr>
<td>biological_process_is_part_of_process</td><td>biological process is part of process</td>
</tr>
<tr>
<td>biological_process_results_from_biological_process</td><td>biological process results from biological process</td>
</tr>
<tr>
<td>biomarker_type_includes_gene_product</td><td>biomarker type includes gene product</td>
</tr>
<tr>
<td>biomarker_type_includes_gene</td><td>biomarker type includes gene</td>
</tr>
<tr>
<td>boss_of</td><td>Basis of strength substance of</td>
</tr>
<tr>
<td>bounded_by</td><td>Bounded by</td>
</tr>
<tr>
<td>bounds</td><td>Bounds</td>
</tr>
<tr>
<td>branch_of</td><td>Branch of</td>
</tr>
<tr>
<td>british_form_of</td><td>British form of</td>
</tr>
<tr>
<td>causative_agent_of</td><td>Causative agent of</td>
</tr>
<tr>
<td>cause_of</td><td>Cause of</td>
</tr>
<tr>
<td>cell_type_is_associated_with_eo_disease</td><td>cell type is associated with eo disease</td>
</tr>
<tr>
<td>cell_type_or_tissue_affected_by_chemical_or_drug</td><td>cell type or tissue affected by chemical or drug</td>
</tr>
<tr>
<td>challenge_of</td><td>Challenge of</td>
</tr>
<tr>
<td>characterized_by</td><td>Characterized by</td>
</tr>
<tr>
<td>characterizes</td><td>Characterizes</td>
</tr>
<tr>
<td>chemical_or_drug_affects_abnormal_cell</td><td>chemical or drug affects abnormal cell</td>
</tr>
<tr>
<td>chemical_or_drug_affects_cell_type_or_tissue</td><td>chemical or drug affects cell type or tissue</td>
</tr>
<tr>
<td>chemical_or_drug_affects_gene_product</td><td>chemical or drug affects gene product</td>
</tr>
<tr>
<td>chemical_or_drug_has_mechanism_of_action</td><td>chemical or drug has mechanism of action</td>
</tr>
<tr>
<td>chemical_or_drug_has_physiologic_effect</td><td>chemical or drug has physiologic effect</td>
</tr>
<tr>
<td>chemical_or_drug_initiates_biological_process</td><td>chemical or drug initiates biological process</td>
</tr>
<tr>
<td>chemical_or_drug_is_metabolized_by_enzyme</td><td>chemical or drug is metabolized by enzyme</td>
</tr>
<tr>
<td>chemical_or_drug_is_product_of_biological_process</td><td>chemical or drug is product of biological process</td>
</tr>
<tr>
<td>chemical_or_drug_metabolism_is_associated_with_allele</td><td>chemical or drug metabolism is associated with allele</td>
</tr>
<tr>
<td>chemical_or_drug_plays_role_in_biological_process</td><td>chemical or drug plays role in biological process</td>
</tr>
<tr>
<td>chemotherapy_regimen_has_component</td><td>chemotherapy regimen has component</td>
</tr>
<tr>
<td>chromosomal_location_of_allele</td><td>chromosomal location of allele</td>
</tr>
<tr>
<td>chromosomal_location_of_wild-type_gene</td><td>chromosomal location of wild-type gene</td>
</tr>
<tr>
<td>chromosome_involved_in_cytogenetic_abnormality</td><td>chromosome involved in cytogenetic abnormality</td>
</tr>
<tr>
<td>chromosome_mapped_to_disease</td><td>chromosome mapped to disease</td>
</tr>
<tr>
<td>class_code_classified_by</td><td>Class code classified by</td>
</tr>
<tr>
<td>class_of</td><td>Class of</td>
</tr>
<tr>
<td>classified_as</td><td>Classified as</td>
</tr>
<tr>
<td>classifies_class_code</td><td>Classifies class code</td>
</tr>
<tr>
<td>classifies</td><td>Classifies</td>
</tr>
<tr>
<td>clinical_course_of</td><td>Clinical course of</td>
</tr>
<tr>
<td>clinically_associated_with</td><td>Clinically associated with</td>
</tr>
<tr>
<td>clinically_similar</td><td>Clinically similar</td>
</tr>
<tr>
<td>clinician_form_of</td><td>Clinician form of</td>
</tr>
<tr>
<td>co-occurs_with</td><td>Co-ocurrs with</td>
</tr>
<tr>
<td>coating_material_of</td><td>Coating material of</td>
</tr>
<tr>
<td>common_name_of</td><td>common name of</td>
</tr>
<tr>
<td>completely_excised_anatomy_has_procedure</td><td>completely excised anatomy has procedure</td>
</tr>
<tr>
<td>completely_excised_anatomy_may_have_procedure</td><td>Completely excised anatomy may have procedure</td>
</tr>
<tr>
<td>complex_has_physical_part</td><td>complex has physical part</td>
</tr>
<tr>
<td>component_of_excluded</td><td>Component of excluded</td>
</tr>
<tr>
<td>component_of_possibly_included</td><td>Component of possibly included</td>
</tr>
<tr>
<td>component_of</td><td>Component of</td>
</tr>
<tr>
<td>compositional_material_of</td><td>Compositional material of</td>
</tr>
<tr>
<td>concentration_strength_denominator_unit_of</td><td>Concentration strength denominator unit of</td>
</tr>
<tr>
<td>concentration_strength_denominator_value_of</td><td>Concentration strength denominator value of</td>
</tr>
<tr>
<td>concentration_strength_numerator_unit_of</td><td>Concentration strength numerator unit of</td>
</tr>
<tr>
<td>concentration_strength_numerator_value_of</td><td>Concentration strength numerator value of</td>
</tr>
<tr>
<td>concept_in_subset</td><td>concept in subset</td>
</tr>
<tr>
<td>conceptual_part_of</td><td>Conceptual part of</td>
</tr>
<tr>
<td>connected_to</td><td>connected to</td>
</tr>
<tr>
<td>connection_type_of</td><td>connection type of</td>
</tr>
<tr>
<td>consider_from</td><td>Consider from</td>
</tr>
<tr>
<td>consider</td><td>Consider</td>
</tr>
<tr>
<td>consists_of</td><td>Consists of</td>
</tr>
<tr>
<td>constitutes</td><td>Constitutes</td>
</tr>
<tr>
<td>constitutional_part_of</td><td>Constitutional part of</td>
</tr>
<tr>
<td>consumer_friendly_form_of</td><td>Consumer friendly form of</td>
</tr>
<tr>
<td>contained_in</td><td>Contained in</td>
</tr>
<tr>
<td>contains</td><td>Contains</td>
</tr>
<tr>
<td>context_binding_of</td><td>Context binding of</td>
</tr>
<tr>
<td>continuation_branch_of</td><td>Continuation branch of</td>
</tr>
<tr>
<td>continuous_distally_with</td><td>continuous distally with</td>
</tr>
<tr>
<td>continuous_proximally_with</td><td>continuous proximally with</td>
</tr>
<tr>
<td>continuous_with</td><td>Continuous with</td>
</tr>
<tr>
<td>contraindicated_class_of</td><td>Contraindicated class of</td>
</tr>
<tr>
<td>contraindicated_mechanism_of_action_of</td><td>Contraindicated mechanism of action of</td>
</tr>
<tr>
<td>contraindicated_physiologic_effect_of</td><td>Contraindicated physiologic effect of</td>
</tr>
<tr>
<td>contraindicated_with_disease</td><td>Contraindicated with disease</td>
</tr>
<tr>
<td>corresponds_to</td><td>corresponds to</td>
</tr>
<tr>
<td>count_of_active_ingredient_of</td><td>Count of active ingredient of</td>
</tr>
<tr>
<td>count_of_base_of_active_ingredient_of</td><td>Count of base of active ingredient of</td>
</tr>
<tr>
<td>count_of</td><td>count_of</td>
</tr>
<tr>
<td>course_of</td><td>Course of</td>
</tr>
<tr>
<td>ctcae_5_parent_of</td><td>ctcae_5 parent of</td>
</tr>
<tr>
<td>cytogenetic_abnormality_involves_chromosome</td><td>cytogenetic abnormality involves chromosome</td>
</tr>
<tr>
<td>data_element_of</td><td>data element of</td>
</tr>
<tr>
<td>ddx</td><td>dxx</td>
</tr>
<tr>
<td>default_inpatient_classification_of</td><td>Default inpatient classification of</td>
</tr>
<tr>
<td>default_mapped_from</td><td>Default mapped from</td>
</tr>
<tr>
<td>default_mapped_to</td><td>Default mapped to</td>
</tr>
<tr>
<td>default_outpatient_classification_of</td><td>Default outpatient classification of</td>
</tr>
<tr>
<td>defining_characteristic_of</td><td>Defining characteristic of</td>
</tr>
<tr>
<td>definitional_manifestation_of</td><td>Definitional manifestation of</td>
</tr>
<tr>
<td>degree_of</td><td>Degree of</td>
</tr>
<tr>
<td>dependent_of</td><td>Dependent of</td>
</tr>
<tr>
<td>derives_from</td><td>derives from</td>
</tr>
<tr>
<td>derives</td><td>derives</td>
</tr>
<tr>
<td>development_type_of</td><td>development type of</td>
</tr>
<tr>
<td>developmental_stage_of</td><td>developmental stage of</td>
</tr>
<tr>
<td>develops_from</td><td>Develops from</td>
</tr>
<tr>
<td>develops_into</td><td>develops into</td>
</tr>
<tr>
<td>device_characteristic_of</td><td>Device characteristic of</td>
</tr>
<tr>
<td>device_intended_site_of</td><td>Device intended site of</td>
</tr>
<tr>
<td>device_used_by</td><td>Device used by</td>
</tr>
<tr>
<td>diagnosed_by</td><td>Diagnosed by</td>
</tr>
<tr>
<td>diagnoses</td><td>Diagnoses</td>
</tr>
<tr>
<td>diagnostic_criteria_of</td><td>DSM-5 diagnostic criteria for ICD code</td>
</tr>
<tr>
<td>direct_cell_shape_of</td><td>direct cell shape of</td>
</tr>
<tr>
<td>direct_device_of</td><td>Direct device of</td>
</tr>
<tr>
<td>direct_left_of</td><td>direct left of</td>
</tr>
<tr>
<td>direct_morphology_of</td><td>Direct morphology of</td>
</tr>
<tr>
<td>direct_procedure_site_of</td><td>Direct procedure site of</td>
</tr>
<tr>
<td>direct_right_of</td><td>direct right of</td>
</tr>
<tr>
<td>direct_site_of</td><td>Direct site of</td>
</tr>
<tr>
<td>direct_substance_of</td><td>Direct substance of</td>
</tr>
<tr>
<td>disease_excludes_abnormal_cell</td><td>disease excludes abnormal cell</td>
</tr>
<tr>
<td>disease_excludes_cytogenetic_abnormality</td><td>disease excludes cytogenetic abnormality</td>
</tr>
<tr>
<td>disease_excludes_finding</td><td>disease excludes finding</td>
</tr>
<tr>
<td>disease_excludes_metastatic_anatomic_site</td><td>disease excludes metastatic anatomic site</td>
</tr>
<tr>
<td>disease_excludes_molecular_abnormality</td><td>disease excludes molecular abnormality</td>
</tr>
<tr>
<td>disease_excludes_normal_cell_origin</td><td>disease excludes normal cell origin</td>
</tr>
<tr>
<td>disease_excludes_normal_tissue_origin</td><td>disease excludes normal tissue origin</td>
</tr>
<tr>
<td>disease_excludes_primary_anatomic_site</td><td>disease excludes primary anatomic site</td>
</tr>
<tr>
<td>disease_has_abnormal_cell</td><td>disease has abnormal cell</td>
</tr>
<tr>
<td>disease_has_accepted_treatment_with_regimen</td><td>Disease Has Accepted Treatment With Regimen</td>
</tr>
<tr>
<td>disease_has_associated_anatomic_site</td><td>disease has associated anatomic site</td>
</tr>
<tr>
<td>disease_has_associated_disease</td><td>disease has associated disease</td>
</tr>
<tr>
<td>disease_has_associated_gene</td><td>disease has associated gene</td>
</tr>
<tr>
<td>disease_has_cytogenetic_abnormality</td><td>disease has cytogenetic abnormality</td>
</tr>
<tr>
<td>disease_has_finding</td><td>disease has finding</td>
</tr>
<tr>
<td>disease_has_metastatic_anatomic_site</td><td>disease has metastatic anatomic site</td>
</tr>
<tr>
<td>disease_has_molecular_abnormality</td><td>disease has molecular abnormality</td>
</tr>
<tr>
<td>disease_has_normal_cell_origin</td><td>disease has normal cell origin</td>
</tr>
<tr>
<td>disease_has_normal_tissue_origin</td><td>disease has normal tissue origin</td>
</tr>
<tr>
<td>disease_has_primary_anatomic_site</td><td>disease has primary anatomic site</td>
</tr>
<tr>
<td>disease_is_grade</td><td>disease is grade</td>
</tr>
<tr>
<td>disease_is_marked_by_gene</td><td>disease is marked by gene</td>
</tr>
<tr>
<td>disease_is_stage</td><td>disease is stage</td>
</tr>
<tr>
<td>disease_mapped_to_chromosome</td><td>disease mapped to chromosome</td>
</tr>
<tr>
<td>disease_mapped_to_gene</td><td>Disease mapped to gene</td>
</tr>
<tr>
<td>disease_may_have_abnormal_cell</td><td>disease may have abnormal cell</td>
</tr>
<tr>
<td>disease_may_have_associated_disease</td><td>disease may have associated disease</td>
</tr>
<tr>
<td>disease_may_have_cytogenetic_abnormality</td><td>disease may have cytogenetic abnormality</td>
</tr>
<tr>
<td>disease_may_have_finding</td><td>disease may have finding</td>
</tr>
<tr>
<td>disease_may_have_molecular_abnormality</td><td>disease may have molecular abnormality</td>
</tr>
<tr>
<td>disease_may_have_normal_cell_origin</td><td>disease may have normal cell origin</td>
</tr>
<tr>
<td>disease_may_have_normal_tissue_origin</td><td>disease may have normal tissue origin</td>
</tr>
<tr>
<td>disposition_of</td><td>Disposition of</td>
</tr>
<tr>
<td>distal_to</td><td>distal to</td>
</tr>
<tr>
<td>divisor_of</td><td>Divisor of</td>
</tr>
<tr>
<td>do_not_code_with</td><td>Do not code with (excludes)</td>
</tr>
<tr>
<td>dose_form_administration_method_of</td><td>Dose form administration method of</td>
</tr>
<tr>
<td>dose_form_intended_site_of</td><td>Dose form intended site of</td>
</tr>
<tr>
<td>dose_form_of</td><td>Dose form of</td>
</tr>
<tr>
<td>dose_form_release_characteristic_of</td><td>Dose form release characteristic of</td>
</tr>
<tr>
<td>dose_form_transformation_of</td><td>Dose form transformation of</td>
</tr>
<tr>
<td>doseformgroup_of</td><td>Dose form group of</td>
</tr>
<tr>
<td>drains_into</td><td>drains into</td>
</tr>
<tr>
<td>due_to</td><td>Due to</td>
</tr>
<tr>
<td>during</td><td>During</td>
</tr>
<tr>
<td>effect_may_be_inhibited_by</td><td>Effect may be inhibited by</td>
</tr>
<tr>
<td>efferent_to</td><td>Efferent to</td>
</tr>
<tr>
<td>endogenous_product_related_to</td><td>Endogenous Product Related To</td>
</tr>
<tr>
<td>ends_during</td><td>Ends during</td>
</tr>
<tr>
<td>energy_used_by</td><td>Energy used by</td>
</tr>
<tr>
<td>entire_anatomy_structure_of</td><td>Entire anatomy structure of</td>
</tr>
<tr>
<td>entry_combination_of</td><td>Entry Combination of</td>
</tr>
<tr>
<td>entry_term_of</td><td>Entry term of</td>
</tr>
<tr>
<td>entry_version_of</td><td>Entry version of</td>
</tr>
<tr>
<td>enzyme_metabolizes_chemical_or_drug</td><td>enzyme metabolizes chemical or drug</td>
</tr>
<tr>
<td>eo_anatomy_is_associated_with_eo_disease</td><td>eo anatomy is associated with eo disease</td>
</tr>
<tr>
<td>eo_disease_has_associated_cell_type</td><td>eo disease has associated cell type</td>
</tr>
<tr>
<td>eo_disease_has_associated_eo_anatomy</td><td>eo disease has associated eo anatomy</td>
</tr>
<tr>
<td>eo_disease_has_property_or_attribute</td><td>eo disease has property or attribute</td>
</tr>
<tr>
<td>eo_disease_maps_to_human_disease</td><td>eo disease maps to human disease</td>
</tr>
<tr>
<td>episodicity_of</td><td>Episodicity of</td>
</tr>
<tr>
<td>evaluation_of</td><td>Evaluation of</td>
</tr>
<tr>
<td>excised_anatomy_has_procedure</td><td>excised anatomy has procedure</td>
</tr>
<tr>
<td>excised_anatomy_may_have_procedure</td><td>excised anatomy may have procedure</td>
</tr>
<tr>
<td>exhibited_by</td><td>Exhibited by</td>
</tr>
<tr>
<td>exhibits</td><td>Exhibits</td>
</tr>
<tr>
<td>expanded_form_of</td><td>Expanded form of</td>
</tr>
<tr>
<td>expected_outcome_of</td><td>Expected outcome of</td>
</tr>
<tr>
<td>extended_to_by_process</td><td>Extended to by process</td>
</tr>
<tr>
<td>extent_of</td><td>Extent of</td>
</tr>
<tr>
<td>external_to</td><td>external to</td>
</tr>
<tr>
<td>filling_of</td><td>Filling of</td>
</tr>
<tr>
<td>finding_context_of</td><td>Finding context of</td>
</tr>
<tr>
<td>finding_informer_of</td><td>Finding informer of</td>
</tr>
<tr>
<td>finding_method_of</td><td>Finding method of</td>
</tr>
<tr>
<td>finding_site_of</td><td>Finding site of</td>
</tr>
<tr>
<td>focus_of</td><td>Focus of</td>
</tr>
<tr>
<td>form_of</td><td>Form of</td>
</tr>
<tr>
<td>formed_by</td><td>formed by</td>
</tr>
<tr>
<td>forms</td><td>forms</td>
</tr>
<tr>
<td>full_grown_phenotype_of</td><td>full grown phenotype of</td>
</tr>
<tr>
<td>fuses_with</td><td>fuses with</td>
</tr>
<tr>
<td>fusion_of</td><td>fusion of</td>
</tr>
<tr>
<td>gene_associated_with_disease</td><td>gene associated with disease</td>
</tr>
<tr>
<td>gene_encodes_gene_product</td><td>gene encodes gene product</td>
</tr>
<tr>
<td>gene_found_in_organism</td><td>gene found in organism</td>
</tr>
<tr>
<td>gene_has_abnormality</td><td>gene has abnormality</td>
</tr>
<tr>
<td>gene_has_physical_location</td><td>gene has physical location</td>
</tr>
<tr>
<td>gene_in_chromosomal_location</td><td>gene in chromosomal location</td>
</tr>
<tr>
<td>gene_involved_in_molecular_abnormality</td><td>gene involved in molecular abnormality</td>
</tr>
<tr>
<td>gene_involved_in_pathogenesis_of_disease</td><td>gene involved in pathogenesis of disease</td>
</tr>
<tr>
<td>gene_is_biomarker_of</td><td>gene is biomarker of</td>
</tr>
<tr>
<td>gene_is_biomarker_type</td><td>gene is biomarker type</td>
</tr>
<tr>
<td>gene_is_element_in_pathway</td><td>gene is element in pathway</td>
</tr>
<tr>
<td>gene_mapped_to_disease</td><td>gene mapped to disease</td>
</tr>
<tr>
<td>gene_mutant_encodes_gene_product_sequence_variation</td><td>gene mutant encodes gene product sequence variation</td>
</tr>
<tr>
<td>gene_plays_role_in_process</td><td>gene plays role in process</td>
</tr>
<tr>
<td>gene_product_affected_by_chemical_or_drug</td><td>gene product affected by chemical or drug</td>
</tr>
<tr>
<td>gene_product_encoded_by_gene</td><td>gene product encoded by gene</td>
</tr>
<tr>
<td>gene_product_expressed_in_tissue</td><td>gene product expressed in tissue</td>
</tr>
<tr>
<td>gene_product_has_abnormality</td><td>gene product has abnormality</td>
</tr>
<tr>
<td>gene_product_has_associated_anatomy</td><td>gene product has associated anatomy</td>
</tr>
<tr>
<td>gene_product_has_biochemical_function</td><td>gene product has biochemical function</td>
</tr>
<tr>
<td>gene_product_has_chemical_classification</td><td>gene product has chemical classification</td>
</tr>
<tr>
<td>gene_product_has_gene_product_variant</td><td>gene product has gene product variant</td>
</tr>
<tr>
<td>gene_product_has_organism_source</td><td>gene product has organism source</td>
</tr>
<tr>
<td>gene_product_has_structural_domain_or_motif</td><td>gene product has structural domain or motif</td>
</tr>
<tr>
<td>gene_product_is_biomarker_of</td><td>gene product is biomarker of</td>
</tr>
<tr>
<td>gene_product_is_biomarker_type</td><td>gene product is biomarker type</td>
</tr>
<tr>
<td>gene_product_is_element_in_pathway</td><td>gene product is element in pathway</td>
</tr>
<tr>
<td>gene_product_is_physical_part_of</td><td>gene product is physical part of</td>
</tr>
<tr>
<td>gene_product_malfunction_associated_with_disease</td><td>gene product malfunction associated with disease</td>
</tr>
<tr>
<td>gene_product_plays_role_in_biological_process</td><td>gene product plays role in biological process</td>
</tr>
<tr>
<td>gene_product_sequence_variation_encoded_by_gene_mutant</td><td>gene product sequence variation encoded by gene mutant</td>
</tr>
<tr>
<td>gene_product_variant_of_gene_product</td><td>gene product variant of gene product</td>
</tr>
<tr>
<td>genetic_biomarker_related_to</td><td>genetic biomarker related to</td>
</tr>
<tr>
<td>germ_origin_of</td><td>Germ origin of</td>
</tr>
<tr>
<td>grade_of</td><td>grade of</td>
</tr>
<tr>
<td>happens_during</td><td>Happens during</td>
</tr>
<tr>
<td>has_absorbability</td><td>Has absorbability</td>
</tr>
<tr>
<td>has_acc-aha_sars2_authorized_value</td><td>has ACC-AHA SARS2 authorized value</td>
</tr>
<tr>
<td>has_access_instrument</td><td>Has access instrument</td>
</tr>
<tr>
<td>has_access</td><td>Has access</td>
</tr>
<tr>
<td>has_action_guidance</td><td>Has action guidance</td>
</tr>
<tr>
<td>has_active_ingredient</td><td>Has active ingredient</td>
</tr>
<tr>
<td>has_active_metabolites</td><td>Has active metabolites</td>
</tr>
<tr>
<td>has_active_moiety</td><td>Has Active Moiety</td>
</tr>
<tr>
<td>has_actual_outcome</td><td>Has actual outcome</td>
</tr>
<tr>
<td>has_add_on_code</td><td>Has add on code (Report with)</td>
</tr>
<tr>
<td>has_adherent</td><td>Has adherent</td>
</tr>
<tr>
<td>has_adjustment</td><td>Has adjustment</td>
</tr>
<tr>
<td>has_aggregation_view</td><td>Has aggregation view</td>
</tr>
<tr>
<td>has_alias</td><td>Has alias</td>
</tr>
<tr>
<td>has_allelic_variant</td><td>Has allelic variant</td>
</tr>
<tr>
<td>has_alternative</td><td>Has alternative</td>
</tr>
<tr>
<td>has_answer</td><td>has answer</td>
</tr>
<tr>
<td>has_approach_guidance</td><td>Has approach guidance</td>
</tr>
<tr>
<td>has_approach</td><td>Has approach</td>
</tr>
<tr>
<td>has_archetype</td><td>Has archetype</td>
</tr>
<tr>
<td>has_arterial_supply</td><td>Has arterial supply</td>
</tr>
<tr>
<td>has_associated_condition</td><td>has_associated_condition</td>
</tr>
<tr>
<td>has_associated_etiologic_finding</td><td>Has associated etiologic finding</td>
</tr>
<tr>
<td>has_associated_finding</td><td>Has associated finding</td>
</tr>
<tr>
<td>has_associated_function</td><td>Has associated function</td>
</tr>
<tr>
<td>has_associated_morphology</td><td>Has associated morphology</td>
</tr>
<tr>
<td>has_associated_observation</td><td>Has associated observation</td>
</tr>
<tr>
<td>has_associated_procedure</td><td>Has associated procedure</td>
</tr>
<tr>
<td>has_at_risk_population</td><td>Has at risk population</td>
</tr>
<tr>
<td>has_basic_dose_form</td><td>Has basic dose form</td>
</tr>
<tr>
<td>has_basis_of_strength_substance</td><td>Has basis of strength substance</td>
</tr>
<tr>
<td>has_boss</td><td>Has basis of strength substance</td>
</tr>
<tr>
<td>has_branch</td><td>Has branch</td>
</tr>
<tr>
<td>has_british_form</td><td>Has British form</td>
</tr>
<tr>
<td>has_causative_agent</td><td>Has causative agent</td>
</tr>
<tr>
<td>has_challenge</td><td>Has challenge</td>
</tr>
<tr>
<td>has_class</td><td>Has class</td>
</tr>
<tr>
<td>has_clinical_course</td><td>Has clinical course</td>
</tr>
<tr>
<td>has_clinician_form</td><td>Has clinician form</td>
</tr>
<tr>
<td>has_coating_material</td><td>Has coating material</td>
</tr>
<tr>
<td>has_common_name</td><td>Has common name</td>
</tr>
<tr>
<td>has_communication_with_wound</td><td>Has communication with wound</td>
</tr>
<tr>
<td>has_component</td><td>Has component</td>
</tr>
<tr>
<td>has_compositional_material</td><td>Has compositional material</td>
</tr>
<tr>
<td>has_concentration_strength_denominator_unit</td><td>Has concentration strength denominator unit</td>
</tr>
<tr>
<td>has_concentration_strength_denominator_value</td><td>Has concentration strength denominator value</td>
</tr>
<tr>
<td>has_concentration_strength_numerator_unit</td><td>Has concentration strength numerator unit</td>
</tr>
<tr>
<td>has_concentration_strength_numerator_value</td><td>Has concentration strength numerator value</td>
</tr>
<tr>
<td>has_conceptual_part</td><td>Has conceptual part</td>
</tr>
<tr>
<td>has_connection_type</td><td>has connection type</td>
</tr>
<tr>
<td>has_constitutional_part</td><td>Has constitutional part</td>
</tr>
<tr>
<td>has_consumer_friendly_form</td><td>Has consumer friendly form</td>
</tr>
<tr>
<td>has_context_binding</td><td>Has context binding</td>
</tr>
<tr>
<td>has_continuation_branch</td><td>Has continuation branch</td>
</tr>
<tr>
<td>has_contraindicated_class</td><td>Has contraindicated class</td>
</tr>
<tr>
<td>has_contraindicated_drug</td><td>Has contraindicated drug</td>
</tr>
<tr>
<td>has_contraindicated_mechanism_of_action</td><td>Has contraindicated mechanism of action</td>
</tr>
<tr>
<td>has_contraindicated_physiologic_effect</td><td>Has contraindicated physiologic effect</td>
</tr>
<tr>
<td>has_count_of_active_ingredient</td><td>Has count of active ingredient</td>
</tr>
<tr>
<td>has_count_of_base_of_active_ingredient</td><td>Has count of base of active ingredient</td>
</tr>
<tr>
<td>has_count</td><td>Has count</td>
</tr>
<tr>
<td>has_course</td><td>Has course</td>
</tr>
<tr>
<td>has_ctcae_5_parent</td><td>has ctcae_5 parent</td>
</tr>
<tr>
<td>has_ctdc_value</td><td>has ctdc value</td>
</tr>
<tr>
<td>has_data_element</td><td>has data element</td>
</tr>
<tr>
<td>has_default_inpatient_classification</td><td>Has default inpatient classification</td>
</tr>
<tr>
<td>has_default_outpatient_classification</td><td>Has default outpatient classification</td>
</tr>
<tr>
<td>has_defining_characteristic</td><td>Has defining characteristic</td>
</tr>
<tr>
<td>has_definitional_manifestation</td><td>Has definitional manifestation</td>
</tr>
<tr>
<td>has_degree</td><td>Has degree</td>
</tr>
<tr>
<td>has_dependent</td><td>Has dependent</td>
</tr>
<tr>
<td>has_development_type</td><td>has development type</td>
</tr>
<tr>
<td>has_developmental_stage</td><td>has developmental stage</td>
</tr>
<tr>
<td>has_device_characteristic</td><td>Has device characteristic</td>
</tr>
<tr>
<td>has_device_intended_site</td><td>Has device intended site</td>
</tr>
<tr>
<td>has_diagnostic_criteria</td><td>ICD has DSM-5 diagnostic criteria	</td>
</tr>
<tr>
<td>has_dipg_dmg_authorized_value</td><td>has DIPG DMG authorized value</td>
</tr>
<tr>
<td>has_direct_cell_shape</td><td>has direct cell shape</td>
</tr>
<tr>
<td>has_direct_device</td><td>Has direct device</td>
</tr>
<tr>
<td>has_direct_morphology</td><td>Has direct morphology</td>
</tr>
<tr>
<td>has_direct_procedure_site</td><td>Has direct procedure site</td>
</tr>
<tr>
<td>has_direct_site</td><td>Has direct site</td>
</tr>
<tr>
<td>has_direct_substance</td><td>Has direct substance</td>
</tr>
<tr>
<td>has_disposition</td><td>Has disposition</td>
</tr>
<tr>
<td>has_divisor</td><td>Has divisor</td>
</tr>
<tr>
<td>has_dose_form_administration_method</td><td>Has dose form administration method</td>
</tr>
<tr>
<td>has_dose_form_intended_site</td><td>Has dose form intended site</td>
</tr>
<tr>
<td>has_dose_form_release_characteristic</td><td>Has dose form release characteristic</td>
</tr>
<tr>
<td>has_dose_form_transformation</td><td>Has dose form transformation</td>
</tr>
<tr>
<td>has_dose_form</td><td>Has dose form</td>
</tr>
<tr>
<td>has_doseformgroup</td><td>Has dose form group</td>
</tr>
<tr>
<td>has_entire_anatomy_structure</td><td>Has entire anatomy structure</td>
</tr>
<tr>
<td>has_entry_combination</td><td>Has Entry Combination</td>
</tr>
<tr>
<td>has_entry_term</td><td>Has entry term</td>
</tr>
<tr>
<td>has_entry_version</td><td>Has entry version</td>
</tr>
<tr>
<td>has_episodicity</td><td>Has episodicity</td>
</tr>
<tr>
<td>has_evaluation</td><td>Has evaluation</td>
</tr>
<tr>
<td>has_exam</td><td>Has exam</td>
</tr>
<tr>
<td>has_excluded_approach</td><td>Has excluded approach</td>
</tr>
<tr>
<td>has_excluded_associated_finding</td><td>Has excluded associated finding</td>
</tr>
<tr>
<td>has_excluded_associated_procedure</td><td>Has excluded associated procedure</td>
</tr>
<tr>
<td>has_excluded_component</td><td>Has excluded component</td>
</tr>
<tr>
<td>has_excluded_locale</td><td>Has excluded locale</td>
</tr>
<tr>
<td>has_excluded_method</td><td>Has excluded method</td>
</tr>
<tr>
<td>has_excluded_pathology</td><td>Has excluded pathology</td>
</tr>
<tr>
<td>has_excluded_patient_type</td><td>Has excluded patient type</td>
</tr>
<tr>
<td>has_excluded_procedure_device</td><td>Has excluded procedure device</td>
</tr>
<tr>
<td>has_excluded_procedure_site</td><td>Has excluded procedure site</td>
</tr>
<tr>
<td>has_excluded_route_of_administration</td><td>Has excluded route of administration</td>
</tr>
<tr>
<td>has_excluded_specimen</td><td>Has excluded specimen</td>
</tr>
<tr>
<td>has_excluded_surgical_extent</td><td>Has excluded surgical extent</td>
</tr>
<tr>
<td>has_expanded_form</td><td>Has expanded form</td>
</tr>
<tr>
<td>has_expected_outcome</td><td>Has expected outcome</td>
</tr>
<tr>
<td>has_extent</td><td>Has extent</td>
</tr>
<tr>
<td>has_filling</td><td>Has filling</td>
</tr>
<tr>
<td>has_finding_context</td><td>Has finding context</td>
</tr>
<tr>
<td>has_finding_informer</td><td>Has finding informer</td>
</tr>
<tr>
<td>has_finding_method</td><td>Has finding method</td>
</tr>
<tr>
<td>has_finding_site</td><td>Has finding site</td>
</tr>
<tr>
<td>has_focus</td><td>Has focus</td>
</tr>
<tr>
<td>has_form</td><td>Has form</td>
</tr>
<tr>
<td>has_free_acid_or_base_form</td><td>has free acid or base form</td>
</tr>
<tr>
<td>has_full_grown_phenotype</td><td>has full grown phenotype</td>
</tr>
<tr>
<td>has_fusion</td><td>has fusion</td>
</tr>
<tr>
<td>has_gdc_value</td><td>has gdc value</td>
</tr>
<tr>
<td>has_gene_product_element</td><td>has gene product element</td>
</tr>
<tr>
<td>has_germ_origin</td><td>Has germ origin</td>
</tr>
<tr>
<td>has_given_pharmaceutical_substance</td><td>Has given pharmaceutical substance</td>
</tr>
<tr>
<td>has_grade</td><td>has grade</td>
</tr>
<tr>
<td>has_icdc_value</td><td>has icdc value</td>
</tr>
<tr>
<td>has_imaged_location</td><td>Has imaged location</td>
</tr>
<tr>
<td>has_imaging_focus</td><td>Has imaging focus</td>
</tr>
<tr>
<td>has_inactive_ingredient</td><td>Has Inactive Ingredient</td>
</tr>
<tr>
<td>has_inc_parent</td><td>has inc parent</td>
</tr>
<tr>
<td>has_indirect_device</td><td>Has indirect device</td>
</tr>
<tr>
<td>has_indirect_morphology</td><td>Has indirect morphology</td>
</tr>
<tr>
<td>has_indirect_procedure_site</td><td>Has indirect procedure site</td>
</tr>
<tr>
<td>has_ingredient_characteristic</td><td>Has ingredient characteristic</td>
</tr>
<tr>
<td>has_ingredient_qualitative_strength</td><td>Has ingredient qualitative strength</td>
</tr>
<tr>
<td>has_ingredients</td><td>Has ingredients</td>
</tr>
<tr>
<td>has_ingredient</td><td>Has ingredient</td>
</tr>
<tr>
<td>has_inherent_3d_shape</td><td>Has inherent 3-D shape</td>
</tr>
<tr>
<td>has_inherent_attribute</td><td>Has inherent attribute</td>
</tr>
<tr>
<td>has_inherent_location</td><td>Has inherent location</td>
</tr>
<tr>
<td>has_inheritance_type</td><td>Has inheritance type</td>
</tr>
<tr>
<td>has_insertion</td><td>has insertion</td>
</tr>
<tr>
<td>has_instrumentation</td><td>Has instrumentation</td>
</tr>
<tr>
<td>has_intent</td><td>Has intent</td>
</tr>
<tr>
<td>has_interpretation</td><td>Has interpretation</td>
</tr>
<tr>
<td>has_lab_number</td><td>Has lab number</td>
</tr>
<tr>
<td>has_lateral_anatomic_location</td><td>Has lateral anatomic location</td>
</tr>
<tr>
<td>has_lateral_location_presence</td><td>Has lateral location presence</td>
</tr>
<tr>
<td>has_laterality</td><td>Has laterality</td>
</tr>
<tr>
<td>has_life_circumstance</td><td>Has life circumstance</td>
</tr>
<tr>
<td>has_locale</td><td>Has locale</td>
</tr>
<tr>
<td>has_location</td><td>Has location</td>
</tr>
<tr>
<td>has_loinc_number</td><td>Has Loinc number</td>
</tr>
<tr>
<td>has_lymphatic_drainage</td><td>Has lymphatic drainage</td>
</tr>
<tr>
<td>has_maneuver_type</td><td>has maneuver type</td>
</tr>
<tr>
<td>has_manifestation</td><td>Has manifestation</td>
</tr>
<tr>
<td>has_mapping_qualifier</td><td>Has mapping qualifier</td>
</tr>
<tr>
<td>has_measured_component</td><td>Has measured component</td>
</tr>
<tr>
<td>has_measurement_method</td><td>Has measurement method</td>
</tr>
<tr>
<td>has_mechanism_of_action</td><td>Has mechanism of action</td>
</tr>
<tr>
<td>has_member</td><td>Has member</td>
</tr>
<tr>
<td>has_method</td><td>Has method</td>
</tr>
<tr>
<td>has_modality_subtype</td><td>Has modality subtype</td>
</tr>
<tr>
<td>has_modality_type</td><td>has modality type</td>
</tr>
<tr>
<td>has_modification</td><td>Has modification</td>
</tr>
<tr>
<td>has_multi_level_category</td><td>Has multi-level category</td>
</tr>
<tr>
<td>has_nerve_supply</td><td>Has nerve supply</td>
</tr>
<tr>
<td>has_object_guidance</td><td>Has object guidance</td>
</tr>
<tr>
<td>has_observed_anatomical_entity</td><td>has observed anatomical entity</td>
</tr>
<tr>
<td>has_occurrence</td><td>Has occurrence</td>
</tr>
<tr>
<td>has_onset</td><td>Has onset</td>
</tr>
<tr>
<td>has_ooro_pc_value</td><td>has OORO PC value</td>
</tr>
<tr>
<td>has_origin</td><td>Has origin</td>
</tr>
<tr>
<td>has_owning_affiliate</td><td>Has owning affiliate</td>
</tr>
<tr>
<td>has_owning_section</td><td>Has owning section</td>
</tr>
<tr>
<td>has_owning_subsection</td><td>Has owning subsection</td>
</tr>
<tr>
<td>has_panel_element</td><td>CPT laboratory panel code has laboratory panel test member</td>
</tr>
<tr>
<td>has_parent_group</td><td>Has parent group</td>
</tr>
<tr>
<td>has_parent</td><td>Has parent</td>
</tr>
<tr>
<td>has_part_anatomy_structure</td><td>Has part anatomy structure</td>
</tr>
<tr>
<td>has_part</td><td>Has part</td>
</tr>
<tr>
<td>has_pathological_process</td><td>Has pathological process</td>
</tr>
<tr>
<td>has_pathology</td><td>Has pathology</td>
</tr>
<tr>
<td>has_patient_type</td><td>Has patient type</td>
</tr>
<tr>
<td>has_pcdc_all_authorized_value</td><td>has PCDC ALL authorized value</td>
</tr>
<tr>
<td>has_pcdc_aml_authorized_value</td><td>has PCDC AML authorized value</td>
</tr>
<tr>
<td>has_pcdc_data_type</td><td>has pcdc data type</td>
</tr>
<tr>
<td>has_pcdc_ews_authorized_value</td><td>has PCDC EWS authorized value</td>
</tr>
<tr>
<td>has_pcdc_gct_authorized_value</td><td>has PCDC GCT authorized value</td>
</tr>
<tr>
<td>has_pcdc_hl_authorized_value</td><td>has PCDC HL authorized value</td>
</tr>
<tr>
<td>has_pcdc_os_authorized_value</td><td>has PCDC OS authorized value</td>
</tr>
<tr>
<td>has_permuted_term</td><td>Has permuted term</td>
</tr>
<tr>
<td>has_pharmaceutical_administration_method</td><td>has pharmaceutical administration method</td>
</tr>
<tr>
<td>has_pharmaceutical_basic_dose_form</td><td>has pharmaceutical basic dose form</td>
</tr>
<tr>
<td>has_pharmaceutical_intended_site</td><td>has pharmaceutical intended site</td>
</tr>
<tr>
<td>has_pharmaceutical_release_characteristics</td><td>has pharmaceutical release characteristics</td>
</tr>
<tr>
<td>has_pharmaceutical_route</td><td>Has_pharmaceutical_route</td>
</tr>
<tr>
<td>has_pharmaceutical_state_of_matter</td><td>has pharmaceutical state of matter</td>
</tr>
<tr>
<td>has_pharmaceutical_transformation</td><td>has pharmaceutical transformation</td>
</tr>
<tr>
<td>has_pharmacokinetics</td><td>Has pharmacokinetics</td>
</tr>
<tr>
<td>has_phenotype</td><td>Has phenotype</td>
</tr>
<tr>
<td>has_physical_part_of_anatomic_structure</td><td>has physical part of anatomic structure</td>
</tr>
<tr>
<td>has_physiologic_effect</td><td>Has physiologic effect</td>
</tr>
<tr>
<td>has_physiologic_state</td><td>Has physiologic state</td>
</tr>
<tr>
<td>has_possibly_included_approach</td><td>Has possibly included approach</td>
</tr>
<tr>
<td>has_possibly_included_associated_finding</td><td>Has possibly included associated finding</td>
</tr>
<tr>
<td>has_possibly_included_associated_procedure</td><td>Has possibly included associated procedure</td>
</tr>
<tr>
<td>has_possibly_included_component</td><td>Has possibly included component</td>
</tr>
<tr>
<td>has_possibly_included_method</td><td>Has possibly included method</td>
</tr>
<tr>
<td>has_possibly_included_panel_element</td><td>CPT laboratory panel code has possibly included laboratory panel test member</td>
</tr>
<tr>
<td>has_possibly_included_pathology</td><td>Has possibly included pathology</td>
</tr>
<tr>
<td>has_possibly_included_patient_type</td><td>Has possibly included patient type</td>
</tr>
<tr>
<td>has_possibly_included_procedure_device</td><td>Has possibly included procedure device</td>
</tr>
<tr>
<td>has_possibly_included_procedure_site</td><td>Has possibly included procedure site</td>
</tr>
<tr>
<td>has_possibly_included_route_of_administration</td><td>Has possibly included route of administration</td>
</tr>
<tr>
<td>has_possibly_included_surgical_extent</td><td>Has possibly included surgical extent</td>
</tr>
<tr>
<td>has_precise_active_ingredient</td><td>Has precise active ingredient</td>
</tr>
<tr>
<td>has_precise_ingredient</td><td>Has precise ingredient</td>
</tr>
<tr>
<td>has_precondition</td><td>Has precondition</td>
</tr>
<tr>
<td>has_presence_guidance</td><td>Has presence guidance</td>
</tr>
<tr>
<td>has_presentation_strength_denominator_unit</td><td>Has presentation strength denominator unit</td>
</tr>
<tr>
<td>has_presentation_strength_denominator_value</td><td>Has presentation strength denominator value</td>
</tr>
<tr>
<td>has_presentation_strength_numerator_unit</td><td>Has presentation strength numerator unit</td>
</tr>
<tr>
<td>has_presentation_strength_numerator_value</td><td>Has presentation strength numerator value</td>
</tr>
<tr>
<td>has_prev_name</td><td>Has previous name</td>
</tr>
<tr>
<td>has_prev_symbol</td><td>Has previous symbol</td>
</tr>
<tr>
<td>has_primary_segmental_supply</td><td>Has primary segmental supply</td>
</tr>
<tr>
<td>has_print_name</td><td>Has print name</td>
</tr>
<tr>
<td>has_priority</td><td>Has priority</td>
</tr>
<tr>
<td>has_procedure_context</td><td>Has procedure context</td>
</tr>
<tr>
<td>has_procedure_device</td><td>Has procedure device</td>
</tr>
<tr>
<td>has_procedure_morphology</td><td>Has procedure morphology</td>
</tr>
<tr>
<td>has_procedure_site</td><td>Has procedure site</td>
</tr>
<tr>
<td>has_process_duration</td><td>Has process duration</td>
</tr>
<tr>
<td>has_process_output</td><td>Has process output</td>
</tr>
<tr>
<td>has_product_characteristic</td><td>Has product characteristic</td>
</tr>
<tr>
<td>has_product_monograph_title</td><td>has product monograph title</td>
</tr>
<tr>
<td>has_projection</td><td>has projection</td>
</tr>
<tr>
<td>has_property_type</td><td>Has property type</td>
</tr>
<tr>
<td>has_property</td><td>Has property</td>
</tr>
<tr>
<td>has_quantified_form</td><td>Has quantified form</td>
</tr>
<tr>
<td>has_realization</td><td>Has realization</td>
</tr>
<tr>
<td>has_recipient_category</td><td>Has recipient category</td>
</tr>
<tr>
<td>has_regional_part</td><td>Has regional part</td>
</tr>
<tr>
<td>has_related_developmental_entity</td><td>has related developmental entity</td>
</tr>
<tr>
<td>has_related_factor</td><td>Has related factor</td>
</tr>
<tr>
<td>has_result</td><td>Has result</td>
</tr>
<tr>
<td>has_revision_status</td><td>Has revision status</td>
</tr>
<tr>
<td>has_risk_factor</td><td>Has risk factor</td>
</tr>
<tr>
<td>has_route_of_administration</td><td>Has route of administration</td>
</tr>
<tr>
<td>has_salt_form</td><td>has salt form</td>
</tr>
<tr>
<td>has_scale_type</td><td>Has scale type</td>
</tr>
<tr>
<td>has_scale</td><td>Has scale</td>
</tr>
<tr>
<td>has_secondary_segmental_supply</td><td>Has secondary segmental supply</td>
</tr>
<tr>
<td>has_segmental_composition</td><td>Has segmental composition</td>
</tr>
<tr>
<td>has_segmental_supply</td><td>Has segmental supply</td>
</tr>
<tr>
<td>has_seronet_authorized_value</td><td>has seronet authorized value</td>
</tr>
<tr>
<td>has_severity</td><td>Has severity</td>
</tr>
<tr>
<td>has_sign_or_symptom</td><td>Has sign or symptom</td>
</tr>
<tr>
<td>has_single_level_category</td><td>Has single level category</td>
</tr>
<tr>
<td>has_sort_version</td><td>Has sort version</td>
</tr>
<tr>
<td>has_specialty</td><td>Has specialty</td>
</tr>
<tr>
<td>has_specimen_procedure</td><td>Has specimen procedure</td>
</tr>
<tr>
<td>has_specimen_source_identity</td><td>Has specimen source identity</td>
</tr>
<tr>
<td>has_specimen_source_morphology</td><td>Has specimen source morphology</td>
</tr>
<tr>
<td>has_specimen_source_topography</td><td>Has specimen source topography</td>
</tr>
<tr>
<td>has_specimen_substance</td><td>Has specimen substance</td>
</tr>
<tr>
<td>has_specimen</td><td>Has specimen</td>
</tr>
<tr>
<td>has_stage</td><td>Has stage</td>
</tr>
<tr>
<td>has_state_of_matter</td><td>Has state of matter</td>
</tr>
<tr>
<td>has_structural_class</td><td>Structural class of</td>
</tr>
<tr>
<td>has_subject_of_information</td><td>Has subject of information</td>
</tr>
<tr>
<td>has_subject_relationship_context</td><td>Has subject relationship context</td>
</tr>
<tr>
<td>has_subject</td><td>Has subject</td>
</tr>
<tr>
<td>has_suffix</td><td>Has suffix</td>
</tr>
<tr>
<td>has_supersystem</td><td>Has supersystem</td>
</tr>
<tr>
<td>has_supported_concept_property</td><td>Has supported concept property</td>
</tr>
<tr>
<td>has_supported_concept_relationship</td><td>Has supported concept relationship</td>
</tr>
<tr>
<td>has_surface_characteristic</td><td>Has surface characteristic</td>
</tr>
<tr>
<td>has_surface_texture</td><td>Has surface texture</td>
</tr>
<tr>
<td>has_surgical_approach</td><td>has surgical approach</td>
</tr>
<tr>
<td>has_surgical_extent</td><td>Has surgical extent</td>
</tr>
<tr>
<td>has_system</td><td>Has system</td>
</tr>
<tr>
<td>has_target_population</td><td>Has target population</td>
</tr>
<tr>
<td>has_target</td><td>has target</td>
</tr>
<tr>
<td>has_technique</td><td>Has technique</td>
</tr>
<tr>
<td>has_temporal_context</td><td>Has temporal context</td>
</tr>
<tr>
<td>has_therapeutic_class</td><td>Has therapeutic class</td>
</tr>
<tr>
<td>has_time_aspect</td><td>Has time aspect</td>
</tr>
<tr>
<td>has_time_modifier</td><td>Has time modifier</td>
</tr>
<tr>
<td>has_timing_of</td><td>Has timing of</td>
</tr>
<tr>
<td>has_tradename</td><td>Has tradename</td>
</tr>
<tr>
<td>has_translation</td><td>Has translation</td>
</tr>
<tr>
<td>has_transliterated_form</td><td>Has transliterated form</td>
</tr>
<tr>
<td>has_tributary</td><td>Has tributary</td>
</tr>
<tr>
<td>has_unit_of_presentation</td><td>Has unit of presentation</td>
</tr>
<tr>
<td>has_units</td><td>Has units</td>
</tr>
<tr>
<td>has_venous_drainage</td><td>Has venous drainage</td>
</tr>
<tr>
<td>has_version</td><td>Has version</td>
</tr>
<tr>
<td>has_view_type</td><td>Has view type</td>
</tr>
<tr>
<td>homonym_for</td><td>Homonym for</td>
</tr>
<tr>
<td>homonym_of</td><td>Homonym of</td>
</tr>
<tr>
<td>human_disease_maps_to_eo_disease</td><td>human disease maps to eo disease</td>
</tr>
<tr>
<td>icd_asterisk</td><td>ICD asterisk</td>
</tr>
<tr>
<td>icd_dagger</td><td>ICD dagger</td>
</tr>
<tr>
<td>icdc_value_of</td><td>icdc value of</td>
</tr>
<tr>
<td>imaged_anatomy_has_procedure</td><td>imaged anatomy has procedure</td>
</tr>
<tr>
<td>inactive_ingredient_of</td><td>Inactive Ingredient Of</td>
</tr>
<tr>
<td>inc_parent_of</td><td>inc parent of</td>
</tr>
<tr>
<td>included_in</td><td>included_in</td>
</tr>
<tr>
<td>includes_sub_specimen</td><td>Includes sub specimen</td>
</tr>
<tr>
<td>includes</td><td>includes</td>
</tr>
<tr>
<td>indirect_device_of</td><td>Indirect device of</td>
</tr>
<tr>
<td>indirect_morphology_of</td><td>Indirect morphology of</td>
</tr>
<tr>
<td>indirect_procedure_site_of</td><td>Indirect procedure site of</td>
</tr>
<tr>
<td>induced_by</td><td>Induced by</td>
</tr>
<tr>
<td>induces</td><td>Induces</td>
</tr>
<tr>
<td>inferior_to</td><td>inferior to</td>
</tr>
<tr>
<td>inferolateral_to</td><td>inferolateral to</td>
</tr>
<tr>
<td>inferomedial_to</td><td>inferomedial to</td>
</tr>
<tr>
<td>ingredient_characteristic_of</td><td>Ingredient characteristic of</td>
</tr>
<tr>
<td>ingredient_of</td><td>Ingredient of</td>
</tr>
<tr>
<td>ingredient_qualitative_strength_of</td><td>Ingredient qualitative strength of</td>
</tr>
<tr>
<td>ingredients_of</td><td>Ingredients of</td>
</tr>
<tr>
<td>inherent_3d_shape_of</td><td>Inherent 3-D shape of</td>
</tr>
<tr>
<td>inherent_location_of</td><td>Inherent location of</td>
</tr>
<tr>
<td>inheres_in</td><td>Inheres in</td>
</tr>
<tr>
<td>inheritance_type_of</td><td>Inheritance type of</td>
</tr>
<tr>
<td>insertion_of</td><td>insertion of</td>
</tr>
<tr>
<td>instrumentation_of</td><td>Instrumentation of</td>
</tr>
<tr>
<td>intent_of</td><td>Intent of</td>
</tr>
<tr>
<td>internal_to</td><td>internal to</td>
</tr>
<tr>
<td>interpretation_of</td><td>Interpretation of</td>
</tr>
<tr>
<td>interprets</td><td>Interprets</td>
</tr>
<tr>
<td>inverse_during</td><td>Inverse during</td>
</tr>
<tr>
<td>inverse_ends_during</td><td>Inverse ends during</td>
</tr>
<tr>
<td>inverse_happens_during</td><td>Inverse happens during</td>
</tr>
<tr>
<td>inverse_isa</td><td>Inverse is a</td>
</tr>
<tr>
<td>inverse_relative_to</td><td>Inverse relative to</td>
</tr>
<tr>
<td>inverse_was_a</td><td>Inverse was a</td>
</tr>
<tr>
<td>is_abnormal_cell_of_disease</td><td>is abnormal cell of disease</td>
</tr>
<tr>
<td>is_abnormality_of_gene_product</td><td>is abnormality of gene product</td>
</tr>
<tr>
<td>is_abnormality_of_gene</td><td>is abnormality of gene</td>
</tr>
<tr>
<td>is_acc-aha_sars2_authorized_value_for_variable</td><td>is ACC-AHA SARS2 authorized value for variable</td>
</tr>
<tr>
<td>is_action_guidance_for</td><td>Is action guidance for</td>
</tr>
<tr>
<td>is_aggregation_view_of</td><td>Is aggregation view of</td>
</tr>
<tr>
<td>is_approach_guidance_for</td><td>Is approach guidance for</td>
</tr>
<tr>
<td>is_associated_anatomic_site_of</td><td>is associated anatomic site of</td>
</tr>
<tr>
<td>is_associated_anatomy_of_gene_product</td><td>is associated anatomy of gene product</td>
</tr>
<tr>
<td>is_associated_disease_of</td><td>is associated disease of</td>
</tr>
<tr>
<td>is_biochemical_function_of_gene_product</td><td>is biochemical function of gene product</td>
</tr>
<tr>
<td>is_chemical_classification_of_gene_product</td><td>is chemical classification of gene product</td>
</tr>
<tr>
<td>is_chromosomal_location_of_gene</td><td>is chromosomal location of gene</td>
</tr>
<tr>
<td>is_component_of_chemotherapy_regimen</td><td>is component of chemotherapy regimen</td>
</tr>
<tr>
<td>is_ctdc_value_of</td><td>is ctdc value of</td>
</tr>
<tr>
<td>is_cytogenetic_abnormality_of_disease</td><td>is cytogenetic abnormality of disease</td>
</tr>
<tr>
<td>is_dipg_dmg_authorized_value_for_variable</td><td>is DIPG DMG authorized value for variable</td>
</tr>
<tr>
<td>is_exam_for</td><td>Is exam for</td>
</tr>
<tr>
<td>is_finding_of_disease</td><td>is finding of disease</td>
</tr>
<tr>
<td>is_given_pharmaceutical_substance_for</td><td>Is given pharmaceutical substance for</td>
</tr>
<tr>
<td>is_grade_of_disease</td><td>is grade of disease</td>
</tr>
<tr>
<td>is_imaged_location_for</td><td>Is imaged location for</td>
</tr>
<tr>
<td>is_imaging_focus_of</td><td>Is imaging focus of</td>
</tr>
<tr>
<td>is_interpreted_by</td><td>Is interpreted by</td>
</tr>
<tr>
<td>is_lateral_anatomic_location_of</td><td>Is lateral anatomic location of</td>
</tr>
<tr>
<td>is_location_of_anatomic_structure</td><td>is location of anatomic structure</td>
</tr>
<tr>
<td>is_location_of_biological_process</td><td>is location of biological process</td>
</tr>
<tr>
<td>is_maneuver_type_for</td><td>Is maneuver type for</td>
</tr>
<tr>
<td>is_marked_by_gene_product</td><td>is marked by gene product</td>
</tr>
<tr>
<td>is_mechanism_of_action_of_chemical_or_drug</td><td>is mechanism of action of chemical or drug</td>
</tr>
<tr>
<td>is_metastatic_anatomic_site_of_disease</td><td>is metastatic anatomic site of disease</td>
</tr>
<tr>
<td>is_modality_subtype_for</td><td>Is modality subtype for</td>
</tr>
<tr>
<td>is_modality_type_for</td><td>Is modality type for</td>
</tr>
<tr>
<td>is_modification_of</td><td>Is modification of</td>
</tr>
<tr>
<td>is_molecular_abnormality_of_disease</td><td>is molecular abnormality of disease</td>
</tr>
<tr>
<td>is_normal_cell_origin_of_disease</td><td>is normal cell origin of disease</td>
</tr>
<tr>
<td>is_normal_tissue_origin_of_disease</td><td>is normal tissue origin of disease</td>
</tr>
<tr>
<td>is_not_abnormal_cell_of_disease</td><td>is not abnormal cell of disease</td>
</tr>
<tr>
<td>is_not_cytogenetic_abnormality_of_disease</td><td>is not cytogenetic abnormality of disease</td>
</tr>
<tr>
<td>is_not_finding_of_disease</td><td>is not finding of disease</td>
</tr>
<tr>
<td>is_not_metastatic_anatomic_site_of_disease</td><td>is not metastatic anatomic site of disease</td>
</tr>
<tr>
<td>is_not_molecular_abnormality_of_disease</td><td>is not molecular abnormality of disease</td>
</tr>
<tr>
<td>is_not_normal_cell_origin_of_disease</td><td>is not normal cell origin of disease</td>
</tr>
<tr>
<td>is_not_normal_tissue_origin_of_disease</td><td>is not normal tissue origin of disease</td>
</tr>
<tr>
<td>is_not_primary_anatomic_site_of_disease</td><td>is not primary anatomic site of disease</td>
</tr>
<tr>
<td>is_object_guidance_for</td><td>Is object guidance for</td>
</tr>
<tr>
<td>is_ooro_pc_value_of</td><td>is OORO PC value of</td>
</tr>
<tr>
<td>is_organism_source_of_gene_product</td><td>is organism source of gene product</td>
</tr>
<tr>
<td>is_paired_with_value_set</td><td>is paired with value set</td>
</tr>
<tr>
<td>is_pcdc_all_authorized_value_for_variable</td><td>is PCDC ALL authorized value for variable</td>
</tr>
<tr>
<td>is_pcdc_aml_authorized_value_for_variable</td><td>is PCDC AML authorized value for variable</td>
</tr>
<tr>
<td>is_pcdc_ews_authorized_value_for_variable</td><td>is PCDC EWS authorized value for variable</td>
</tr>
<tr>
<td>is_pcdc_gct_authorized_value_for_variable</td><td>is PCDC GCT authorized value for variable</td>
</tr>
<tr>
<td>is_pcdc_hl_authorized_value_for_variable</td><td>is PCDC HL authorized value for variable</td>
</tr>
<tr>
<td>is_pcdc_os_authorized_value_for_variable</td><td>is PCDC OS authorized value for variable</td>
</tr>
<tr>
<td>is_pharmaceutical_route_for</td><td>Is pharmaceutical route for</td>
</tr>
<tr>
<td>is_physical_location_of_gene</td><td>is physical location of gene</td>
</tr>
<tr>
<td>is_physiologic_effect_of_chemical_or_drug</td><td>is physiologic effect of chemical or drug</td>
</tr>
<tr>
<td>is_presence_guidance_for</td><td>Is presence guidance for</td>
</tr>
<tr>
<td>is_presence_of_lateral_location</td><td>Is presence of lateral location</td>
</tr>
<tr>
<td>is_primary_anatomic_site_of_disease</td><td>is primary anatomic site of disease</td>
</tr>
<tr>
<td>is_property_or_attribute_of_eo_disease</td><td>is property or attribute of eo disease</td>
</tr>
<tr>
<td>is_qualified_by</td><td>is qualified by</td>
</tr>
<tr>
<td>is_related_to_endogenous_product</td><td>Is Related To Endogenous Product</td>
</tr>
<tr>
<td>is_seronet_authorized_value_for_variable</td><td>is seronet authorized value for variable</td>
</tr>
<tr>
<td>is_stage_of_disease</td><td>is stage of disease</td>
</tr>
<tr>
<td>is_sterile</td><td>Is sterile</td>
</tr>
<tr>
<td>is_structural_domain_or_motif_of_gene_product</td><td>is structural domain or motif of gene product</td>
</tr>
<tr>
<td>is_subject_of</td><td>Is subject of</td>
</tr>
<tr>
<td>is_target</td><td>is target</td>
</tr>
<tr>
<td>is_timing_for</td><td>Is timing for</td>
</tr>
<tr>
<td>is_value_for_gdc_property</td><td>is value for gdc property</td>
</tr>
<tr>
<td>is_view_type_for</td><td>Is view type for</td>
</tr>
<tr>
<td>isa</td><td>Is a</td>
</tr>
<tr>
<td>kind_is_domain_of</td><td>kind is domain of</td>
</tr>
<tr>
<td>kind_is_range_of</td><td>kind is range of</td>
</tr>
<tr>
<td>lab_number_of</td><td>Lab number</td>
</tr>
<tr>
<td>larger_than</td><td>Larger Than</td>
</tr>
<tr>
<td>lateral_to</td><td>lateral to</td>
</tr>
<tr>
<td>laterality_of</td><td>Laterality of</td>
</tr>
<tr>
<td>left_lateral_to</td><td>left lateral to</td>
</tr>
<tr>
<td>left_medial_to</td><td>left medial to</td>
</tr>
<tr>
<td>life_circumstance_of</td><td>Life circumstance of</td>
</tr>
<tr>
<td>locale_of_excluded</td><td>Locale of excluded</td>
</tr>
<tr>
<td>locale_of</td><td>Locale of</td>
</tr>
<tr>
<td>location_of</td><td>Location of</td>
</tr>
<tr>
<td>loinc_number_of</td><td>Loinc number of</td>
</tr>
<tr>
<td>lymphatic_drainage_of</td><td>Lymphatic drainage of</td>
</tr>
<tr>
<td>manifestation_of</td><td>Manifestation of</td>
</tr>
<tr>
<td>manufactured_by</td><td>manufactured by</td>
</tr>
<tr>
<td>manufactures</td><td>manufactures</td>
</tr>
<tr>
<td>mapped_from</td><td>Mapped from</td>
</tr>
<tr>
<td>mapped_to</td><td>Mapped to</td>
</tr>
<tr>
<td>mapping_qualifier_of</td><td>Mapping qualifier of</td>
</tr>
<tr>
<td>matures_from</td><td>matures from</td>
</tr>
<tr>
<td>matures_into</td><td>matures into</td>
</tr>
<tr>
<td>may_be_abnormal_cell_of_disease</td><td>may be abnormal cell of disease</td>
</tr>
<tr>
<td>may_be_associated_disease_of_disease</td><td>may be associated disease of disease</td>
</tr>
<tr>
<td>may_be_cytogenetic_abnormality_of_disease</td><td>may be cytogenetic abnormality of disease</td>
</tr>
<tr>
<td>may_be_diagnosed_by</td><td>May be diagnosed by</td>
</tr>
<tr>
<td>may_be_finding_of_disease</td><td>may be finding of disease</td>
</tr>
<tr>
<td>may_be_molecular_abnormality_of_disease</td><td>may be molecular abnormality of disease</td>
</tr>
<tr>
<td>may_be_normal_cell_origin_of_disease</td><td>may be normal cell origin of disease</td>
</tr>
<tr>
<td>may_be_normal_tissue_origin_of_disease</td><td>may be normal tissue origin of disease</td>
</tr>
<tr>
<td>may_be_prevented_by</td><td>May be prevented by</td>
</tr>
<tr>
<td>may_be_qualified_by</td><td>May be qualified by</td>
</tr>
<tr>
<td>may_be_treated_by</td><td>May be treated by</td>
</tr>
<tr>
<td>may_diagnose</td><td>May diagnose</td>
</tr>
<tr>
<td>may_inhibit_effect_of</td><td>May inhibit effect of</td>
</tr>
<tr>
<td>may_prevent</td><td>May prevent</td>
</tr>
<tr>
<td>may_qualify</td><td>May qualify</td>
</tr>
<tr>
<td>may_treat</td><td>May treat</td>
</tr>
<tr>
<td>measured_by</td><td>Measured by</td>
</tr>
<tr>
<td>measured_component_of</td><td>Measured component of</td>
</tr>
<tr>
<td>measurement_method_of</td><td>Measurement method of</td>
</tr>
<tr>
<td>measures</td><td>Measures</td>
</tr>
<tr>
<td>mechanism_of_action_of</td><td>Mechanism of action of</td>
</tr>
<tr>
<td>medial_to</td><td>medial to</td>
</tr>
<tr>
<td>member_of</td><td>Member of</td>
</tr>
<tr>
<td>merges_with</td><td>merges with</td>
</tr>
<tr>
<td>metabolic_site_of</td><td>Metabolic site of</td>
</tr>
<tr>
<td>method_of_excluded</td><td>Method of excluded</td>
</tr>
<tr>
<td>method_of_possibly_included</td><td>Method of possibly included</td>
</tr>
<tr>
<td>method_of</td><td>Method of</td>
</tr>
<tr>
<td>modified_by</td><td>Modified by</td>
</tr>
<tr>
<td>modifies</td><td>Modifies</td>
</tr>
<tr>
<td>molecular_abnormality_involves_gene</td><td>molecular abnormality involves gene</td>
</tr>
<tr>
<td>moved_from</td><td>Moved from</td>
</tr>
<tr>
<td>moved_to</td><td>Moved to</td>
</tr>
<tr>
<td>mth_british_form_of</td><td>MTH British form of</td>
</tr>
<tr>
<td>mth_expanded_form_of</td><td>MTH expanded form of</td>
</tr>
<tr>
<td>mth_has_british_form</td><td>MTH has British form</td>
</tr>
<tr>
<td>mth_has_expanded_form</td><td>MTH has expanded form</td>
</tr>
<tr>
<td>mth_has_plain_text_form</td><td>Has plain text form, Metathesaurus-asserted</td>
</tr>
<tr>
<td>mth_has_xml_form</td><td>Has XML form, Metathesaurus-asserted</td>
</tr>
<tr>
<td>mth_plain_text_form_of</td><td>Plain text form of, Metathesaurus-asserted</td>
</tr>
<tr>
<td>mth_xml_form_of</td><td>XML form of, Metathesaurus-asserted</td>
</tr>
<tr>
<td>multiply_mapped_from</td><td>Multiply mapped from</td>
</tr>
<tr>
<td>multiply_mapped_to</td><td>Multiply mapped to</td>
</tr>
<tr>
<td>negatively_regulated_by</td><td>Negatively regulated by</td>
</tr>
<tr>
<td>negatively_regulates</td><td>Negatively regulates</td>
</tr>
<tr>
<td>neoplasm_has_special_category</td><td>neoplasm has special category</td>
</tr>
<tr>
<td>nerve_supply_of</td><td>Nerve supply of</td>
</tr>
<tr>
<td>noun_form_of</td><td>Noun form of</td>
</tr>
<tr>
<td>occurs_after</td><td>Occurs after</td>
</tr>
<tr>
<td>occurs_before</td><td>Occurs before</td>
</tr>
<tr>
<td>occurs_in</td><td>Occurs in</td>
</tr>
<tr>
<td>onset_of</td><td>Onset of</td>
</tr>
<tr>
<td>organism_has_gene</td><td>organism has gene</td>
</tr>
<tr>
<td>origin_of</td><td>Origin of</td>
</tr>
<tr>
<td>other_mapped_from</td><td>Other mapped from</td>
</tr>
<tr>
<td>other_mapped_to</td><td>Other mapped to</td>
</tr>
<tr>
<td>owning_affiliate_of</td><td>Owning affiliate of</td>
</tr>
<tr>
<td>owning_section_of</td><td>Owning section of</td>
</tr>
<tr>
<td>owning_subsection_of</td><td>Owning subsection of</td>
</tr>
<tr>
<td>panel_element_of_possibly_included</td><td>CPT laboratory panel test member is possibly included element of laboratory code</td>
</tr>
<tr>
<td>panel_element_of</td><td>CPT laboratory panel test member is element of laboratory code</td>
</tr>
<tr>
<td>parent_group_of</td><td>Parent group of</td>
</tr>
<tr>
<td>parent_of</td><td>Parent of</td>
</tr>
<tr>
<td>part_anatomy_structure_of</td><td>Part anatomy structure of</td>
</tr>
<tr>
<td>part_of</td><td>Part of</td>
</tr>
<tr>
<td>part_referred_to_by</td><td>Part referred to by</td>
</tr>
<tr>
<td>partially_equivalent_to</td><td>Partially equivalent to</td>
</tr>
<tr>
<td>partially_excised_anatomy_has_procedure</td><td>partially excised anatomy has procedure</td>
</tr>
<tr>
<td>partially_excised_anatomy_may_have_procedure</td><td>partially excised anatomy may have procedure</td>
</tr>
<tr>
<td>pathogenesis_of_disease_involves_gene</td><td>pathogenesis of disease involves gene</td>
</tr>
<tr>
<td>pathological_process_of</td><td>Pathological process of</td>
</tr>
<tr>
<td>pathology_of_excluded</td><td>Pathology of excluded</td>
</tr>
<tr>
<td>pathology_of_possibly_included</td><td>Pathology of possibly included</td>
</tr>
<tr>
<td>pathology_of</td><td>Pathology of</td>
</tr>
<tr>
<td>pathway_has_gene_element</td><td>pathway has gene element</td>
</tr>
<tr>
<td>patient_type_of_excluded</td><td>Patient type of excluded</td>
</tr>
<tr>
<td>patient_type_of_possibly_included</td><td>Patient type of possibly included</td>
</tr>
<tr>
<td>patient_type_of</td><td>Patient type of</td>
</tr>
<tr>
<td>pcdc_data_type_of</td><td>pcdc data type of</td>
</tr>
<tr>
<td>permuted_term_of</td><td>Permuted term of</td>
</tr>
<tr>
<td>pharmaceutical_administration_method_of</td><td>pharmaceutical administration method of</td>
</tr>
<tr>
<td>pharmaceutical_basic_dose_form_of</td><td>pharmaceutical basic dose form of</td>
</tr>
<tr>
<td>pharmaceutical_intended_site_of</td><td>pharmaceutical intended site of</td>
</tr>
<tr>
<td>pharmaceutical_release_characteristics_of</td><td>pharmaceutical release characteristics of</td>
</tr>
<tr>
<td>pharmaceutical_state_of_matter_of</td><td>pharmaceutical state of matter of</td>
</tr>
<tr>
<td>pharmaceutical_transformation_of</td><td>pharmaceutical transformation_of</td>
</tr>
<tr>
<td>pharmacokinetics_of</td><td>Pharmacokinetics of</td>
</tr>
<tr>
<td>phenotype_of</td><td>Phenotype of</td>
</tr>
<tr>
<td>physiologic_effect_of</td><td>Physiologic effect of</td>
</tr>
<tr>
<td>physiologic_state_of</td><td>Physiologic state of</td>
</tr>
<tr>
<td>place_traveled_from</td><td>Place traveled from</td>
</tr>
<tr>
<td>place_traveled_to</td><td>Place traveled to or resided in</td>
</tr>
<tr>
<td>plays_role</td><td>Plays role</td>
</tr>
<tr>
<td>population_at_risk_for</td><td>Population at risk for</td>
</tr>
<tr>
<td>positively_regulated_by</td><td>Positively regulated by</td>
</tr>
<tr>
<td>positively_regulates</td><td>Positively regulates</td>
</tr>
<tr>
<td>possibly_equivalent_to</td><td>Possibly equivalent to</td>
</tr>
<tr>
<td>possibly_replaced_by</td><td>Possibly replaced by</td>
</tr>
<tr>
<td>possibly_replaces</td><td>Possibly replaces</td>
</tr>
<tr>
<td>posterior_to</td><td>posterior to</td>
</tr>
<tr>
<td>posteroinferior_to</td><td>posteroinferior to</td>
</tr>
<tr>
<td>posterolateral_to</td><td>posterolateral to</td>
</tr>
<tr>
<td>posteromedial_to</td><td>posteromedial to</td>
</tr>
<tr>
<td>posterosuperior_to</td><td>posterosuperior to</td>
</tr>
<tr>
<td>precise_active_ingredient_of</td><td>Precise active ingredient of</td>
</tr>
<tr>
<td>precise_ingredient_of</td><td>Precise ingredient of</td>
</tr>
<tr>
<td>precondition_of</td><td>Precondition of</td>
</tr>
<tr>
<td>presentation_strength_denominator_unit_of</td><td>Presentation strength denominator unit of</td>
</tr>
<tr>
<td>presentation_strength_denominator_value_of</td><td>Presentation strength denominator value of</td>
</tr>
<tr>
<td>presentation_strength_numerator_unit_of</td><td>Presentation strength numerator unit of</td>
</tr>
<tr>
<td>presentation_strength_numerator_value_of</td><td>Presentation strength numerator value of</td>
</tr>
<tr>
<td>prev_name_of</td><td>Previous name of</td>
</tr>
<tr>
<td>prev_symbol_of</td><td>Previous symbol of</td>
</tr>
<tr>
<td>primary_mapped_from</td><td>Primary mapped from</td>
</tr>
<tr>
<td>primary_mapped_to</td><td>Primary mapped to</td>
</tr>
<tr>
<td>primary_segmental_supply_of</td><td>Primary segmental supply of</td>
</tr>
<tr>
<td>print_name_of</td><td>Print name of</td>
</tr>
<tr>
<td>priority_of</td><td>Priority of</td>
</tr>
<tr>
<td>procedure_context_of</td><td>Procedure context of</td>
</tr>
<tr>
<td>procedure_device_of_excluded</td><td>Procedure device of excluded</td>
</tr>
<tr>
<td>procedure_device_of_possibly_included</td><td>Procedure device of possibly included</td>
</tr>
<tr>
<td>procedure_device_of</td><td>Procedure device of</td>
</tr>
<tr>
<td>procedure_has_completely_excised_anatomy</td><td>procedure has completely excised anatomy</td>
</tr>
<tr>
<td>procedure_has_excised_anatomy</td><td>procedure has excised anatomy</td>
</tr>
<tr>
<td>procedure_has_imaged_anatomy</td><td>procedure has imaged anatomy</td>
</tr>
<tr>
<td>procedure_has_partially_excised_anatomy</td><td>procedure has partially excised anatomy</td>
</tr>
<tr>
<td>procedure_has_target_anatomy</td><td>procedure has target anatomy</td>
</tr>
<tr>
<td>procedure_may_have_completely_excised_anatomy</td><td>procedure may have completely excised anatomy</td>
</tr>
<tr>
<td>procedure_may_have_excised_anatomy</td><td>procedure may have excised anatomy</td>
</tr>
<tr>
<td>procedure_may_have_partially_excised_anatomy</td><td>procedure may have partially excised anatomy</td>
</tr>
<tr>
<td>procedure_morphology_of</td><td>Procedure morphology of</td>
</tr>
<tr>
<td>procedure_site_of_excluded</td><td>Procedure site of excluded</td>
</tr>
<tr>
<td>procedure_site_of_possibly_included</td><td>Procedure site of possibly included</td>
</tr>
<tr>
<td>procedure_site_of</td><td>Procedure site of</td>
</tr>
<tr>
<td>process_acts_on</td><td>Process acts on</td>
</tr>
<tr>
<td>process_altered_by_allele</td><td>process altered by allele</td>
</tr>
<tr>
<td>process_duration_of</td><td>Process duration of</td>
</tr>
<tr>
<td>process_extends_to</td><td>Process extends to</td>
</tr>
<tr>
<td>process_includes_biological_process</td><td>process includes biological process</td>
</tr>
<tr>
<td>process_initiates_biological_process</td><td>process initiates biological process</td>
</tr>
<tr>
<td>process_involves_gene</td><td>process involves gene</td>
</tr>
<tr>
<td>process_output_of</td><td>Process output of</td>
</tr>
<tr>
<td>product_characteristic_of</td><td>Product characteristic of</td>
</tr>
<tr>
<td>product_monograph_title_of</td><td>product monograph title of</td>
</tr>
<tr>
<td>projects_from</td><td>projects from</td>
</tr>
<tr>
<td>projects_to</td><td>projects to</td>
</tr>
<tr>
<td>property_of</td><td>Property of</td>
</tr>
<tr>
<td>property_type_of</td><td>Property type of</td>
</tr>
<tr>
<td>proximal_to</td><td>proximal to</td>
</tr>
<tr>
<td>qualifier_applies_to</td><td>qualifier applies to</td>
</tr>
<tr>
<td>quantified_form_of</td><td>Quantified form of</td>
</tr>
<tr>
<td>realization_of</td><td>Realization of</td>
</tr>
<tr>
<td>receives_attachment_from</td><td>receives attachment from</td>
</tr>
<tr>
<td>receives_drainage_from</td><td>receives drainage from</td>
</tr>
<tr>
<td>receives_input_from</td><td>Receives input from</td>
</tr>
<tr>
<td>receives_projection</td><td>receives projection</td>
</tr>
<tr>
<td>recipient_category_of</td><td>Recipient category of</td>
</tr>
<tr>
<td>referred_to_by</td><td>referred to by</td>
</tr>
<tr>
<td>refers_to</td><td>refers to</td>
</tr>
<tr>
<td>reformulated_to</td><td>Reformulated to</td>
</tr>
<tr>
<td>reformulation_of</td><td>Reformulation of</td>
</tr>
<tr>
<td>regimen_has_accepted_use_for_disease</td><td>Regimen Has Accepted Use For Disease</td>
</tr>
<tr>
<td>regional_part_of</td><td>Regional part of</td>
</tr>
<tr>
<td>regulated_by</td><td>Regulated by</td>
</tr>
<tr>
<td>regulates</td><td>Regulates</td>
</tr>
<tr>
<td>related_developmental_entity_of</td><td>related developmental entity of</td>
</tr>
<tr>
<td>related_factor_of</td><td>Related factor of</td>
</tr>
<tr>
<td>related_object</td><td>related object</td>
</tr>
<tr>
<td>related_part</td><td>related part</td>
</tr>
<tr>
<td>related_to_genetic_biomarker</td><td>related to genetic biomarker</td>
</tr>
<tr>
<td>related_to</td><td>Is Related to</td>
</tr>
<tr>
<td>relative_to_part_of</td><td>Relative to part of</td>
</tr>
<tr>
<td>relative_to</td><td>Relative to</td>
</tr>
<tr>
<td>replaced_by</td><td>Replaced by</td>
</tr>
<tr>
<td>replaces</td><td>Replaces</td>
</tr>
<tr>
<td>result_of</td><td>Result of</td>
</tr>
<tr>
<td>revision_status_of</td><td>Revision status of</td>
</tr>
<tr>
<td>right_lateral_to</td><td>right lateral to</td>
</tr>
<tr>
<td>right_medial_to</td><td>right medial to</td>
</tr>
<tr>
<td>risk_factor_of</td><td>Risk factor of</td>
</tr>
<tr>
<td>role_has_domain</td><td>role has domain</td>
</tr>
<tr>
<td>role_has_parent</td><td>role has parent</td>
</tr>
<tr>
<td>role_has_range</td><td>role has range</td>
</tr>
<tr>
<td>role_is_parent_of</td><td>role is parent of</td>
</tr>
<tr>
<td>role_played_by</td><td>Role played by</td>
</tr>
<tr>
<td>route_of_administration_of_excluded</td><td>Route of administration of excluded</td>
</tr>
<tr>
<td>route_of_administration_of_possibly_included</td><td>Route of administration of possibly included</td>
</tr>
<tr>
<td>route_of_administration_of</td><td>Route of administration of</td>
</tr>
<tr>
<td>same_as</td><td>Same as</td>
</tr>
<tr>
<td>scale_of</td><td>Scale of</td>
</tr>
<tr>
<td>scale_type_of</td><td>Scale type of</td>
</tr>
<tr>
<td>secondary_segmental_supply_of</td><td>Secondary segmental supply of</td>
</tr>
<tr>
<td>see_from</td><td>See from</td>
</tr>
<tr>
<td>see</td><td>See</td>
</tr>
<tr>
<td>segmental_composition_of</td><td>Segmental composition of</td>
</tr>
<tr>
<td>segmental_supply_of</td><td>Segmental supply of</td>
</tr>
<tr>
<td>sends_output_to</td><td>Sends output to</td>
</tr>
<tr>
<td>severity_of</td><td>Severity of</td>
</tr>
<tr>
<td>sign_or_symptom_of</td><td>Sign or symptom of</td>
</tr>
<tr>
<td>site_of_metabolism</td><td>Site of metabolism</td>
</tr>
<tr>
<td>smaller_than</td><td>Smaller Than</td>
</tr>
<tr>
<td>sort_version_of</td><td>Sort version of</td>
</tr>
<tr>
<td>special_category_includes_neoplasm</td><td>special category includes neoplasm</td>
</tr>
<tr>
<td>specialty_of</td><td>Specialty of</td>
</tr>
<tr>
<td>specimen_of_excluded</td><td>Specimen of excluded</td>
</tr>
<tr>
<td>specimen_of</td><td>Specimen of</td>
</tr>
<tr>
<td>specimen_procedure_of</td><td>Specimen procedure of</td>
</tr>
<tr>
<td>specimen_source_identity_of</td><td>Specimen source identity of</td>
</tr>
<tr>
<td>specimen_source_morphology_of</td><td>Specimen source morphology of</td>
</tr>
<tr>
<td>specimen_source_topography_of</td><td>specimen source topography of</td>
</tr>
<tr>
<td>specimen_substance_of</td><td>Specimen substance of</td>
</tr>
<tr>
<td>ssc</td><td>scc</td>
</tr>
<tr>
<td>stage_of</td><td>Stage of</td>
</tr>
<tr>
<td>state_of_matter_of</td><td>State of matter of</td>
</tr>
<tr>
<td>structural_class_of</td><td>Has structural class</td>
</tr>
<tr>
<td>sub_specimen_included_by</td><td>Sub specimen included by</td>
</tr>
<tr>
<td>subject_of_information_of</td><td>Subject of information of</td>
</tr>
<tr>
<td>subject_relationship_context_of</td><td>Subject relationship context of</td>
</tr>
<tr>
<td>subset_includes_concept</td><td>subset includes concept</td>
</tr>
<tr>
<td>substance_used_by_excluded</td><td>Substance used by excluded</td>
</tr>
<tr>
<td>substance_used_by_possibly_included</td><td>Substance used by possibly included</td>
</tr>
<tr>
<td>substance_used_by</td><td>Substance used by</td>
</tr>
<tr>
<td>suffix_of</td><td>Suffix of</td>
</tr>
<tr>
<td>superior_to</td><td>superior to</td>
</tr>
<tr>
<td>superolateral_to</td><td>superolateral to</td>
</tr>
<tr>
<td>superomedial_to</td><td>superomedial to</td>
</tr>
<tr>
<td>supersystem_of</td><td>Supersystem of</td>
</tr>
<tr>
<td>supported_concept_property_in</td><td>Supported concept property in</td>
</tr>
<tr>
<td>supported_concept_relationship_in</td><td>Supported concept relationship in</td>
</tr>
<tr>
<td>surface_characteristic_of</td><td>Surface characteristic of</td>
</tr>
<tr>
<td>surface_texture_of</td><td>Surface texture of</td>
</tr>
<tr>
<td>surgical_approach_of</td><td>surgical approach of</td>
</tr>
<tr>
<td>surgical_extent_of_excluded</td><td>Surgical extent of excluded</td>
</tr>
<tr>
<td>surgical_extent_of_possibly_included</td><td>Surgical extent of possibly included</td>
</tr>
<tr>
<td>surgical_extent_of</td><td>Surgical extent of</td>
</tr>
<tr>
<td>surrounded_by</td><td>Surrounded by</td>
</tr>
<tr>
<td>surrounds</td><td>Surrounds</td>
</tr>
<tr>
<td>system_of</td><td>System of</td>
</tr>
<tr>
<td>target_anatomy_has_procedure</td><td>target anatomy has procedure</td>
</tr>
<tr>
<td>target_population_of</td><td>Target population of</td>
</tr>
<tr>
<td>technique_of</td><td>Technique of</td>
</tr>
<tr>
<td>temporal_context_of</td><td>Temporal context of</td>
</tr>
<tr>
<td>temporally_followed_by</td><td>Temporally followed by</td>
</tr>
<tr>
<td>temporally_follows</td><td>Temporally follows</td>
</tr>
<tr>
<td>temporally_related_to</td><td>Temporally related to</td>
</tr>
<tr>
<td>therapeutic_class_of</td><td>Therapeutic class of</td>
</tr>
<tr>
<td>time_aspect_of</td><td>Time aspect of</td>
</tr>
<tr>
<td>time_modifier_of</td><td>Time modifier of</td>
</tr>
<tr>
<td>tissue_is_expression_site_of_gene_product</td><td>tissue is expression site of gene product</td>
</tr>
<tr>
<td>towards</td><td>Towards</td>
</tr>
<tr>
<td>tradename_of</td><td>Tradename of</td>
</tr>
<tr>
<td>transforms_from</td><td>transforms from</td>
</tr>
<tr>
<td>transforms_into</td><td>transforms into</td>
</tr>
<tr>
<td>translation_of</td><td>Translation of</td>
</tr>
<tr>
<td>transliterated_form_of</td><td>transliterated_form_of</td>
</tr>
<tr>
<td>treated_by</td><td>Treated by</td>
</tr>
<tr>
<td>treats</td><td>Treats</td>
</tr>
<tr>
<td>tributary_of</td><td>Tributary of</td>
</tr>
<tr>
<td>uniquely_mapped_from</td><td>Uniquely mapped from</td>
</tr>
<tr>
<td>uniquely_mapped_to</td><td>Uniquely mapped to</td>
</tr>
<tr>
<td>unit_of_presentation_of</td><td>Unit of presentation of</td>
</tr>
<tr>
<td>units_of</td><td>Units of</td>
</tr>
<tr>
<td>used_by</td><td>Used by</td>
</tr>
<tr>
<td>used_for</td><td>Used for</td>
</tr>
<tr>
<td>uses_access_device</td><td>Uses access device</td>
</tr>
<tr>
<td>uses_device</td><td>Uses device</td>
</tr>
<tr>
<td>uses_energy</td><td>Uses energy</td>
</tr>
<tr>
<td>uses_excluded_substance</td><td>Uses excluded substance</td>
</tr>
<tr>
<td>uses_possibly_included_substance</td><td>Uses possibly included substance</td>
</tr>
<tr>
<td>uses_substance</td><td>Uses substance</td>
</tr>
<tr>
<td>uses</td><td>Uses</td>
</tr>
<tr>
<td>use</td><td>Use</td>
</tr>
<tr>
<td>value_set_is_paired_with</td><td>value set is paired with</td>
</tr>
<tr>
<td>venous_drainage_of</td><td>Venous drainage of</td>
</tr>
<tr>
<td>version_of</td><td>Version of</td>
</tr>
<tr>
<td>was_a</td><td>Was a</td>
</tr>
<tr>
<td>wound_has_communication_with</td><td>Wound has communication with</td>
</tr>
<tr>
<td></td><td>Empty relationship attribute</td>
</tr>
<tr>
<td>0</td><td>No additional restrictions; general terms of the license agreement apply.</td>
</tr>
<tr>
<td>1</td><td>General terms + additional restrictions in category 12.1</td>
</tr>
<tr>
<td>2</td><td>General terms + additional restrictions in category 12.2</td>
</tr>
<tr>
<td>3</td><td>General terms + additional restrictions in category 12.3</td>
</tr>
<tr>
<td>4</td><td>General terms + additional restrictions in category 12.4</td>
</tr>
<tr>
<td>9</td><td>General terms + SNOMED CT Affiliate License in Appendix 2</td>
</tr>
<tr>
<td>PF</td><td>Preferred form of term </td>
</tr>
<tr>
<td>VCW</td><td>Case and word-order variant of the preferred form</td>
</tr>
<tr>
<td>VC</td><td>Case variant of the preferred form</td>
</tr>
<tr>
<td>VO</td><td>Variant of the preferred form</td>
</tr>
<tr>
<td>VW</td><td>Word-order variant of the preferred form</td>
</tr>
<tr>
<td>AUI</td><td>Atom identifier</td>
</tr>
<tr>
<td>CODE</td><td>Unique Identifier or code for string in source</td>
</tr>
<tr>
<td>CUI</td><td>Concept unique identifier</td>
</tr>
<tr>
<td>RUI</td><td>Relationship identifier</td>
</tr>
<tr>
<td>SCUI</td><td>Source asserted concept unique identifier</td>
</tr>
<tr>
<td>SDUI</td><td>Source asserted descriptor identifier</td>
</tr>
<tr>
<td>AUI</td><td>Atom identifier</td>
</tr>
<tr>
<td>CODE</td><td>Unique Identifier or code for string in source</td>
</tr>
<tr>
<td>CUI</td><td>Concept unique identifier</td>
</tr>
<tr>
<td>SCUI</td><td>Source asserted concept unique identifier</td>
</tr>
<tr>
<td>SDUI</td><td>Source asserted descriptor identifier</td>
</tr>
<tr>
<td>AUI</td><td>Atom identifier</td>
</tr>
<tr>
<td>CODE</td><td>Unique Identifier or code for string in source</td>
</tr>
<tr>
<td>CUI</td><td>Concept unique identifier</td>
</tr>
<tr>
<td>SCUI</td><td>Source asserted concept unique identifier</td>
</tr>
<tr>
<td>SDUI</td><td>Source asserted descriptor identifier</td>
</tr>
<tr>
<td>E</td><td>Suppressible due to editor decision</td>
</tr>
<tr>
<td>N</td><td>Not suppressible</td>
</tr>
<tr>
<td>O</td><td>Obsolete, SAB,TTY may be independently suppressible</td>
</tr>
<tr>
<td>Y</td><td>Suppressible due to SAB,TTY</td>
</tr>
<tr>
<td></td><td>Suppressibility not yet assigned by the UMLS</td>
</tr>
<tr>
<td>BOOLEAN_EXPRESSION_CODE</td><td>Boolean expression of CODE values</td>
</tr>
<tr>
<td>BOOLEAN_EXPRESSION_SDUI</td><td>Boolean expression of SDUI values</td>
</tr>
<tr>
<td>BOOLEAN_EXPRESSION_STR</td><td>Boolean expression of STR values</td>
</tr>
<tr>
<td>CODE</td><td>Unique Identifier or code for string in source</td>
</tr>
<tr>
<td>SCUI</td><td>Source asserted concept unique identifier</td>
</tr>
<tr>
<td>SDUI</td><td>Source asserted descriptor identifier</td>
</tr>
<tr>
<td></td><td>No type, used for XR mappings.</td>
</tr>
<tr>
<td>P</td><td>Preferred LUI of the CUI</td>
</tr>
<tr>
<td>S</td><td>Non-Preferred LUI of the CUI</td>
</tr>
<tr>
<td>p</td><td>Preferred LUI of the CUI, suppressible (only used in ORF MRCON)</td>
</tr>
<tr>
<td>s</td><td>Non-Preferred LUI of the CUI, suppressible (only used in ORF MRCON)</td>
</tr>
<tr>
<td>AA</td><td>Attribute type abbreviation</td>
</tr>
<tr>
<td>AB</td><td>Abbreviation in any source vocabulary</td>
</tr>
<tr>
<td>ACR</td><td>Acronym</td>
</tr>
<tr>
<td>AC</td><td>Activities</td>
</tr>
<tr>
<td>AD</td><td>Adjective</td>
</tr>
<tr>
<td>AM</td><td>Short form of modifier</td>
</tr>
<tr>
<td>AS</td><td>Attribute type synonym</td>
</tr>
<tr>
<td>AT</td><td>Attribute type</td>
</tr>
<tr>
<td>BD</td><td>Fully-specified drug brand name that can be prescribed</td>
</tr>
<tr>
<td>BN</td><td>Fully-specified drug brand name that can not be prescribed</td>
</tr>
<tr>
<td>BPCK</td><td>Branded Drug Delivery Device</td>
</tr>
<tr>
<td>BR</td><td>Binding realm</td>
</tr>
<tr>
<td>CA2</td><td>ISO 3166-1 standard country code in alpha-2 (two-letter) format</td>
</tr>
<tr>
<td>CA3</td><td>ISO 3166-1 standard country code in alpha-3 (three-letter) format</td>
</tr>
<tr>
<td>CCN</td><td>Chemical code name</td>
</tr>
<tr>
<td>CCS</td><td>FIPS 10-4 country code</td>
</tr>
<tr>
<td>CC</td><td>Trimmed ICPC component process</td>
</tr>
<tr>
<td>CDA</td><td>Clinical drug name in abbreviated format</td>
</tr>
<tr>
<td>CDC</td><td>Clinical drug name in concatenated format (NDDF)</td>
</tr>
<tr>
<td>CDD</td><td>Clinical drug name in delimited format</td>
</tr>
<tr>
<td>CDO</td><td>Concept domain</td>
</tr>
<tr>
<td>CD</td><td>Clinical Drug</td>
</tr>
<tr>
<td>CE</td><td>Entry term for a Supplementary Concept</td>
</tr>
<tr>
<td>CHN</td><td>Chemical structure name</td>
</tr>
<tr>
<td>CL</td><td>Class</td>
</tr>
<tr>
<td>CMN</td><td>Common name</td>
</tr>
<tr>
<td>CN</td><td>LOINC official component name</td>
</tr>
<tr>
<td>CO</td><td>Component name (these are hierarchical terms, as opposed to the LOINC component names which are analytes)</td>
</tr>
<tr>
<td>CPR</td><td>Concept property</td>
</tr>
<tr>
<td>CP</td><td>ICPC component process (in original form)</td>
</tr>
<tr>
<td>CR</td><td>Concept relationship</td>
</tr>
<tr>
<td>CSN</td><td>Chemical Structure Name</td>
</tr>
<tr>
<td>CSY</td><td>Code system</td>
</tr>
<tr>
<td>CS</td><td>Short component process in ICPC, i.e. include some abbreviations</td>
</tr>
<tr>
<td>CU</td><td>Common usage</td>
</tr>
<tr>
<td>CV</td><td>Content view</td>
</tr>
<tr>
<td>CX</td><td>Component, with abbreviations expanded.</td>
</tr>
<tr>
<td>DC10</td><td>Diagnostic criteria for ICD10 code</td>
</tr>
<tr>
<td>DC9</td><td>Diagnostic criteria for ICD9 code</td>
</tr>
<tr>
<td>DEV</td><td>Descriptor entry version</td>
</tr>
<tr>
<td>DE</td><td>Descriptor</td>
</tr>
<tr>
<td>DFG</td><td>Dose Form Group</td>
</tr>
<tr>
<td>DF</td><td>Dose Form</td>
</tr>
<tr>
<td>DI</td><td>Disease name</td>
</tr>
<tr>
<td>DN</td><td>Display Name</td>
</tr>
<tr>
<td>DO</td><td>Domain</td>
</tr>
<tr>
<td>DP</td><td>Drug Product</td>
</tr>
<tr>
<td>DSV</td><td>Descriptor sort version</td>
</tr>
<tr>
<td>DS</td><td>Short form of descriptor</td>
</tr>
<tr>
<td>DT</td><td>Definitional term, present in the Metathesaurus because of its connection to a Dorland's definition or to a definition created especially for the Metathesaurus</td>
</tr>
<tr>
<td>EP</td><td>Print entry term</td>
</tr>
<tr>
<td>EQ</td><td>Equivalent name</td>
</tr>
<tr>
<td>ES</td><td>Short form of entry term</td>
</tr>
<tr>
<td>ETAL</td><td>Entry Term Alias</td>
</tr>
<tr>
<td>ETCF</td><td>Entry term, consumer friendly description</td>
</tr>
<tr>
<td>ETCLIN</td><td>Entry term, clinician description</td>
</tr>
<tr>
<td>ET</td><td>Entry term</td>
</tr>
<tr>
<td>EX</td><td>Expanded form of entry term</td>
</tr>
<tr>
<td>FBD</td><td>Foreign brand name</td>
</tr>
<tr>
<td>FI</td><td>Finding name</td>
</tr>
<tr>
<td>FN</td><td>Full form of descriptor</td>
</tr>
<tr>
<td>FSY</td><td>Foreign Synonym</td>
</tr>
<tr>
<td>GLP</td><td>Global period</td>
</tr>
<tr>
<td>GN</td><td>Generic drug name</td>
</tr>
<tr>
<td>GO</td><td>Goal</td>
</tr>
<tr>
<td>GPCK</td><td>Generic Drug Delivery Device</td>
</tr>
<tr>
<td>GT</td><td>Glossary term</td>
</tr>
<tr>
<td>HC</td><td>Hierarchical class</td>
</tr>
<tr>
<td>HD</td><td>Hierarchical descriptor</td>
</tr>
<tr>
<td>HGJKN1</td><td>Japanese High Level Group Term (kana1)</td>
</tr>
<tr>
<td>HGJKN</td><td>Japanese High Level Group Term (kana)</td>
</tr>
<tr>
<td>HG</td><td>High Level Group Term</td>
</tr>
<tr>
<td>HS</td><td>Short or alternate version of hierarchical term</td>
</tr>
<tr>
<td>HTJKN1</td><td>Japanese Hierarchical term (kana1)</td>
</tr>
<tr>
<td>HTJKN</td><td>Japanese Hierarchical term (kana)</td>
</tr>
<tr>
<td>HTN</td><td>HL7 Table Name</td>
</tr>
<tr>
<td>HT</td><td>Hierarchical term</td>
</tr>
<tr>
<td>HX</td><td>Expanded version of short hierarchical term</td>
</tr>
<tr>
<td>ID</td><td>Nursing indicator</td>
</tr>
<tr>
<td>IN</td><td>Name for an ingredient</td>
</tr>
<tr>
<td>IS</td><td>Obsolete Synonym</td>
</tr>
<tr>
<td>IT</td><td>Index term</td>
</tr>
<tr>
<td>IVC</td><td>Intervention categories</td>
</tr>
<tr>
<td>IV</td><td>Intervention</td>
</tr>
<tr>
<td>LA</td><td>LOINC answer</td>
</tr>
<tr>
<td>LC</td><td>Long common name</td>
</tr>
<tr>
<td>LG</td><td>LOINC group</td>
</tr>
<tr>
<td>LLTJKN1</td><td>Japanese Lower Level Term (kana1)</td>
</tr>
<tr>
<td>LLTJKN</td><td>Japanese Lower Level Term (kana)</td>
</tr>
<tr>
<td>LLT</td><td>Lower Level Term</td>
</tr>
<tr>
<td>LN</td><td>LOINC official fully specified name</td>
</tr>
<tr>
<td>LO</td><td>Obsolete official fully specified name</td>
</tr>
<tr>
<td>LPDN</td><td>LOINC parts display name</td>
</tr>
<tr>
<td>LPN</td><td>LOINC parts name</td>
</tr>
<tr>
<td>LS</td><td>Expanded system/sample type (The expanded version was created for the Metathesaurus and includes the full name of some abbreviations.)</td>
</tr>
<tr>
<td>LVDN</td><td>Linguistic variant display name</td>
</tr>
<tr>
<td>LV</td><td>Lexical variant</td>
</tr>
<tr>
<td>MD</td><td>CCS multi-level diagnosis categories</td>
</tr>
<tr>
<td>MH</td><td>Main heading</td>
</tr>
<tr>
<td>MIN</td><td>name for a multi-ingredient</td>
</tr>
<tr>
<td>MP</td><td>Preferred names of modifiers</td>
</tr>
<tr>
<td>MS</td><td>Multum names of branded and generic supplies or supplements</td>
</tr>
<tr>
<td>MTH_ACR</td><td>MTH acronym</td>
</tr>
<tr>
<td>MTH_CN</td><td>MTH Component, with abbreviations expanded.</td>
</tr>
<tr>
<td>MTH_ET</td><td>Metathesaurus entry term</td>
</tr>
<tr>
<td>MTH_FN</td><td>MTH Full form of descriptor</td>
</tr>
<tr>
<td>MTH_HG</td><td>MTH High Level Group Term</td>
</tr>
<tr>
<td>MTH_HT</td><td>MTH Hierarchical term</td>
</tr>
<tr>
<td>MTH_HX</td><td>MTH Hierarchical term expanded</td>
</tr>
<tr>
<td>MTH_IS</td><td>Metathesaurus-supplied form of obsolete synonym</td>
</tr>
<tr>
<td>MTH_LLT</td><td>MTH Lower Level Term</td>
</tr>
<tr>
<td>MTH_LN</td><td>MTH Official fully specified name with expanded abbreviations</td>
</tr>
<tr>
<td>MTH_LO</td><td>MTH Expanded LOINC obsolete fully specified name</td>
</tr>
<tr>
<td>MTH_OAF</td><td>Metathesaurus-supplied form of obsolete active fully specified name</td>
</tr>
<tr>
<td>MTH_OAP</td><td>Metathesaurus-supplied form of obsolete active preferred term</td>
</tr>
<tr>
<td>MTH_OAS</td><td>Metathesaurus-supplied form of obsolete active synonym</td>
</tr>
<tr>
<td>MTH_OET</td><td>Metathesaurus obsolete entry term</td>
</tr>
<tr>
<td>MTH_OF</td><td>Metathesaurus-supplied form of obsolete fully specified name</td>
</tr>
<tr>
<td>MTH_OL</td><td>MTH Non-current Lower Level Term</td>
</tr>
<tr>
<td>MTH_OPN</td><td>Metathesaurus obsolete preferred term, natural language form</td>
</tr>
<tr>
<td>MTH_OP</td><td>Metathesaurus obsolete preferred term</td>
</tr>
<tr>
<td>MTH_OS</td><td>MTH System-organ class</td>
</tr>
<tr>
<td>MTH_PTGB</td><td>Metathesaurus-supplied form of British preferred term</td>
</tr>
<tr>
<td>MTH_PTN</td><td>Metathesaurus preferred term, natural language form</td>
</tr>
<tr>
<td>MTH_PT</td><td>Metathesaurus preferred term</td>
</tr>
<tr>
<td>MTH_RXN_BD</td><td>RxNorm Created BD</td>
</tr>
<tr>
<td>MTH_RXN_CDC</td><td>RxNorm Created CDC</td>
</tr>
<tr>
<td>MTH_RXN_CD</td><td>RxNorm Created CD</td>
</tr>
<tr>
<td>MTH_RXN_DP</td><td>RxNorm Created DP</td>
</tr>
<tr>
<td>MTH_SI</td><td>MTH Sign or symptom of</td>
</tr>
<tr>
<td>MTH_SMQ</td><td>Metathesaurus version of Standardised MedDRA Query</td>
</tr>
<tr>
<td>MTH_SYGB</td><td>Metathesaurus-supplied form of British synonym</td>
</tr>
<tr>
<td>MTH_SY</td><td>MTH Designated synonym</td>
</tr>
<tr>
<td>MV</td><td>Multi-level procedure category</td>
</tr>
<tr>
<td>N1</td><td>Chemical Abstracts Service Type 1 name of a chemical</td>
</tr>
<tr>
<td>NA</td><td>Name aliases</td>
</tr>
<tr>
<td>NM</td><td>Name of Supplementary Concept</td>
</tr>
<tr>
<td>NPT</td><td>HL7 non-preferred for language term</td>
</tr>
<tr>
<td>NP</td><td>Non-preferred term</td>
</tr>
<tr>
<td>NS</td><td>Short form of non-preferred term</td>
</tr>
<tr>
<td>NX</td><td>Expanded form of non-preferred term</td>
</tr>
<tr>
<td>OAF</td><td>Obsolete active fully specified name</td>
</tr>
<tr>
<td>OAM</td><td>Obsolete Modifier Abbreviation</td>
</tr>
<tr>
<td>OAP</td><td>Obsolete active preferred term</td>
</tr>
<tr>
<td>OAS</td><td>Obsolete active synonym</td>
</tr>
<tr>
<td>OA</td><td>Obsolete abbreviation</td>
</tr>
<tr>
<td>OC</td><td>Nursing outcomes</td>
</tr>
<tr>
<td>OET</td><td>Obsolete entry term</td>
</tr>
<tr>
<td>OF</td><td>Obsolete fully specified name</td>
</tr>
<tr>
<td>OLC</td><td>Obsolete Long common name</td>
</tr>
<tr>
<td>OLG</td><td>Obsolete LOINC group name</td>
</tr>
<tr>
<td>OLJKN1</td><td>Japanese Non-current Lower Level Term (kana1)</td>
</tr>
<tr>
<td>OLJKN</td><td>Japanese Non-current Lower Level Term (kana)</td>
</tr>
<tr>
<td>OL</td><td>Non-current Lower Level Term</td>
</tr>
<tr>
<td>OM</td><td>Obsolete modifiers in HCPCS</td>
</tr>
<tr>
<td>ONP</td><td>Obsolete non-preferred for language term</td>
</tr>
<tr>
<td>OOSN</td><td>Obsolete official short name</td>
</tr>
<tr>
<td>OPN</td><td>Obsolete preferred term, natural language form</td>
</tr>
<tr>
<td>OP</td><td>Obsolete preferred name</td>
</tr>
<tr>
<td>OR</td><td>Orders</td>
</tr>
<tr>
<td>OSJKN1</td><td>Japanese System-organ class in the WHO Adverse Reaction Terminology (kana1)</td>
</tr>
<tr>
<td>OSJKN</td><td>Japanese System-organ class in the WHO Adverse Reaction Terminology (kana)</td>
</tr>
<tr>
<td>OSN</td><td>Official short name</td>
</tr>
<tr>
<td>OS</td><td>System-organ class</td>
</tr>
<tr>
<td>PCE</td><td>Preferred entry term for Supplementary Concept</td>
</tr>
<tr>
<td>PC</td><td>Preferred "trimmed term" in ICPC</td>
</tr>
<tr>
<td>PEP</td><td>Preferred entry term</td>
</tr>
<tr>
<td>PHENO_ET</td><td>Phenotype entry term</td>
</tr>
<tr>
<td>PHENO</td><td>Phenotype</td>
</tr>
<tr>
<td>PIN</td><td>Name from a precise ingredient</td>
</tr>
<tr>
<td>PM</td><td>Machine permutation</td>
</tr>
<tr>
<td>PN</td><td>Metathesaurus preferred name</td>
</tr>
<tr>
<td>POS</td><td>Place of service</td>
</tr>
<tr>
<td>PQ</td><td>Qualifier for a problem</td>
</tr>
<tr>
<td>PR</td><td>Name of a problem</td>
</tr>
<tr>
<td>PSC</td><td>Protocol selection criteria</td>
</tr>
<tr>
<td>PSN</td><td>Prescribable Names</td>
</tr>
<tr>
<td>PS</td><td>Short forms that needed full specification</td>
</tr>
<tr>
<td>PTAV</td><td>Preferred Allelic Variant</td>
</tr>
<tr>
<td>PTCS</td><td>Preferred Clinical Synopsis</td>
</tr>
<tr>
<td>PTGB</td><td>British preferred term</td>
</tr>
<tr>
<td>PTJKN1</td><td>Japanese Designated preferred name (kana1)</td>
</tr>
<tr>
<td>PTJKN</td><td>Japanese Designated preferred name (kana)</td>
</tr>
<tr>
<td>PTN</td><td>Preferred term, natural language form</td>
</tr>
<tr>
<td>PT</td><td>Designated preferred name</td>
</tr>
<tr>
<td>PXQ</td><td>Preferred qualifier term</td>
</tr>
<tr>
<td>PX</td><td>Expanded preferred terms (pair with PS)</td>
</tr>
<tr>
<td>QAB</td><td>Qualifier abbreviation</td>
</tr>
<tr>
<td>QEV</td><td>Qualifier entry version</td>
</tr>
<tr>
<td>QSV</td><td>Qualifier sort version</td>
</tr>
<tr>
<td>RAB</td><td>Root abbreviation</td>
</tr>
<tr>
<td>RHT</td><td>Root hierarchical term</td>
</tr>
<tr>
<td>RPT</td><td>Root preferred term</td>
</tr>
<tr>
<td>RSY</td><td>Root synonym</td>
</tr>
<tr>
<td>RS</td><td>Extracted related names in SNOMED2</td>
</tr>
<tr>
<td>RT</td><td>Term that is related to, but often considered non-synonymous with, the preferred term</td>
</tr>
<tr>
<td>RXN_IN</td><td>Rxnorm Preferred Ingredient</td>
</tr>
<tr>
<td>RXN_PT</td><td>Rxnorm Preferred</td>
</tr>
<tr>
<td>SBDC</td><td>Semantic Branded Drug Component</td>
</tr>
<tr>
<td>SBDFP</td><td>Semantic branded drug and form with precise ingredient as basis of strength</td>
</tr>
<tr>
<td>SBDF</td><td>Semantic branded drug and form</td>
</tr>
<tr>
<td>SBDG</td><td>Semantic branded drug group</td>
</tr>
<tr>
<td>SBD</td><td>Semantic branded drug</td>
</tr>
<tr>
<td>SB</td><td>Named subset of a source</td>
</tr>
<tr>
<td>SCALE</td><td>Scale</td>
</tr>
<tr>
<td>SCDC</td><td>Semantic Drug Component</td>
</tr>
<tr>
<td>SCDFP</td><td>Semantic clinical drug and form with precise ingredient as basis of strength</td>
</tr>
<tr>
<td>SCDF</td><td>Semantic clinical drug and form</td>
</tr>
<tr>
<td>SCDGP</td><td>Semantic clinical drug group with precise ingredient as basis of strength</td>
</tr>
<tr>
<td>SCDG</td><td>Semantic clinical drug group</td>
</tr>
<tr>
<td>SCD</td><td>Semantic Clinical Drug</td>
</tr>
<tr>
<td>SCN</td><td>Scientific name</td>
</tr>
<tr>
<td>SC</td><td>Special Category term</td>
</tr>
<tr>
<td>SD</td><td>CCS diagnosis categories</td>
</tr>
<tr>
<td>SI</td><td>Name of a sign or symptom of a problem</td>
</tr>
<tr>
<td>SMQ</td><td>Standardised MedDRA Query</td>
</tr>
<tr>
<td>SP</td><td>CCS procedure categories</td>
</tr>
<tr>
<td>SSN</td><td>Source short name, used in the UMLS Knowledge Source Server</td>
</tr>
<tr>
<td>SS</td><td>Synonymous "short" forms</td>
</tr>
<tr>
<td>ST</td><td>Step</td>
</tr>
<tr>
<td>SU</td><td>Active Substance</td>
</tr>
<tr>
<td>SX</td><td>Mixed-case component synonym with expanded abbreviations</td>
</tr>
<tr>
<td>SYGB</td><td>British synonym</td>
</tr>
<tr>
<td>SYN</td><td>Designated alias</td>
</tr>
<tr>
<td>SY</td><td>Designated synonym</td>
</tr>
<tr>
<td>TA</td><td>Task</td>
</tr>
<tr>
<td>TC</td><td>Term class</td>
</tr>
<tr>
<td>TG</td><td>Name of the target of an intervention</td>
</tr>
<tr>
<td>TMSY</td><td>Tall Man synonym</td>
</tr>
<tr>
<td>TQ</td><td>Topical qualifier</td>
</tr>
<tr>
<td>TX</td><td>CCPSS synthesized problems for TC termgroup</td>
</tr>
<tr>
<td>UCN</td><td>Unique common name</td>
</tr>
<tr>
<td>UE</td><td>Unique equivalent name</td>
</tr>
<tr>
<td>USN</td><td>Unique scientific name</td>
</tr>
<tr>
<td>USY</td><td>Unique synonym</td>
</tr>
<tr>
<td>VAB</td><td>Versioned abbreviation</td>
</tr>
<tr>
<td>VPT</td><td>Versioned preferred term</td>
</tr>
<tr>
<td>VSY</td><td>Versioned synonym</td>
</tr>
<tr>
<td>VS</td><td>Value Set</td>
</tr>
<tr>
<td>XD</td><td>Expanded descriptor in AOD</td>
</tr>
<tr>
<td>XM</td><td>Cross mapping set</td>
</tr>
<tr>
<td>XQ</td><td>Alternate name for a qualifier</td>
</tr>
</tbody>
</table>

</body></html>