title=OMIM (Online Mendelian Inheritance in Man) - Source Representation
date=2018-05-07
updated=2020-05-04
type=page
status=published
~~~~~~


<!--menu-->
<div class="row">
 <div class="container-fluid navbar navbar-default vocab-docs-navbar" role="navigation">
  <div class="btn-group navbar-btn visible-*" role="group">
   <a class="btn btn-md" role="button" href="index.html">Synopsis</a>
   <a class="btn btn-md" role="button" href="metadata.html">Metadata</a>
   <a class="btn btn-md" role="button" href="stats.html">Statistics</a>
   <a class="btn btn-md" role="button" href="sourcerepresentation.html">Source Representation</a>
   <a class="btn btn-md" role="button" href="metarepresentation.html">Metathesaurus Representation</a>
  </div>
 </div>
</div>
<!--endmenu-->
<br/>
    <p>This page lists specific source data elements and provides
      information on their representation in the UMLS Metathesaurus.</p>
    <div class="row py-2 mb-2 mini-menu">  
    	<div class="col-sm-auto">Skip to:</div>
    	<div class="col-sm-auto"><a href="#change_summary">Summary of Changes</a></div>
    	<div class="col-sm-auto border-left"><a href="#file_summary">Summary of Source-Provided Files</a></div>
    	<div class="col-sm-auto border-left"><a href="#file_details">Source-Provided File Details</a></div>
    </div>
    <p><strong>VSAB: OMIM2020_07_27</strong>
      </p>
      <hr>
    <div id="change_summary" class="rep_block_level_one"> 
            <h3>Summary of Changes</h3>
        <p>No changes were made to the OMIM source format, however significant changes were made to UMLS source processing:</p>
		New ATNs: <br>
		<ul>
			<li>QUALIFIER: Frequency information previously enclosed in parentheses for a PTCS, PHENO or PHENO_ET atom.  STYPE = RUI</li>
			<li>SOS: Previously represented as PTCS atoms treed under "MISCELLANEOUS" or "MOLECULAR BASIS".</li>
		</ul>
		Explanation:<br>
		
		<p>QUALIFIER: For atoms with TTY of PTCS, PHENO or PHENO_ET, frequency information was often included parenthetically.  Based on pattern matching, this information was stripped from the atom 
			string and is now represented as an attribute on the relationship between the atom and its preferred form (ATN = QUALIFIER).  Examples of "stripped" information 
			include "in some patients," "rare," "1 family," and "less common." </p>
			Example: <br>
			Previous representaton: <br>
			<ul>
				<li>ATOM:  STR = "Doughy or puffy skin (rare)"; TTY = PTCS</li>
				<li>RELATIONSHIP:  RELA = "manifestation_of" to OMIM PT atom "DESBUQUOIS DYSPLASIA 2" (CODE = 615777)</li>
				<li>ATTRIBUTE: N/A</li>
			</ul>
			Current representation: <br>
			<ul>
				<li>ATOM:  STR = "Doughy or puffy skin"; TTY = PTCS</li>
				<li>RELATIONSHIP:  RELA = "manifestation_of" to OMIM PT atom "DESBUQUOIS DYSPLASIA 2" (CODE = 615777)</li>
				<li>ATTRIBUTE:  ATN = "QUALIFIER"; ATV = "rare"; STYPE = "RUI"</li>
			</ul>
		<p>SOS: Atoms in the "MISCELLANEOUS" and "MOLECULAR BASIS" branches of the PTCS tree were removed and are now represented as attributes (ATN = SOS; STYPE = AUI)</p>
		Examples:<br>
		<ul>
			<li>Caused by mutations in merlin (NF2, 101000.0001) [MOLECULAR BASIS]</li>
			<li>Incidence of 1 in 25,000 livebirths [MISCELLANEOUS]</li>
		</ul>
</div>
      <hr>
    <div id="file_summary" class="rep_block_level_one">  
      <h3>Source-Provided Files: Summary</h3>

      <p>XML records are supplied according to Metathesaurus
        specifications. </p>

      <h4>Documentation and Reference</h4>
      <div class="rep_block_level_two">
        <table>
          <tbody>
            <tr>
              <th>File Name</th>
              <th>Description</th>
            </tr>
            <tr>
              <td class="file_field_name"><a
                  href="http://www.ncbi.nlm.nih.gov/dtd/NCBI_Mim.mod.dtd">http://www.ncbi.nlm.nih.gov/dtd/NCBI_Mim.mod.dtd</a>
              </td>
              <td>XML dtd</td>
            </tr>
            <tr>
              <td class="file_field_name"><a
                  href="http://www.ncbi.nlm.nih.gov/Omim/omimfaq.html">http://omim.org/help/faq</a><br/>
              </td>
              <td>FAQ</td>
            </tr>
          </tbody>
        </table>
        
        <h4>XML Data</h4>
        <p><strong>Not included</strong>:
    Data from
      many tags are not included in Metathesaurus source processing,
			because they do not contain terminological content.</p></div>
      <hr>
      <h3 id="file_details">Source-Provided Data: Details </h3>

      <p>The following is a list of elements available for
        OMIM data in XML format.
      </p>
      <p><strong>Required Elements: </strong>
      </p>
        <table>
          <tbody>
            <tr>
              <th>Element</th>
              <th>Description</th>
              <th>Representation</th>
            </tr>
            <tr>
              <td>&lt;Mim-entry_mimNumber&gt; </td>
              <td>Unique identifier</td>
              <td>MRCONSO.CODE<br/>
                MRCONSO.SDUI<br/>
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_mimType&gt; </td>
              <td>The type of OMIM records (asterisk (*), number (#),
                plus (+), percent (%), caret (^), or none)<br/>
                <p><strong>An asterisk (*)</strong> before an entry number
                  indicates a gene of known sequence.</p>
                <p><strong>A number symbol (#)</strong> before an entry number
                  indicates that it is a descriptive entry, usually of a
                  phenotype, and does not represent a unique locus. The
                  reason for the use of the #-sign is given in the first
                  paragraph of the entry. Discussion of any gene(s)
                  related to the phenotype resides in another entry(ies)
                  as described in the first paragraph.</p>
                <p><strong>A plus sign (+)</strong> before an entry number
                  indicates that the entry contains the description of a
                  gene of known sequence and a phenotype.</p>
                <p><strong>A percent sign (%)</strong> before an entry number
                  indicates that the entry describes a confirmed
                  mendelian phenotype or phenotypic locus for which the
                  underlying molecular basis is not known.</p>
                <p><strong>No symbol</strong> before an entry number generally
                  indicates a description of a phenotype for which the
                  mendelian basis, although suspected, has not been
                  clearly established or that the separateness of this
                  phenotype from that in another entry is unclear.</p>
                <p><strong>A caret symbol (^)</strong> before an entry number
                  means the entry no longer exists because it was
                  removed from the database or moved to another entry as
                  indicated.</p>
                <br/>
              </td>
              <td>MRSAT.ATN = "MIMTYPEMEANING"<br/>
                <br/>
                MRSAT.ATN = "MIMTYPEVALUE"<br/>
                <br/>
                MRSAT.ATN = "MIMTYPE"<br/>
                <br/>
                Records with a &lt;Mim-entry_mimType&gt; value of caret
                (^) are obsolete and are not included in Metathesaurus
                source processing.<br/>
                <br/>
                <table>
                  <tbody>
                    <tr>
                      <th>MIMTYPE</th>
                      <th>MIMTYPEVALUE</th>
                      <th>MIMTYPEMEANING</th>
                    </tr>
                    <tr>
                      <td>0</td>
                      <td>none</td>
                      <td>Other, mainly phenotypes with suspected Mendelian basis</td>
                    </tr>
                    <tr>
                      <td>1</td>
                      <td>star</td>
                      <td>Gene with known sequence</td>
                    </tr>
                    <tr>
                      <td>3</td>
                      <td>pound</td>
                      <td>Phenotype description, molecular basis known</td>
                    </tr>
                    <tr>
                      <td>4</td>
                      <td>plus</td>
                      <td>Gene with known sequence and phenotype</td>
                    </tr>
                    <tr>
                      <td>5</td>
                      <td>perc</td>
                      <td>Mendelian phenotype or locus, molecular basis unknown</td>
                    </tr>
                  </tbody>
                </table>
                <br/>
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_title&gt;</td>
              <td>Contains the preferred term naming the entry. May include semicolon-separated acronyms. </td>
              <td>MRCONSO.TTY = "PT", MRCONSO.TTY = "ACR"<br/>
                <br/>
                Information before the first semicolon is used to create
                PT atoms. Information following the first
                semicolon is used to create ACR atoms.<br/>
                <br/>
                Relationships are created to connect PT to ACR atoms:<br/>
                REL = SY<br/>
                RELA = expanded_form_of/has_expanded_form
              </td>
            </tr>
          </tbody>
        </table>
      <p><strong>Optional Elements</strong>
      </p>
      
        <table>
          <tbody>
            <tr>
              <th>Element</th>
              <th>Description</th>
              <th>Representation</th>
            </tr>
            <tr>
              <td>&lt;Mim-entry_symbol&gt;</td>
              <td>One or more commas-separated gene symbols</td>
              <td>MRSAT.ATN = "GENESYMBOL"</td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_locus&gt;</td>
              <td>Notation for one or more loci for the genes associated
                with this entry</td>
              <td>MRSAT.ATN = "GENELOCUS"</td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_aliases_E&gt;</td>
              <td>Synonyms/acronyms</td>
              <td>MRCONSO.TTY = "ETAL"<br/>
                MRCONSO.TTY = "ACR"<br/>
                <br/>
                Information before the first semicolon is used to create
                ETAL atoms<br/>
                <br/>
                Information after the semicolon is parsed to create ACR
                atoms<br/>
                <br/>
                ", INCLUDED" is removed from both ETAL and ACR atoms<br/>
                <br/>
                No ETAL atom is created if the string already exists as
                an ACR atom<br/>
                <br/>
                Relationships are created to connect PT to ETAL atoms:<br/>
                REL = RQ<br/>
                RELA = has_alias/alias_of<br/>
                <br/>
                Relationships are created to connect ETAL to ACR atoms:<br/>
                REL = SY<br/>
                RELA = expanded_form_of/has_expanded_form
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_included_E&gt; </td>
              <td>Entry terms/acronyms</td>
              <td>MRCONSO.TTY = "ET"<br/>
                MRCONSO.TTY = "ACR"<br/>
                <br/>
                Information before the first semicolon is used
                  to create ET atoms<br/>
                  <br/>
                Information after the semicolon is parsed to
                create ACR atoms<br/>
                <br/>
                The pattern ", INCLUDED" is removed from the string<br/>
                <br/>
                Relationships are created to connect ET to ACR atoms:<br/>
                REL = SY<br/>
                RELA = expanded_form_of/has_expanded_form
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-allelic-variant_number&gt;</td>
              <td>Allelic variant number (4 digits)</td>
              <td>MRCONSO.CODE, MRCONSO.SDUI: Appended to the
                Mim-entry_mimNumber for TTY= PTAV
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-allelic-variant_name&gt;</td>
              <td>Allelic variant name.  Most allelic variants
                represent disease-producing mutations.  A few
                polymorphisms are included. For most genes, only
                selected mutations are included as specific entries. In many cases, parenthetical information such as "1 family" or "1 patient" indicates frequency.  
				  This segment of the string has been stripped and is represented as an attribute connected to the PHENO - PTAV RUI.
              </td>
              <td>MRCONSO.TTY = "PHENO"<br/>
                <br/>
                Relationships are created to connect PHENO to PTAV
                atoms:<br/>
                REL = RO<br/>
                RELA = phenotype_of/has_phenotype
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-allelic-variant_aliases_E&gt;</td>
              <td>Allelic variant aliases</td>
              <td>MRCONSO.TTY = "PHENO_ET"<br/>
                <br/>
                The pattern ", INCLUDED" is removed from the string. In many cases, parenthetical information such as "1 family" or "1 patient" indicates frequency.  
				  This segment of the string has been stripped and is represented as an attribute connected to the PHENO_ET - PTAV RUI.<br/>
                <br/>
                Relationships are created to connect PHENO_ET to PTAV
                atoms:<br/>
                REL = RO<br/>
                RELA = phenotype_of/has_phenotype
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-allelic-variant_mutation&gt;&lt;Mim_text_text&gt;
              </td>
              <td>The type of mutations which show statistical
                correlation to the particular disorders.
              </td>
              <td>MRCONSO.TTY = "PTAV"<br/>
                <br/>
                Relationships are created to connect PTAV to PT atoms:<br/>
                REL = RO<br/>
                RELA = allelic_variant_of/has_allelic_variant<br/>
                <br/>
                Relationships are created to connect PTAV to PHENO and
                PHENO_ET atoms:<br/>
                REL = RO<br/>
                RELA = has_phenotype/phenotype_of<br/>
                <br/>
                The CODE/SDUI has the
                    Mim-allelic-variant_number appended.

              </td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_clinicalSynopsis&gt;&lt;Mim-index-term_key&gt;</td>
              <td>Clinical synopsis header terms
              </td>
              <td>MRCONSO.TTY = "HT"<br/>
                <br/>
                Values are excluded from Metathesaurus source processing
                if they meet any of the following criteria:
                <ul>
                  <li>Mixed-case/lower-case string. These values
                    have not yet been reviewed by OMIM </li>
                  <li>Have an
                    &lt;Mim-entry_synopsisEditHistory&gt;&lt;Mim-date_year&gt;
                    before 1996 </li>
                  <li>Do not have a "MIM-entry_synopsisEditHistory" tag<br/>
                  </li>
                </ul>
              </td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_clinicalSynopsis&gt;&lt;Mim-index-term_terms_E&gt;</td>
              <td>Clinical synopsis terms, may be subheadings or leaf nodes</td>
              <td>MRCONSO.TTY = "HT"<br/>
                MRCONSO.TTY = "PTCS"<br/>
                <br/>
                If the value is contained
                  within square brackets, the entry is considered a
                  subheading and TTY = "HT" is assigned; otherwise it is
                  considered a leaf node and TTY= "PTCS" is assigned. In many cases, parenthetical information such as "in some families" or "rare" indicates frequency.  
				  This segment of the string has been stripped and is represented as an attribute connected to the PTCS - PT RUI.  Values that are treed under "MISCELLANEOUS"
				  or "MOLECULAR BASIS" are represented as attributes (ATN = SOS).<br/>
                <br/>
                Relationships are created to connect PTCS to PT atoms:<br/>
                REL = RO<br/>
                RELA = manifestation_of/has_manifestation<br/>
                <br/>
                Relationships are created to connect PTCSto PT atoms:<br/>
                REL = RO<br/>
                RELA = inheritance_type_of/has_inheritance_type<br/>

              </td>
            </tr>
            <tr>
              <td>&lt;Mim-entry_synopsisEditHistory&gt;&lt;Mim-date_year&gt;</td>
              <td>Clinical synopsis year</td>
              <td>Used to determine which clinical synopsis records to
                process for inclusion in the Metathesaurus. <br/>
                Records added before 1996 are not processed.
              </td>
            </tr>
          </tbody>
        </table>
    </div>
