<p>This page lists specific source data elements and provides information on their representation in the UMLS Metathesaurus.</p>
<!--begin tabbed navigation area-->
   <div class="section-5">
      <ul id="menu">
         <li id="nav-1"><a href="index.html">Synopsis</a></li>
         <li id="nav-2"><a href="metadata.html">Source Metadata</a></li>
         <li id="nav-3"><a href="stats.html">Statistics</a></li>
         <li id="nav-4"><a href="samples.html">Samples</a></li>
         <li id="nav-5"><a href="sourcerepresentation.html">Source Representation</a></li>
         <li id="nav-6"><a href="metarepresentation.html">Metathesaurus Representation</a></li>
      </ul>
   </div>
<!-- end tabbed navigation area -->
<br/>
<a name = "top"></a>
<span class = "nav">Skip to: <a href = "#general">General Notes/Comments</a>, <a href = "#change_summary">Summary of Changes</a>, <a href = "#file_summary">Summary of Source-Provided Files</a> <a href = "#file_details">Source-Provided File Details</a></span>
<br/>
<br/>
<strong>VSAB:</strong> HGNC2015_03
<a name = "general"></a>
<div id = "rep_general" class = "rep_block_level_one">
<h3>General Notes/Comments:</h3>
<p>The HUGO Gene Nomenclature Committee (HGNC) is an international organization whose primary ethos is to promote and sustain collaboration in the field of human genetics.</p>
<p>Data is supplied in a single, tab-delimited, text file.</p>
</div>
<div id = "rep_change_summary" class = "rep_block_level_one">
<a name = "change_summary"></a>
<h3>Summary of Changes</h3>&nbsp;&nbsp;<span class = "nav">(return to <a href = "#top">top</a></span>)
<p>There are no changes for this version.</p>
</div>

<div id = "rep_file_summary" class = "rep_block_level_one">
<a name = "file_summary"></a>
<h3>Summary of Source-Provided Files</h3>&nbsp;&nbsp;<span class = "nav">(return to <a href = "#top">top</a></span>)

<div class = "rep_block_level_two">
<!-- Make a table of each file that comes from the source   -->
<h4>Documentation and Reference</h4>
<table>
<tr><th>Web site</th></tr>
<tr><td>www.genenames.org</td></tr>
<tr><td>www.hugo-international.org</td></tr>
</table>
<br/>
<table>
<tr><th>File Name</th><th>Description</th></tr>
<tr><td  class = "file_file_name">hgnc_complete_set_03232015.txt</td><td>Tab-delimited text file.</td></tr>
</table>
</div>
</div>

<div id = "rep_file_details" class = "rep_block_level_one">
<a name = "file_details"></a>
<h3>Source-Provided File Details</h3>&nbsp;&nbsp;<span class = "nav">(return to <a href = "#top">top</a></span>)

<div class = "rep_block_level_two">
<a name = "file1"></a>
<table>
<tr><th>Field Number</th><th>Field Name</th><th>Description</th><th>Representation</th></tr>
<tr><td  class = "file_field_name">1</td><td>HGNC ID</td><td>A unique ID provided by the HGNC</td><td>MRCONSO.CODE; MRCONSO.SCUI</td></tr>
<tr><td  class = "file_field_name">2</td><td>Approved Symbol</td><td>Official gene symbol</td><td>MRCONSO.STR; MRSAT.ATN=GENESYMBOL</td></tr>
<tr><td  class = "file_field_name">3</td><td>Approved Name</td><td>Official gene name</td><td>MRCONSO.STR</td></tr>
<tr><td  class = "file_field_name">4</td><td>Status</td><td>Indicates gene classification (e.g., withdrawn, approved)</td><td>Not processed</td></tr>
<tr><td  class = "file_field_name">5</td><td>Locus Type</td><td>Specifies type of locus (e.g., pseudogene, protocadherin)</td><td>MRSAT.ATN=LOCUS_TYPE</td></tr>
<tr><td  class = "file_field_name">6</td><td>Locus Group</td><td>Locus Group</td><td>MRSAT.ATN=LOCUS_GROUP</td></tr>
<tr><td  class = "file_field_name">7</td><td>Previous Symbols</td><td>Symbols previously approved</td><td>MRSAT.ATN=PREV_SYMBOL</td></tr>
<tr><td  class = "file_field_name">8</td><td>Previous Names</td><td>Gene names previously approved</td><td>MRSAT.ATN=PREV_NAME</td></tr>
<tr><td  class = "file_field_name">9</td><td>Synonyms</td><td>Other symbols used to refer to the gene</td><td>MRCONSO.STR</td></tr>
<tr><td  class = "file_field_name">10</td><td>Name Synonyms</td><td>Other names used to refer to the gene</td><td>MRCONSO.STR</td></tr>
<tr><td  class = "file_field_name">11</td><td>Chromosome</td><td>Indicates the location of the gene on the chromosome</td><td>MRSAT.ATN=CHROMOSOME</td></tr>
<tr><td  class = "file_field_name">12</td><td>Date Approved</td><td>Date gene symbol and name were approved</td><td>MRSAT.ATN=DATE_CREATED</td></tr>
<tr><td  class = "file_field_name">13</td><td>Date Modified</td><td>Date entry was modified</td><td>MRSAT.ATN=DATE_LAST_MODIFIED</td></tr>
<tr><td  class = "file_field_name">14</td><td>Date Symbol Changed</td><td>Date gene symbol was last changed</td><td>MRSAT.ATN=DATE_SYMBOL_CHANGED</td></tr>
<tr><td  class = "file_field_name">15</td><td>Date Name Changed</td><td>Date the gene name was last changed</td><td>MRSAT.ATN=DATE_NAME_CHANGED</td></tr>
<tr><td  class = "file_field_name">16</td><td>Accession Numbers</td><td>Accession numbers</td><td>MRSAT.ATN=ACCESSION_NO</td></tr>
<tr><td  class = "file_field_name">17</td><td>Enzyme IDs</td><td>Enzyme Commission (EC) number</td><td>MRSAT.ATN=EZ</td></tr>
<tr><td  class = "file_field_name">18</td><td>Entrez Gene ID</td><td>Entrez Gene ID</td><td>MRSAT.ATN=ENTREZGENE_ID</td></tr>
<tr><td  class = "file_field_name">19</td><td>Ensembl Gene ID</td><td>Ensembl Gene ID</td><td>MRSAT.ATN=ENSEMBLGENE_ID</td></tr>
<tr><td  class = "file_field_name">20</td><td>Mouse Genome Database ID</td><td>Mouse Genome Database ID</td><td>MRSAT.ATN=MGD_ID</td></tr>
<tr><td  class = "file_field_name">21</td><td>Specialist Database Links</td><td>Contains HTML links to specialist databases</td><td>MRSAT.ATN=DB_XR</td></tr>
<tr><td  class = "file_field_name">22</td><td>Specialist Database IDs</td><td>Contains the ID within each specialist database</td><td>MRSAT.ATN=DB_XR_ID</td></tr>
<tr><td  class = "file_field_name">23</td><td>Pubmed IDs</td><td>Identifier that links to published articles</td><td>MRSAT.ATN=PMID</td></tr>
<tr><td  class = "file_field_name">24</td><td>RefSeqIDs</td><td>RefSeqIDs</td><td>MRSAT.ATN=REFSEQ_ID</td></tr>
<tr><td  class = "file_field_name">25</td><td>Gene Family Tag</td><td>Name of the family the gene has been assigned to</td><td>MRSAT.ATN=GENE_FAM</td></tr>
<tr><td  class = "file_field_name">26</td><td>Gene Family Description</td><td>Name given to a particular gene family.</td><td>MRSAT.ATN=GENE_FAM_DESC</td></tr>
<tr><td  class = "file_field_name">27</td><td>Record Type</td><td>valid values: Parent, Standard</td><td>MRSAT.ATN=RECORD_TYPE</td></tr>
<tr><td  class = "file_field_name">28</td><td>Primary IDs</td><td>valid values: blank, hypen</td><td>MRSAT.ATN=PRIMARY_ID</td></tr>
<tr><td  class = "file_field_name">29</td><td>Secondary IDs</td><td>numeric value</td><td>MRSAT.ATN=SECONDARY_ID</td></tr>
<tr><td  class = "file_field_name">30</td><td>CCDS IDs</td><td>Consensus CDS (CCDS) project ID</td><td>MRSAT.ATN=CCDS_ID</td></tr>
<tr><td  class = "file_field_name">31</td><td>VEGA IDs</td><td>VEGA ID</td><td>MRSAT.ATN=VEGA_ID</td></tr>
<tr><td  class = "file_field_name">32</td><td>Locus Specific Databases</td><td>Contains links to databases pertinent to the gene</td><td>MRSAT.ATN=LOCUS_SPECIFIC_DB_XR</td></tr>
<tr><td  class = "file_field_name">33</td><td>Entrez Gene ID (mapped data)</td><td>Entrez Gene ID (mapped data supplied by NCBI)</td><td>MRSAT.ATN=MAPPED_ENTREZGENE_ID</td></tr>
<tr><td  class = "file_field_name">34</td><td>OMIM ID (mapped data)</td><td>OMIM ID (mapped data supplied by NCBI)</td><td>MRSAT.ATN=OMIM_NUMBER</td></tr>
<tr><td  class = "file_field_name">35</td><td>RefSeq (mapped data)</td><td>RefSeq (mapped data supplied by NCBI)</td><td>MRSAT.ATN=MAPPED_REFSEQ_ID</td></tr>
<tr><td  class = "file_field_name">36</td><td>UniProt ID (mapped data)</td><td>UniProt ID (mapped data supplied by UniProt)</td><td>MRSAT.ATN=SWP</td></tr>
<tr><td  class = "file_field_name">37</td><td>Ensembl Gene ID (mapped data)</td><td>Ensembl Gene ID (mapped data supplied by Ensembl)</td><td>MRSAT.ATN=MAPPED_ENSEMBLGENE_ID</td></tr>
<tr><td  class = "file_field_name">38</td><td>UCSC (mapped data)</td><td>UCSC (mapped data supplied by UCSC)</td><td>MRSAT.ATN=UCSC_ID</td></tr>
<tr><td  class = "file_field_name">39</td><td>Mouse Genome Database ID (mapped data)</td><td>Mouse Genome Database ID (mapped data supplied by MGI)</td><td>MRSAT.ATN=MAPPED_MGD_ID</td></tr>
<tr><td  class = "file_field_name">40</td><td>Rat Genome Database ID (mapped data)</td><td>Rat Genome Database ID (mapped data supplied by RGD)</td><td>MRSAT.ATN=MAPPED_RGD_ID</td></tr>
</table><br/>
<p>(1) The following fields can contain multiple values in a comma delimited list:<br/>
Accession Numbers<br/>
Enzyme ID<br/>
Gene Family Tag<br/>
Gene Family Description<br/>
Name Synonyms<br/>
Previous Names<br/>
Previous Symbols<br/>
Pubmed ID<br/>
RefSeq IDs<br/>
Specialist Database ID<br/>
Specialist Database Links<br/>
Synonyms<br/> <br/>

(2) HGNC's mapped data is derived from external sources and is not subject to their checking and curation procedures.</p>
</div></div>
